{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "1. download feature table of whatever complete organism\n",
    "2. parse name column for search term of interest (e.g. nitrile hydratase + related terms)\n",
    "\tif no, download genome (fasta) to neg directory and increase a neg_counter\n",
    "\t(also download gbk file and parse organism and isolation source out of it)\n",
    "\t\tif counter is hit, stop downloading negative genomes\n",
    "\tif yes, download genome (fasta) to pos directory and increase a pos_counter\n",
    "\t(also download gbk file and parse organism and isolation source out of it - could also parse the sequence for TP genes out of it and assemble it automatically)\n",
    "\t\tif counter is hit, stop downloading pos genomes\n",
    "\tstop downloading once both counters are hit\n",
    "3. pass directories on to the make coverage table scripts\n",
    "\n",
    "\n",
    "Also I need a protein alignment of homologues, so should possibly download more positive genomes than I actually need to be able to build the \n",
    "alignment as well.\n",
    "\n",
    "IMPORTANT! How to validate TPs?\n",
    "\n",
    "\n",
    "YcaO: Currently downloaded 49 of 50 pos genome(s).\n",
    "Currently downloaded 150 of 150 neg genome(s).\n",
    "Checked a total of:\t 1019 genome(s). "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Output directory will be named after BGC_type\n",
    "BGC_type = 'Rieske'\n",
    "#amount genomes does not equal amount sequences as 1 genome can have multiple hits for a search term.\n",
    "amount_neg_genomes = 25\n",
    "amount_pos_genomes = 15 #use some as TP genes and make prot alignment from some others\n",
    "#names are converted to lower case for searching, so search term should be all lower case letters\n",
    "search_term = 'lanthionine synthetasecC family protein'\n",
    "#if isolation source doesn't matter, include '' and 'not annotated' in the filter\n",
    "#isolation source information is also converted to lower case, so keep search terms lower case\n",
    "#can modify positive and negative isolation source separately.\n",
    "neg_isolation_source_filter = ['', 'not annotated', 'marine', 'sea', 'sponge', 'ocean', 'porifera', 'seafloor', 'sediment', 'water', 'tidal', 'coral', 'reef', 'coast', 'ship', 'fish', 'aquaculture', 'atlantic', 'pacific', 'mediterranean', 'baltic', 'pond']\n",
    "pos_isolation_source_filter =  ['', 'not annotated', 'marine', 'sea', 'sponge', 'ocean', 'porifera', 'seafloor', 'sediment', 'water', 'tidal', 'coral', 'reef', 'coast', 'ship', 'fish', 'aquaculture', 'atlantic', 'pacific', 'mediterranean', 'baltic', 'pond']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "from os import listdir, mkdir\n",
    "from os.path import isfile, join\n",
    "from pathlib import Path\n",
    "import pandas as pd\n",
    "import random\n",
    "import glob\n",
    "from Bio import SeqIO"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Making /nesi/project/vuw03285/Rieske\n",
      "Making /nesi/project/vuw03285/Rieske/base_genomes\n",
      "Making /nesi/project/vuw03285/Rieske/base_genomes/temp_neg_genomes\n",
      "Making /nesi/project/vuw03285/Rieske/base_genomes/temp_pos_genomes\n",
      "Making /nesi/project/vuw03285/Rieske/base_genomes/feature_tables\n",
      "Making /nesi/project/vuw03285/Rieske/base_genomes/neg_gbk_files\n",
      "Making /nesi/project/vuw03285/Rieske/base_genomes/pos_gbk_files\n"
     ]
    }
   ],
   "source": [
    "def makedir(dirpath):\n",
    "    if os.path.isdir(dirpath):\n",
    "        print(dirpath,'exists already')\n",
    "    else:\n",
    "        print('Making', dirpath)    \n",
    "        os.mkdir(dirpath)\n",
    "\n",
    "        \n",
    "# Defining paths for required directory structure for input and output files relative to parent directory\n",
    "parent_dir='/nesi/project/vuw03285/'\n",
    "BGC_path=os.path.join(parent_dir, BGC_type)\n",
    "base_genomes_path=os.path.join(BGC_path, 'base_genomes')\n",
    "neg_genomes_path=os.path.join(BGC_path, 'base_genomes/temp_neg_genomes')\n",
    "pos_genomes_path=os.path.join(BGC_path, 'base_genomes/temp_pos_genomes')\n",
    "feature_tables_path=os.path.join(BGC_path, 'base_genomes/feature_tables')\n",
    "neg_gbk_path=os.path.join(BGC_path, 'base_genomes/neg_gbk_files')\n",
    "pos_gbk_path=os.path.join(BGC_path, 'base_genomes/pos_gbk_files')\n",
    "\n",
    "\n",
    "\n",
    "# Calling function to make directories if they don't exist yet\n",
    "makedir(BGC_path)\n",
    "makedir(base_genomes_path)\n",
    "makedir(neg_genomes_path)\n",
    "makedir(pos_genomes_path)\n",
    "makedir(feature_tables_path)\n",
    "makedir(neg_gbk_path)\n",
    "makedir(pos_gbk_path)\n",
    "\n",
    "os.chdir(parent_dir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%capture cap --no-stderr\n",
    "# Generating a report file for this particular script\n",
    "print('\\nBGC_type =', BGC_type)\n",
    "print('\\namount_neg_genomes =', amount_neg_genomes)\n",
    "print('\\namount_pos_genomes =', amount_pos_genomes)\n",
    "print('\\nsearch_term =', search_term)\n",
    "print('\\nneg_isolation_source_filter =', neg_isolation_source_filter)\n",
    "print('\\npos_isolation_source_filter =', pos_isolation_source_filter, '\\n')\n",
    "with open(BGC_path+'/'+'report_download_base_genomes.txt', 'w') as f:\n",
    "    f.write(str(cap))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Using ncbi-genome-download to get a list of accession numbers of complete bacterial and archaeal genomes from refseq. Save this file as a tsv.\n",
    "Run only if file doesn't exist yet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "#%%bash\n",
    "# Download table of GCF accessions\n",
    "#including archaea: 21739 (374 more)\n",
    "#excluding archaea: 21365\n",
    "#/home/blankma/.local/bin/ncbi-genome-download bacteria,archaea -s refseq -F features -l complete --flat-output --dry-run > /nesi/project/vuw03285/complete_refseq_accessions.tsv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "# read .tsv file into a pandas dataframe\n",
    "gcf_df = pd.read_csv('complete_refseq_accessions.tsv', header=None, sep='\\t', skiprows=[0])\n",
    "gcf_df.columns=['accession','organism','strain'] \n",
    "\n",
    "#turn first column into a list of accession number to randomly select from\n",
    "accession_list = list(gcf_df.loc[:,'accession'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "# https://stackoverflow.com/questions/54662605/how-to-pass-a-python-variables-to-shell-script-in-azure-databricks-notebookbles\n",
    "#os.environ['LIST'] = ' '.join(random_gcf)\n",
    "#print(os.getenv('LIST'))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "- Also make a shutoff condition if coming back to download more genomes. I.e. parse feature table folder and see if GCF_... already exists, if so, skip (also should append to summary file in this case, but that's probably too much)\n",
    "- pos only appended to summary if the protein id is in the last or only feature"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Currently downloaded 0 of 15 pos genome(s).\n",
      "Currently downloaded 0 of 25 neg genome(s).\n",
      "Checked a total of:\t 0 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_008330925.1_ASM833092v1_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_008330925.1\n",
      "Downloading gbk of GCF_008330925.1 to negative gbk directory\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/blankma/.local/lib/python3.8/site-packages/pandas/core/indexing.py:1843: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  self.obj[item_labels[indexer[info_axis]]] = value\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Parsing gbk file of GCF_008330925.1 for organism and isolation source.\n",
      "['', 'not annotated', 'isolation_source not annotated'] in desired isolation source list.\n",
      "Add row for GCF_008330925.1 in Neisseria meningitidis strain 13-600 chromosome, complete genome to summary table\n",
      "Downloading fasta of GCF_008330925.1 to negative genomes directory.\n",
      "\n",
      "Currently downloaded 0 of 15 pos genome(s).\n",
      "Currently downloaded 1 of 25 neg genome(s).\n",
      "Checked a total of:\t 1 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_003382725.1_ASM338272v1_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_003382725.1\n",
      "Downloading gbk of GCF_003382725.1 to negative gbk directory\n",
      "Parsing gbk file of GCF_003382725.1 for organism and isolation source.\n",
      "['', 'not annotated', 'isolation_source not annotated'] in desired isolation source list.\n",
      "Add row for GCF_003382725.1 in Enterobacter hormaechei subsp. xiangfangensis strain OSUVMCKPC4-2 plasmid pOSUEC_F, complete sequence to summary table\n",
      "Downloading fasta of GCF_003382725.1 to negative genomes directory.\n",
      "\n",
      "Currently downloaded 0 of 15 pos genome(s).\n",
      "Currently downloaded 2 of 25 neg genome(s).\n",
      "Checked a total of:\t 2 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_001708555.1_ASM170855v1_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_001708555.1\n",
      "Downloading gbk of GCF_001708555.1 to negative gbk directory\n",
      "Parsing gbk file of GCF_001708555.1 for organism and isolation source.\n",
      "[''] in desired isolation source list.\n",
      "Add row for GCF_001708555.1 in Yersinia enterocolitica strain YE7 plasmid unnamed2, complete sequence to summary table\n",
      "Downloading fasta of GCF_001708555.1 to negative genomes directory.\n",
      "\n",
      "Currently downloaded 0 of 15 pos genome(s).\n",
      "Currently downloaded 3 of 25 neg genome(s).\n",
      "Checked a total of:\t 3 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_013459415.1_ASM1345941v1_feature_table.txt\n",
      "Found following hit/s for search term rieske 2fe-2s domain-containing protein: ['WP_180899995.1', 'WP_180897212.1', 'WP_180898274.1', 'WP_180898593.1']\n",
      "Downloading gbk of GCF_013459415.1 to positive gbk directory.\n",
      "Parsing gbk file of GCF_013459415.1 for organism and isolation source.\n",
      "Add row for WP_180899995.1 in Martelella sp. NC18 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_180899995.1 to TP file.\n",
      "Add row for WP_180897212.1 in Martelella sp. NC18 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_180897212.1 to TP file.\n",
      "Add row for WP_180898274.1 in Martelella sp. NC18 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_180898274.1 to TP file.\n",
      "Add row for WP_180898593.1 in Martelella sp. NC18 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_180898593.1 to TP file.\n",
      "['', 'sediment'] in desired isolation source list.\n",
      "Downloading fasta of GCF_013459415.1 to positive genomes directory\n",
      "\n",
      "Currently downloaded 1 of 15 pos genome(s).\n",
      "Currently downloaded 3 of 25 neg genome(s).\n",
      "Checked a total of:\t 4 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_000828995.1_ASM82899v1_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_000828995.1\n",
      "Downloading gbk of GCF_000828995.1 to negative gbk directory\n",
      "Parsing gbk file of GCF_000828995.1 for organism and isolation source.\n",
      "[''] in desired isolation source list.\n",
      "Add row for GCF_000828995.1 in Mycobacterium tuberculosis str. Kurono chromosome, complete genome to summary table\n",
      "Downloading fasta of GCF_000828995.1 to negative genomes directory.\n",
      "\n",
      "Currently downloaded 1 of 15 pos genome(s).\n",
      "Currently downloaded 4 of 25 neg genome(s).\n",
      "Checked a total of:\t 5 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_013836745.1_ASM1383674v1_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_013836745.1\n",
      "Downloading gbk of GCF_013836745.1 to negative gbk directory\n",
      "Parsing gbk file of GCF_013836745.1 for organism and isolation source.\n",
      "[''] in desired isolation source list.\n",
      "Add row for GCF_013836745.1 in Staphylococcus aureus strain 128254 plasmid unnamed, complete sequence to summary table\n",
      "Downloading fasta of GCF_013836745.1 to negative genomes directory.\n",
      "\n",
      "Currently downloaded 1 of 15 pos genome(s).\n",
      "Currently downloaded 5 of 25 neg genome(s).\n",
      "Checked a total of:\t 6 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_003030245.1_ASM303024v1_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_003030245.1\n",
      "Downloading gbk of GCF_003030245.1 to negative gbk directory\n",
      "Parsing gbk file of GCF_003030245.1 for organism and isolation source.\n",
      "[''] in desired isolation source list.\n",
      "Add row for GCF_003030245.1 in Escherichia coli strain CFSAN018748 plasmid pGMI14-004_3, complete sequence to summary table\n",
      "Downloading fasta of GCF_003030245.1 to negative genomes directory.\n",
      "\n",
      "Currently downloaded 1 of 15 pos genome(s).\n",
      "Currently downloaded 6 of 25 neg genome(s).\n",
      "Checked a total of:\t 7 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_014788985.1_ASM1478898v1_feature_table.txt\n",
      "Found following hit/s for search term rieske 2fe-2s domain-containing protein: ['WP_043590225.1']\n",
      "Downloading gbk of GCF_014788985.1 to positive gbk directory.\n",
      "Parsing gbk file of GCF_014788985.1 for organism and isolation source.\n",
      "Add row for WP_043590225.1 in Chromobacterium haemolyticum strain Bb2 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_043590225.1 to TP file.\n",
      "[''] in desired isolation source list.\n",
      "Downloading fasta of GCF_014788985.1 to positive genomes directory\n",
      "\n",
      "Currently downloaded 2 of 15 pos genome(s).\n",
      "Currently downloaded 6 of 25 neg genome(s).\n",
      "Checked a total of:\t 8 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_017309545.1_ASM1730954v1_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_017309545.1\n",
      "Downloading gbk of GCF_017309545.1 to negative gbk directory\n",
      "Parsing gbk file of GCF_017309545.1 for organism and isolation source.\n",
      "[''] in desired isolation source list.\n",
      "Add row for GCF_017309545.1 in Lactococcus sp. LG1267 plasmid unnamed2, complete sequence to summary table\n",
      "Downloading fasta of GCF_017309545.1 to negative genomes directory.\n",
      "\n",
      "Currently downloaded 2 of 15 pos genome(s).\n",
      "Currently downloaded 7 of 25 neg genome(s).\n",
      "Checked a total of:\t 9 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_004377995.2_ASM437799v2_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_004377995.2\n",
      "Downloading gbk of GCF_004377995.2 to negative gbk directory\n",
      "Parsing gbk file of GCF_004377995.2 for organism and isolation source.\n",
      "['', 'not annotated', 'isolation_source not annotated'] in desired isolation source list.\n",
      "Add row for GCF_004377995.2 in Escherichia coli strain 1517k chromosome, complete genome to summary table\n",
      "Downloading fasta of GCF_004377995.2 to negative genomes directory.\n",
      "\n",
      "Currently downloaded 2 of 15 pos genome(s).\n",
      "Currently downloaded 8 of 25 neg genome(s).\n",
      "Checked a total of:\t 10 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_004795815.1_ASM479581v1_feature_table.txt\n",
      "Found following hit/s for search term rieske 2fe-2s domain-containing protein: ['WP_136188286.1', 'WP_017502794.1']\n",
      "Downloading gbk of GCF_004795815.1 to positive gbk directory.\n",
      "Parsing gbk file of GCF_004795815.1 for organism and isolation source.\n",
      "Add row for WP_136188286.1 in Sphingobium sp. PAMC28499 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_136188286.1 to TP file.\n",
      "Add row for WP_017502794.1 in Sphingobium sp. PAMC28499 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_017502794.1 to TP file.\n",
      "['', 'not annotated'] in desired isolation source list.\n",
      "Downloading fasta of GCF_004795815.1 to positive genomes directory\n",
      "\n",
      "Currently downloaded 3 of 15 pos genome(s).\n",
      "Currently downloaded 8 of 25 neg genome(s).\n",
      "Checked a total of:\t 11 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_016026735.1_ASM1602673v1_feature_table.txt\n",
      "Found following hit/s for search term rieske 2fe-2s domain-containing protein: ['WP_054306216.1', 'WP_054306041.1']\n",
      "Downloading gbk of GCF_016026735.1 to positive gbk directory.\n",
      "Parsing gbk file of GCF_016026735.1 for organism and isolation source.\n",
      "Add row for WP_054306216.1 in Serratia rubidaea strain FDAARGOS_926 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_054306216.1 to TP file.\n",
      "Add row for WP_054306041.1 in Serratia rubidaea strain FDAARGOS_926 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_054306041.1 to TP file.\n",
      "['', 'not annotated'] in desired isolation source list.\n",
      "Downloading fasta of GCF_016026735.1 to positive genomes directory\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Currently downloaded 4 of 15 pos genome(s).\n",
      "Currently downloaded 8 of 25 neg genome(s).\n",
      "Checked a total of:\t 12 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_001610935.1_ASM161093v1_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_001610935.1\n",
      "Downloading gbk of GCF_001610935.1 to negative gbk directory\n",
      "Parsing gbk file of GCF_001610935.1 for organism and isolation source.\n",
      "['', 'not annotated', 'isolation_source not annotated'] in desired isolation source list.\n",
      "Add row for GCF_001610935.1 in Streptococcus suis strain 90-1330 chromosome, complete genome to summary table\n",
      "Downloading fasta of GCF_001610935.1 to negative genomes directory.\n",
      "\n",
      "Currently downloaded 4 of 15 pos genome(s).\n",
      "Currently downloaded 9 of 25 neg genome(s).\n",
      "Checked a total of:\t 13 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_005576555.1_ASM557655v1_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_005576555.1\n",
      "Downloading gbk of GCF_005576555.1 to negative gbk directory\n",
      "Parsing gbk file of GCF_005576555.1 for organism and isolation source.\n",
      "[''] in desired isolation source list.\n",
      "Add row for GCF_005576555.1 in Collinsella aerofaciens strain P10wA7 chromosome, complete genome to summary table\n",
      "Downloading fasta of GCF_005576555.1 to negative genomes directory.\n",
      "\n",
      "Currently downloaded 4 of 15 pos genome(s).\n",
      "Currently downloaded 10 of 25 neg genome(s).\n",
      "Checked a total of:\t 14 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_000959245.1_ASM95924v1_feature_table.txt\n",
      "Found following hit/s for search term rieske 2fe-2s domain-containing protein: ['WP_010091823.1', 'WP_045564529.1', 'WP_045565555.1', 'WP_045565556.1', 'WP_045565591.1', 'WP_045565610.1']\n",
      "Downloading gbk of GCF_000959245.1 to positive gbk directory.\n",
      "Parsing gbk file of GCF_000959245.1 for organism and isolation source.\n",
      "Add row for WP_010091823.1 in Burkholderia ubonensis MSMB22 chromosome I, complete sequence to summary table\n",
      "Adding nucleotide sequence of WP_010091823.1 to TP file.\n",
      "Add row for WP_045564529.1 in Burkholderia ubonensis MSMB22 chromosome II, complete sequence to summary table\n",
      "Adding nucleotide sequence of WP_045564529.1 to TP file.\n",
      "Add row for WP_045565555.1 in Burkholderia ubonensis MSMB22 chromosome III, complete sequence to summary table\n",
      "Adding nucleotide sequence of WP_045565555.1 to TP file.\n",
      "Add row for WP_045565556.1 in Burkholderia ubonensis MSMB22 chromosome III, complete sequence to summary table\n",
      "Adding nucleotide sequence of WP_045565556.1 to TP file.\n",
      "Add row for WP_045565591.1 in Burkholderia ubonensis MSMB22 chromosome III, complete sequence to summary table\n",
      "Adding nucleotide sequence of WP_045565591.1 to TP file.\n",
      "Add row for WP_045565610.1 in Burkholderia ubonensis MSMB22 chromosome III, complete sequence to summary table\n",
      "Adding nucleotide sequence of WP_045565610.1 to TP file.\n",
      "['', 'not annotated'] in desired isolation source list.\n",
      "Downloading fasta of GCF_000959245.1 to positive genomes directory\n",
      "\n",
      "Currently downloaded 5 of 15 pos genome(s).\n",
      "Currently downloaded 10 of 25 neg genome(s).\n",
      "Checked a total of:\t 15 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_002213725.1_ASM221372v1_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_002213725.1\n",
      "Downloading gbk of GCF_002213725.1 to negative gbk directory\n",
      "Parsing gbk file of GCF_002213725.1 for organism and isolation source.\n",
      "[''] in desired isolation source list.\n",
      "Add row for GCF_002213725.1 in Listeria monocytogenes serotype 1/2a str. 99-6370 chromosome, complete genome to summary table\n",
      "Downloading fasta of GCF_002213725.1 to negative genomes directory.\n",
      "\n",
      "Currently downloaded 5 of 15 pos genome(s).\n",
      "Currently downloaded 11 of 25 neg genome(s).\n",
      "Checked a total of:\t 16 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_009363655.1_ASM936365v1_feature_table.txt\n",
      "Found following hit/s for search term rieske 2fe-2s domain-containing protein: ['WP_152491338.1', 'WP_152491520.1', 'WP_152491611.1', 'WP_152492280.1', 'WP_152492283.1', 'WP_152492589.1', 'WP_152493209.1']\n",
      "Downloading gbk of GCF_009363655.1 to positive gbk directory.\n",
      "Parsing gbk file of GCF_009363655.1 for organism and isolation source.\n",
      "Add row for WP_152491338.1 in Roseovarius sp. THAF27 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_152491338.1 to TP file.\n",
      "Add row for WP_152491520.1 in Roseovarius sp. THAF27 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_152491520.1 to TP file.\n",
      "Add row for WP_152491611.1 in Roseovarius sp. THAF27 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_152491611.1 to TP file.\n",
      "Add row for WP_152492280.1 in Roseovarius sp. THAF27 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_152492280.1 to TP file.\n",
      "Add row for WP_152492283.1 in Roseovarius sp. THAF27 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_152492283.1 to TP file.\n",
      "Add row for WP_152492589.1 in Roseovarius sp. THAF27 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_152492589.1 to TP file.\n",
      "Add row for WP_152493209.1 in Roseovarius sp. THAF27 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_152493209.1 to TP file.\n",
      "['', 'marine', 'water', 'coral'] in desired isolation source list.\n",
      "Downloading fasta of GCF_009363655.1 to positive genomes directory\n",
      "\n",
      "Currently downloaded 6 of 15 pos genome(s).\n",
      "Currently downloaded 11 of 25 neg genome(s).\n",
      "Checked a total of:\t 17 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_001702435.1_ASM170243v1_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_001702435.1\n",
      "Downloading gbk of GCF_001702435.1 to negative gbk directory\n",
      "Parsing gbk file of GCF_001702435.1 for organism and isolation source.\n",
      "[''] in desired isolation source list.\n",
      "Add row for GCF_001702435.1 in Mycobacterium tuberculosis strain SCAID 320.0 chromosome, complete genome to summary table\n",
      "Downloading fasta of GCF_001702435.1 to negative genomes directory.\n",
      "\n",
      "Currently downloaded 6 of 15 pos genome(s).\n",
      "Currently downloaded 12 of 25 neg genome(s).\n",
      "Checked a total of:\t 18 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_014490645.1_ASM1449064v1_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_014490645.1\n",
      "Downloading gbk of GCF_014490645.1 to negative gbk directory\n",
      "Parsing gbk file of GCF_014490645.1 for organism and isolation source.\n",
      "['', 'not annotated', 'isolation_source not annotated'] in desired isolation source list.\n",
      "Add row for GCF_014490645.1 in Pseudomonas aeruginosa strain PA3 chromosome, complete genome to summary table\n",
      "Downloading fasta of GCF_014490645.1 to negative genomes directory.\n",
      "\n",
      "Currently downloaded 6 of 15 pos genome(s).\n",
      "Currently downloaded 13 of 25 neg genome(s).\n",
      "Checked a total of:\t 19 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_016724885.1_ASM1672488v1_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_016724885.1\n",
      "Downloading gbk of GCF_016724885.1 to negative gbk directory\n",
      "Parsing gbk file of GCF_016724885.1 for organism and isolation source.\n",
      "['', 'not annotated', 'isolation_source not annotated'] in desired isolation source list.\n",
      "Add row for GCF_016724885.1 in Streptococcus dysgalactiae strain FDAARGOS_1157 plasmid unnamed2, complete sequence to summary table\n",
      "Downloading fasta of GCF_016724885.1 to negative genomes directory.\n",
      "\n",
      "Currently downloaded 6 of 15 pos genome(s).\n",
      "Currently downloaded 14 of 25 neg genome(s).\n",
      "Checked a total of:\t 20 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_000026145.1_ASM2614v1_feature_table.txt\n",
      "Found following hit/s for search term rieske 2fe-2s domain-containing protein: ['WP_011924944.1', 'WP_011926521.1', 'WP_041757723.1']\n",
      "Downloading gbk of GCF_000026145.1 to positive gbk directory.\n",
      "Parsing gbk file of GCF_000026145.1 for organism and isolation source.\n",
      "Add row for WP_011924944.1 in Bradyrhizobium sp. ORS 278, complete sequence to summary table\n",
      "Adding nucleotide sequence of WP_011924944.1 to TP file.\n",
      "Add row for WP_011926521.1 in Bradyrhizobium sp. ORS 278, complete sequence to summary table\n",
      "Adding nucleotide sequence of WP_011926521.1 to TP file.\n",
      "Add row for WP_041757723.1 in Bradyrhizobium sp. ORS 278, complete sequence to summary table\n",
      "Adding nucleotide sequence of WP_041757723.1 to TP file.\n",
      "['', 'not annotated'] in desired isolation source list.\n",
      "Downloading fasta of GCF_000026145.1 to positive genomes directory\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Currently downloaded 7 of 15 pos genome(s).\n",
      "Currently downloaded 14 of 25 neg genome(s).\n",
      "Checked a total of:\t 21 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_005938665.2_ASM593866v2_feature_table.txt\n",
      "Found following hit/s for search term rieske 2fe-2s domain-containing protein: ['WP_026286598.1', 'WP_138867373.1', 'WP_019648621.1', 'WP_138866314.1']\n",
      "Downloading gbk of GCF_005938665.2 to positive gbk directory.\n",
      "Parsing gbk file of GCF_005938665.2 for organism and isolation source.\n",
      "Add row for WP_026286598.1 in Pseudomonas sp. ATCC 43928 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_026286598.1 to TP file.\n",
      "Add row for WP_138867373.1 in Pseudomonas sp. ATCC 43928 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_138867373.1 to TP file.\n",
      "Add row for WP_019648621.1 in Pseudomonas sp. ATCC 43928 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_019648621.1 to TP file.\n",
      "Add row for WP_138866314.1 in Pseudomonas sp. ATCC 43928 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_138866314.1 to TP file.\n",
      "[''] in desired isolation source list.\n",
      "Downloading fasta of GCF_005938665.2 to positive genomes directory\n",
      "\n",
      "Currently downloaded 8 of 15 pos genome(s).\n",
      "Currently downloaded 14 of 25 neg genome(s).\n",
      "Checked a total of:\t 22 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_014304795.1_ASM1430479v1_feature_table.txt\n",
      "Found following hit/s for search term rieske 2fe-2s domain-containing protein: ['WP_130128985.1', 'WP_130129476.1']\n",
      "Downloading gbk of GCF_014304795.1 to positive gbk directory.\n",
      "Parsing gbk file of GCF_014304795.1 for organism and isolation source.\n",
      "Add row for WP_130128985.1 in Synechococcus sp. WH 8101 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_130128985.1 to TP file.\n",
      "Add row for WP_130129476.1 in Synechococcus sp. WH 8101 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_130129476.1 to TP file.\n",
      "['', 'marine'] in desired isolation source list.\n",
      "Downloading fasta of GCF_014304795.1 to positive genomes directory\n",
      "\n",
      "Currently downloaded 9 of 15 pos genome(s).\n",
      "Currently downloaded 14 of 25 neg genome(s).\n",
      "Checked a total of:\t 23 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_013267615.1_ASM1326761v1_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_013267615.1\n",
      "Downloading gbk of GCF_013267615.1 to negative gbk directory\n",
      "Parsing gbk file of GCF_013267615.1 for organism and isolation source.\n",
      "[''] in desired isolation source list.\n",
      "Add row for GCF_013267615.1 in Parabacteroides distasonis strain FDAARGOS_759 chromosome, complete genome to summary table\n",
      "Downloading fasta of GCF_013267615.1 to negative genomes directory.\n",
      "\n",
      "Currently downloaded 9 of 15 pos genome(s).\n",
      "Currently downloaded 15 of 25 neg genome(s).\n",
      "Checked a total of:\t 24 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_000195575.1_ASM19557v1_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_000195575.1\n",
      "Downloading gbk of GCF_000195575.1 to negative gbk directory\n",
      "Parsing gbk file of GCF_000195575.1 for organism and isolation source.\n",
      "['', 'sea', 'water'] in desired isolation source list.\n",
      "Add row for GCF_000195575.1 in Carnobacterium sp. 17-4 plasmid pCAR50, complete sequence to summary table\n",
      "Downloading fasta of GCF_000195575.1 to negative genomes directory.\n",
      "\n",
      "Currently downloaded 9 of 15 pos genome(s).\n",
      "Currently downloaded 16 of 25 neg genome(s).\n",
      "Checked a total of:\t 25 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_002982175.1_ASM298217v1_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_002982175.1\n",
      "Downloading gbk of GCF_002982175.1 to negative gbk directory\n",
      "Parsing gbk file of GCF_002982175.1 for organism and isolation source.\n",
      "[''] in desired isolation source list.\n",
      "Add row for GCF_002982175.1 in Bacillus subtilis strain ATCC 21228 plasmid pLDW-15, complete sequence to summary table\n",
      "Downloading fasta of GCF_002982175.1 to negative genomes directory.\n",
      "\n",
      "Currently downloaded 9 of 15 pos genome(s).\n",
      "Currently downloaded 17 of 25 neg genome(s).\n",
      "Checked a total of:\t 26 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_016944615.1_ASM1694461v1_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_016944615.1\n",
      "Downloading gbk of GCF_016944615.1 to negative gbk directory\n",
      "Parsing gbk file of GCF_016944615.1 for organism and isolation source.\n",
      "['', 'not annotated', 'isolation_source not annotated'] in desired isolation source list.\n",
      "Add row for GCF_016944615.1 in Pectobacterium brasiliense strain IPO:3649 NAK:375 chromosome, complete genome to summary table\n",
      "Downloading fasta of GCF_016944615.1 to negative genomes directory.\n",
      "\n",
      "Currently downloaded 9 of 15 pos genome(s).\n",
      "Currently downloaded 18 of 25 neg genome(s).\n",
      "Checked a total of:\t 27 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_001516325.2_ASM151632v2_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_001516325.2\n",
      "Downloading gbk of GCF_001516325.2 to negative gbk directory\n",
      "Parsing gbk file of GCF_001516325.2 for organism and isolation source.\n",
      "['', 'not annotated', 'isolation_source not annotated'] in desired isolation source list.\n",
      "Add row for GCF_001516325.2 in Pseudomonas aeruginosa strain H27930 chromosome, complete genome to summary table\n",
      "Downloading fasta of GCF_001516325.2 to negative genomes directory.\n",
      "\n",
      "Currently downloaded 9 of 15 pos genome(s).\n",
      "Currently downloaded 19 of 25 neg genome(s).\n",
      "Checked a total of:\t 28 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_003076535.1_ASM307653v1_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_003076535.1\n",
      "Downloading gbk of GCF_003076535.1 to negative gbk directory\n",
      "Parsing gbk file of GCF_003076535.1 for organism and isolation source.\n",
      "[''] in desired isolation source list.\n",
      "Add row for GCF_003076535.1 in Bacillus velezensis strain BS-37 chromosome, complete genome to summary table\n",
      "Downloading fasta of GCF_003076535.1 to negative genomes directory.\n",
      "\n",
      "Currently downloaded 9 of 15 pos genome(s).\n",
      "Currently downloaded 20 of 25 neg genome(s).\n",
      "Checked a total of:\t 29 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_005161825.1_ASM516182v1_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_005161825.1\n",
      "Downloading gbk of GCF_005161825.1 to negative gbk directory\n",
      "Parsing gbk file of GCF_005161825.1 for organism and isolation source.\n",
      "[''] in desired isolation source list.\n",
      "Add row for GCF_005161825.1 in Streptococcus pyogenes strain emm124 chromosome, complete genome to summary table\n",
      "Downloading fasta of GCF_005161825.1 to negative genomes directory.\n",
      "\n",
      "Currently downloaded 9 of 15 pos genome(s).\n",
      "Currently downloaded 21 of 25 neg genome(s).\n",
      "Checked a total of:\t 30 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_003354405.1_ASM335440v1_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_003354405.1\n",
      "Downloading gbk of GCF_003354405.1 to negative gbk directory\n",
      "Parsing gbk file of GCF_003354405.1 for organism and isolation source.\n",
      "[''] in desired isolation source list.\n",
      "Add row for GCF_003354405.1 in Mycoplasma bovis strain NADC61 chromosome, complete genome to summary table\n",
      "Downloading fasta of GCF_003354405.1 to negative genomes directory.\n",
      "\n",
      "Currently downloaded 9 of 15 pos genome(s).\n",
      "Currently downloaded 22 of 25 neg genome(s).\n",
      "Checked a total of:\t 31 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_900478265.1_33962_C02_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_900478265.1\n",
      "Downloading gbk of GCF_900478265.1 to negative gbk directory\n",
      "Parsing gbk file of GCF_900478265.1 for organism and isolation source.\n",
      "[''] in desired isolation source list.\n",
      "Add row for GCF_900478265.1 in Aggregatibacter aphrophilus strain NCTC11096 chromosome 1, complete sequence to summary table\n",
      "Downloading fasta of GCF_900478265.1 to negative genomes directory.\n",
      "\n",
      "Currently downloaded 9 of 15 pos genome(s).\n",
      "Currently downloaded 23 of 25 neg genome(s).\n",
      "Checked a total of:\t 32 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_003031425.1_ASM303142v1_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_003031425.1\n",
      "Downloading gbk of GCF_003031425.1 to negative gbk directory\n",
      "Parsing gbk file of GCF_003031425.1 for organism and isolation source.\n",
      "[''] in desired isolation source list.\n",
      "Add row for GCF_003031425.1 in Staphylococcus aureus strain CFSAN007896 chromosome, complete genome to summary table\n",
      "Downloading fasta of GCF_003031425.1 to negative genomes directory.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Currently downloaded 9 of 15 pos genome(s).\n",
      "Currently downloaded 24 of 25 neg genome(s).\n",
      "Checked a total of:\t 33 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_011404475.1_ASM1140447v1_feature_table.txt\n",
      "No hits for rieske 2fe-2s domain-containing protein in GCF_011404475.1\n",
      "Downloading gbk of GCF_011404475.1 to negative gbk directory\n",
      "Parsing gbk file of GCF_011404475.1 for organism and isolation source.\n",
      "[''] in desired isolation source list.\n",
      "Add row for GCF_011404475.1 in Escherichia coli strain 2018-06-4CC chromosome, complete genome to summary table\n",
      "Downloading fasta of GCF_011404475.1 to negative genomes directory.\n",
      "\n",
      "Currently downloaded 9 of 15 pos genome(s).\n",
      "Currently downloaded 25 of 25 neg genome(s).\n",
      "Checked a total of:\t 34 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_012935005.1_ASM1293500v1_feature_table.txt\n",
      "Found following hit/s for search term rieske 2fe-2s domain-containing protein: ['WP_001277084.1']\n",
      "Downloading gbk of GCF_012935005.1 to positive gbk directory.\n",
      "Parsing gbk file of GCF_012935005.1 for organism and isolation source.\n",
      "Add row for WP_001277084.1 in Acinetobacter baumannii strain VB2486 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_001277084.1 to TP file.\n",
      "[''] in desired isolation source list.\n",
      "Downloading fasta of GCF_012935005.1 to positive genomes directory\n",
      "\n",
      "Currently downloaded 10 of 15 pos genome(s).\n",
      "Currently downloaded 25 of 25 neg genome(s).\n",
      "Checked a total of:\t 35 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_001028705.1_ASM102870v1_feature_table.txt\n",
      "Found following hit/s for search term rieske 2fe-2s domain-containing protein: ['WP_053058992.1', 'WP_082134890.1', 'WP_082134965.1', 'WP_007015975.1', 'WP_007015972.1', 'WP_007015968.1']\n",
      "Downloading gbk of GCF_001028705.1 to positive gbk directory.\n",
      "Parsing gbk file of GCF_001028705.1 for organism and isolation source.\n",
      "Add row for WP_053058992.1 in Croceicoccus naphthovorans strain PQ-2 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_053058992.1 to TP file.\n",
      "Add row for WP_082134890.1 in Croceicoccus naphthovorans strain PQ-2 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_082134890.1 to TP file.\n",
      "Add row for WP_082134965.1 in Croceicoccus naphthovorans strain PQ-2 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_082134965.1 to TP file.\n",
      "Add row for WP_007015975.1 in Croceicoccus naphthovorans strain PQ-2 plasmid p1, complete sequence to summary table\n",
      "Adding nucleotide sequence of WP_007015975.1 to TP file.\n",
      "Add row for WP_007015972.1 in Croceicoccus naphthovorans strain PQ-2 plasmid p1, complete sequence to summary table\n",
      "Adding nucleotide sequence of WP_007015972.1 to TP file.\n",
      "Add row for WP_007015968.1 in Croceicoccus naphthovorans strain PQ-2 plasmid p1, complete sequence to summary table\n",
      "Adding nucleotide sequence of WP_007015968.1 to TP file.\n",
      "[''] in desired isolation source list.\n",
      "Downloading fasta of GCF_001028705.1 to positive genomes directory\n",
      "\n",
      "Currently downloaded 11 of 15 pos genome(s).\n",
      "Currently downloaded 25 of 25 neg genome(s).\n",
      "Checked a total of:\t 36 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_003112185.1_ASM311218v1_feature_table.txt\n",
      "GCF_003112185.1 is negative for rieske 2fe-2s domain-containing protein, but specified number of negative genomes reached.\n",
      "Removing feature table of GCF_003112185.1\n",
      "\n",
      "Currently downloaded 11 of 15 pos genome(s).\n",
      "Currently downloaded 25 of 25 neg genome(s).\n",
      "Checked a total of:\t 37 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_009497815.1_ASM949781v1_feature_table.txt\n",
      "GCF_009497815.1 is negative for rieske 2fe-2s domain-containing protein, but specified number of negative genomes reached.\n",
      "Removing feature table of GCF_009497815.1\n",
      "\n",
      "Currently downloaded 11 of 15 pos genome(s).\n",
      "Currently downloaded 25 of 25 neg genome(s).\n",
      "Checked a total of:\t 38 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_002163545.1_ASM216354v1_feature_table.txt\n",
      "GCF_002163545.1 is negative for rieske 2fe-2s domain-containing protein, but specified number of negative genomes reached.\n",
      "Removing feature table of GCF_002163545.1\n",
      "\n",
      "Currently downloaded 11 of 15 pos genome(s).\n",
      "Currently downloaded 25 of 25 neg genome(s).\n",
      "Checked a total of:\t 39 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_007980785.1_ASM798078v1_feature_table.txt\n",
      "GCF_007980785.1 is negative for rieske 2fe-2s domain-containing protein, but specified number of negative genomes reached.\n",
      "Removing feature table of GCF_007980785.1\n",
      "\n",
      "Currently downloaded 11 of 15 pos genome(s).\n",
      "Currently downloaded 25 of 25 neg genome(s).\n",
      "Checked a total of:\t 40 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_000193185.2_ASM19318v2_feature_table.txt\n",
      "GCF_000193185.2 is negative for rieske 2fe-2s domain-containing protein, but specified number of negative genomes reached.\n",
      "Removing feature table of GCF_000193185.2\n",
      "\n",
      "Currently downloaded 11 of 15 pos genome(s).\n",
      "Currently downloaded 25 of 25 neg genome(s).\n",
      "Checked a total of:\t 41 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_002116735.1_ASM211673v1_feature_table.txt\n",
      "Found following hit/s for search term rieske 2fe-2s domain-containing protein: ['WP_085680916.1']\n",
      "Downloading gbk of GCF_002116735.1 to positive gbk directory.\n",
      "Parsing gbk file of GCF_002116735.1 for organism and isolation source.\n",
      "Add row for WP_085680916.1 in Marinobacter salarius strain SMR5 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_085680916.1 to TP file.\n",
      "['', 'sediment'] in desired isolation source list.\n",
      "Downloading fasta of GCF_002116735.1 to positive genomes directory\n",
      "\n",
      "Currently downloaded 12 of 15 pos genome(s).\n",
      "Currently downloaded 25 of 25 neg genome(s).\n",
      "Checked a total of:\t 42 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_003254785.1_ASM325478v1_feature_table.txt\n",
      "GCF_003254785.1 is negative for rieske 2fe-2s domain-containing protein, but specified number of negative genomes reached.\n",
      "Removing feature table of GCF_003254785.1\n",
      "\n",
      "Currently downloaded 12 of 15 pos genome(s).\n",
      "Currently downloaded 25 of 25 neg genome(s).\n",
      "Checked a total of:\t 43 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_003050665.1_ASM305066v1_feature_table.txt\n",
      "GCF_003050665.1 is negative for rieske 2fe-2s domain-containing protein, but specified number of negative genomes reached.\n",
      "Removing feature table of GCF_003050665.1\n",
      "\n",
      "Currently downloaded 12 of 15 pos genome(s).\n",
      "Currently downloaded 25 of 25 neg genome(s).\n",
      "Checked a total of:\t 44 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_014054945.1_ASM1405494v1_feature_table.txt\n",
      "Found following hit/s for search term rieske 2fe-2s domain-containing protein: ['WP_003755471.1']\n",
      "Downloading gbk of GCF_014054945.1 to positive gbk directory.\n",
      "Parsing gbk file of GCF_014054945.1 for organism and isolation source.\n",
      "Add row for WP_003755471.1 in Neisseria sicca strain DSM 17713 chromosome, complete genome to summary table\n",
      "Adding nucleotide sequence of WP_003755471.1 to TP file.\n",
      "[''] in desired isolation source list.\n",
      "Downloading fasta of GCF_014054945.1 to positive genomes directory\n",
      "\n",
      "Currently downloaded 13 of 15 pos genome(s).\n",
      "Currently downloaded 25 of 25 neg genome(s).\n",
      "Checked a total of:\t 45 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_013729995.1_ASM1372999v1_feature_table.txt\n",
      "GCF_013729995.1 is negative for rieske 2fe-2s domain-containing protein, but specified number of negative genomes reached.\n",
      "Removing feature table of GCF_013729995.1\n",
      "\n",
      "Currently downloaded 13 of 15 pos genome(s).\n",
      "Currently downloaded 25 of 25 neg genome(s).\n",
      "Checked a total of:\t 46 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_002706425.1_ASM270642v1_feature_table.txt\n",
      "GCF_002706425.1 is negative for rieske 2fe-2s domain-containing protein, but specified number of negative genomes reached.\n",
      "Removing feature table of GCF_002706425.1\n",
      "\n",
      "Currently downloaded 13 of 15 pos genome(s).\n",
      "Currently downloaded 25 of 25 neg genome(s).\n",
      "Checked a total of:\t 47 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_000973585.1_ASM97358v1_feature_table.txt\n",
      "GCF_000973585.1 is negative for rieske 2fe-2s domain-containing protein, but specified number of negative genomes reached.\n",
      "Removing feature table of GCF_000973585.1\n",
      "\n",
      "Currently downloaded 13 of 15 pos genome(s).\n",
      "Currently downloaded 25 of 25 neg genome(s).\n",
      "Checked a total of:\t 48 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_003354765.1_ASM335476v1_feature_table.txt\n",
      "GCF_003354765.1 is negative for rieske 2fe-2s domain-containing protein, but specified number of negative genomes reached.\n",
      "Removing feature table of GCF_003354765.1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Currently downloaded 13 of 15 pos genome(s).\n",
      "Currently downloaded 25 of 25 neg genome(s).\n",
      "Checked a total of:\t 49 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_001896025.1_ASM189602v1_feature_table.txt\n",
      "GCF_001896025.1 is negative for rieske 2fe-2s domain-containing protein, but specified number of negative genomes reached.\n",
      "Removing feature table of GCF_001896025.1\n",
      "\n",
      "Currently downloaded 13 of 15 pos genome(s).\n",
      "Currently downloaded 25 of 25 neg genome(s).\n",
      "Checked a total of:\t 50 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_900638315.1_57195_D02_feature_table.txt\n",
      "GCF_900638315.1 is negative for rieske 2fe-2s domain-containing protein, but specified number of negative genomes reached.\n",
      "Removing feature table of GCF_900638315.1\n",
      "\n",
      "Currently downloaded 13 of 15 pos genome(s).\n",
      "Currently downloaded 25 of 25 neg genome(s).\n",
      "Checked a total of:\t 51 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_900087795.2_WHOV_feature_table.txt\n",
      "GCF_900087795.2 is negative for rieske 2fe-2s domain-containing protein, but specified number of negative genomes reached.\n",
      "Removing feature table of GCF_900087795.2\n",
      "\n",
      "Currently downloaded 13 of 15 pos genome(s).\n",
      "Currently downloaded 25 of 25 neg genome(s).\n",
      "Checked a total of:\t 52 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_008728375.1_ASM872837v1_feature_table.txt\n",
      "Found following hit/s for search term rieske 2fe-2s domain-containing protein: ['WP_004525711.1', 'WP_004193415.1', 'WP_004527094.1', 'WP_150977318.1', 'WP_004524918.1', 'WP_038765579.1', 'WP_004533114.1']\n",
      "Downloading gbk of GCF_008728375.1 to positive gbk directory.\n",
      "Parsing gbk file of GCF_008728375.1 for organism and isolation source.\n",
      "Add row for WP_004525711.1 in Burkholderia pseudomallei strain Yap1 chromosome 1, complete sequence to summary table\n",
      "Adding nucleotide sequence of WP_004525711.1 to TP file.\n",
      "Add row for WP_004193415.1 in Burkholderia pseudomallei strain Yap1 chromosome 1, complete sequence to summary table\n",
      "Adding nucleotide sequence of WP_004193415.1 to TP file.\n",
      "Add row for WP_004527094.1 in Burkholderia pseudomallei strain Yap1 chromosome 1, complete sequence to summary table\n",
      "Adding nucleotide sequence of WP_004527094.1 to TP file.\n",
      "Add row for WP_150977318.1 in Burkholderia pseudomallei strain Yap1 chromosome 2, complete sequence to summary table\n",
      "Adding nucleotide sequence of WP_150977318.1 to TP file.\n",
      "Add row for WP_004524918.1 in Burkholderia pseudomallei strain Yap1 chromosome 2, complete sequence to summary table\n",
      "Adding nucleotide sequence of WP_004524918.1 to TP file.\n",
      "Add row for WP_038765579.1 in Burkholderia pseudomallei strain Yap1 chromosome 2, complete sequence to summary table\n",
      "Adding nucleotide sequence of WP_038765579.1 to TP file.\n",
      "Add row for WP_004533114.1 in Burkholderia pseudomallei strain Yap1 chromosome 2, complete sequence to summary table\n",
      "Adding nucleotide sequence of WP_004533114.1 to TP file.\n",
      "[''] in desired isolation source list.\n",
      "Downloading fasta of GCF_008728375.1 to positive genomes directory\n",
      "\n",
      "Currently downloaded 14 of 15 pos genome(s).\n",
      "Currently downloaded 25 of 25 neg genome(s).\n",
      "Checked a total of:\t 53 genome(s). \n",
      "\n",
      "\n",
      "\n",
      "Filtering GCF_000297155.3_ASM29715v3_feature_table.txt\n",
      "Found following hit/s for search term rieske 2fe-2s domain-containing protein: ['WP_198965077.1', 'WP_078902306.1', 'WP_130585076.1']\n",
      "Downloading gbk of GCF_000297155.3 to positive gbk directory.\n",
      "Parsing gbk file of GCF_000297155.3 for organism and isolation source.\n",
      "Add row for WP_198965077.1 in Streptomyces tsukubensis NRRL18488 chromosome to summary table\n",
      "Adding nucleotide sequence of WP_198965077.1 to TP file.\n",
      "Add row for WP_078902306.1 in Streptomyces tsukubensis NRRL18488 chromosome to summary table\n",
      "Adding nucleotide sequence of WP_078902306.1 to TP file.\n",
      "Add row for WP_130585076.1 in Streptomyces tsukubensis NRRL18488 chromosome to summary table\n",
      "Adding nucleotide sequence of WP_130585076.1 to TP file.\n",
      "['', 'not annotated'] in desired isolation source list.\n",
      "Downloading fasta of GCF_000297155.3 to positive genomes directory\n",
      "\n",
      "Currently downloaded 15 of 15 pos genome(s).\n",
      "Currently downloaded 25 of 25 neg genome(s).\n",
      "\n",
      "\n",
      "\n",
      "Downloaded specified amount of genomes.\n",
      "Done\n"
     ]
    }
   ],
   "source": [
    "####TINKER CELL####\n",
    "# This cell is a modified version of the previous one with the difference that\n",
    "# the isolation source can be filtered for, here.\n",
    "\n",
    "\n",
    "#scramble accession_list\n",
    "random_gcf_list=random.sample(accession_list,len(accession_list))\n",
    "\n",
    "# pre-definiting headers and data types for faster loading of data frames\n",
    "feature_table_headers = [\"# feature\", \"class\", \"assembly\", \"assembly_unit\", \"seq_type\", \"chromosome\", \"genomic_accession\", \"start\", \"end\", \"strand\", \"product_accession\", \"non-redundant_refseq\", \"related_accession\", \"name\", \"symbol\", \"GeneID\", \"locus_tag\", \"feature_interval_length\", \"product_length\", \"attributes\"]\n",
    "data_types = {'# feature':str,'class':str,'assembly':str,'assembly_unit':str,'seq_type':str,'chromosome':str,'genomic_accession':str,'strand':str,'product_accession':str,'non-redundant_refseq':str,'related_accession':str,'name':str,'symbol':str,'locus_tag':str,'attributes':str}\n",
    "\n",
    "#initiate summary dictionary\n",
    "summary_dict={'dir':[],'assembly':[],'isolation_source':[],'organism':[],'description':[],'protein_id':[],'sequence':[]}\n",
    "\n",
    "#initialise counters to keep track of the amount of checked and downloaded genomes\n",
    "gcf_checked_counter=0\n",
    "neg_genomes_counter=0\n",
    "pos_genomes_counter=0\n",
    "\n",
    "\n",
    "#open a file to parse positive sequences into --> generate the TP file for this gene\n",
    "\n",
    "for i in range(0,len(random_gcf_list)): \n",
    "\n",
    "    os.chdir(feature_tables_path)\n",
    "    print('\\nCurrently downloaded',pos_genomes_counter, 'of', amount_pos_genomes, 'pos genome(s).')\n",
    "    print('Currently downloaded',neg_genomes_counter, 'of', amount_neg_genomes, 'neg genome(s).')\n",
    "\n",
    "\n",
    "    #exit from the loop once the pre-set amount of genomes has been downloaded\n",
    "    if neg_genomes_counter == amount_neg_genomes and pos_genomes_counter == amount_pos_genomes:\n",
    "        print('\\n\\n\\nDownloaded specified amount of genomes.')\n",
    "        break\n",
    "       \n",
    "    print('Checked a total of:\\t', gcf_checked_counter, 'genome(s). \\n\\n\\n')\n",
    "    gcf_checked_counter+=1\n",
    "    \n",
    "    #exit from the loop if all genomes have been checked (~22000, so that would take a while)\n",
    "    if gcf_checked_counter == len(random_gcf_list):\n",
    "        print('Checked all possible genomes. Exiting.')\n",
    "        break\n",
    "\n",
    "        \n",
    "    random_gcf=random_gcf_list[i]\n",
    "    !/home/blankma/.local/bin/ncbi-genome-download bacteria,archaea -s refseq -F features -l complete -A \"{random_gcf}\" --flat-output --output-folder \"{feature_tables_path}\"\n",
    "    !gunzip -f \"{feature_tables_path}\"/\"{random_gcf}\"*\n",
    "        \n",
    "    # Pre-filter individual feature table by a range of criteria to make it more compact\n",
    "    for feature_table in glob.glob(random_gcf+'*_feature_table.txt'):\n",
    "        print('Filtering', feature_table)\n",
    "        features_df = pd.read_csv(feature_table, index_col=None, sep = '\\t', names = feature_table_headers, dtype=data_types)\n",
    "        mask = (features_df['# feature'] == 'CDS')\n",
    "        export_df = features_df[mask]\n",
    "        mask2 = (export_df['class'] == 'with_protein') # removes pseudogenes 'without protein'\n",
    "        export_df2 = export_df[mask2]\n",
    "        mask3 = (export_df2['assembly_unit'] == 'Primary Assembly')\n",
    "        export_df3 = export_df2[mask3]\n",
    "        mask4 = (export_df3['seq_type'] == 'chromosome') | (export_df3['seq_type'] == 'plasmid')\n",
    "        export_df4 = export_df3[mask4]\n",
    "        mask5 = pd.isnull(export_df4['attributes'])\n",
    "        export_df5 = export_df4[mask5]\n",
    "        export_df5.loc[:,'name'] = export_df5.loc[:,'name'].str.lower()\n",
    "        mask6 = (~export_df5['name'].str.contains('hypothetical protein'))\n",
    "        export_df6 = export_df5[mask6]\n",
    "\n",
    "        \n",
    "        # search for search term in reduced features table and then conditionally download fasta and gbk either into\n",
    "        # pos or neg genomes directories\n",
    "        interest_mask = export_df6['name'].str.contains(search_term)\n",
    "        search_names = export_df6[interest_mask]    \n",
    "        \n",
    "\n",
    "        \n",
    "##### NEGATIVE GENOMES #####\n",
    "\n",
    "    # if search term is not found in genome and the set amount of negative genomes has not been reached\n",
    "    if len(search_names) == 0 and neg_genomes_counter < amount_neg_genomes:\n",
    "        \n",
    "        # Download gbk file\n",
    "        print('No hits for', search_term, 'in', random_gcf)\n",
    "        print('Downloading gbk of', random_gcf, 'to negative gbk directory')\n",
    "        !/home/blankma/.local/bin/ncbi-genome-download bacteria,archaea -s refseq -F genbank -l complete -A \"{random_gcf}\" --flat-output --output-folder \"{neg_gbk_path}\"\n",
    "        !gunzip -f \"{neg_gbk_path}\"/\"{random_gcf}\"*\n",
    " \n",
    "        #parse gbk files for isolation source and organism\n",
    "        os.chdir(neg_gbk_path)\n",
    "        for gbk_file in glob.glob(random_gcf+'*.gbff'):\n",
    "            print('Parsing gbk file of', random_gcf, 'for organism and isolation source.')\n",
    "            with open(gbk_file):\n",
    "                for record in SeqIO.parse(gbk_file,'gb'):\n",
    "                    features = record.features[0]\n",
    "                    \n",
    "                    #parse organism if annotated, handle if not\n",
    "                    if 'organism' in features.qualifiers:\n",
    "                        organism = features.qualifiers['organism']\n",
    "                    else:\n",
    "                        organism = ['organism not annotated']\n",
    "\n",
    "                    #parse isolation source if annotated, handle if not\n",
    "                    if 'isolation_source' in features.qualifiers:\n",
    "                        isolation_source = [entry.lower() for entry in record.features[0].qualifiers['isolation_source']]\n",
    "                    else:\n",
    "                        isolation_source = ['isolation_source not annotated']\n",
    "\n",
    "                # if predefined search words are found in isolation source, increment a counter\n",
    "                #download_indicator = 0\n",
    "                wordlist = []\n",
    "                for word in neg_isolation_source_filter:\n",
    "                    if word in isolation_source[0]:\n",
    "                        #download_indicator+=1\n",
    "                        wordlist.append(word)\n",
    "                print(wordlist, 'in desired isolation source list.')\n",
    "\n",
    "                        \n",
    "                # if isolation source condition is met...\n",
    "                if len(wordlist) >= 1:\n",
    "                    #...add row with required information to the summary table...\n",
    "                    print('Add row for', random_gcf, 'in', record.description, 'to summary table')\n",
    "                    summary_dict['dir'].append('-')\n",
    "                    summary_dict['assembly'].append(random_gcf)\n",
    "                    summary_dict['isolation_source'].append(isolation_source[0])\n",
    "                    summary_dict['organism'].append(organism[0])\n",
    "                    summary_dict['description'].append(record.description)\n",
    "                    summary_dict['protein_id'].append('None')\n",
    "                    summary_dict['sequence'].append('None')\n",
    "\n",
    "                    #...and download the fasta file for the corresponding genome and increment the counter.\n",
    "                    os.chdir(feature_tables_path)\n",
    "                    print('Downloading fasta of', random_gcf, 'to negative genomes directory.')\n",
    "                    !/home/blankma/.local/bin/ncbi-genome-download bacteria,archaea -s refseq -F fasta -l complete -A \"{random_gcf}\" --flat-output --output-folder \"{neg_genomes_path}\"\n",
    "                    !gunzip -f \"{neg_genomes_path}\"/\"{random_gcf}\"*\n",
    "                    #download_indicator = 0\n",
    "                    wordlist = []\n",
    "                    neg_genomes_counter+=1\n",
    "\n",
    "                # if isolation source condition is not met...\n",
    "                else:\n",
    "                    print(random_gcf, 'negative for', search_term, 'but', isolation_source[0], 'not in desired list.')\n",
    "                    print('Removing gbk file and feature table of', random_gcf)\n",
    "                    !rm \"{neg_gbk_path}\"/\"{random_gcf}\"*\n",
    "                    !rm \"{feature_tables_path}\"/\"{random_gcf}\"*\n",
    "                    wordlist = []\n",
    "                        \n",
    "\n",
    "    # if search term is not found in genome and the set amount of negative genomes has been reached\n",
    "    elif len(search_names) == 0 and neg_genomes_counter == amount_neg_genomes:\n",
    "        print(random_gcf, 'is negative for', search_term+',', 'but specified number of negative genomes reached.')\n",
    "        print('Removing feature table of', random_gcf)\n",
    "        !rm \"{feature_tables_path}\"/\"{random_gcf}\"*\n",
    "    \n",
    "    \n",
    "##### POSITIVE GENOMES #####\n",
    "    # if search term is found in genome and the set amount of positive genomes has not been reached\n",
    "    elif len(search_names) != 0 and pos_genomes_counter < amount_pos_genomes:\n",
    "\n",
    "        # Download gbk file\n",
    "        product_accessions = list(search_names.loc[:,'product_accession'])\n",
    "        print('Found following hit/s for search term', search_term+':', product_accessions)\n",
    "        print('Downloading gbk of', random_gcf, 'to positive gbk directory.')\n",
    "        !/home/blankma/.local/bin/ncbi-genome-download bacteria,archaea -s refseq -F genbank -l complete -A \"{random_gcf}\" --flat-output --output-folder \"{pos_gbk_path}\"\n",
    "        !gunzip -f \"{pos_gbk_path}\"/\"{random_gcf}\"*\n",
    "        \n",
    "        \n",
    "        # Adding an entry for each found hit in each genome into summary_dict\n",
    "        os.chdir(pos_gbk_path)\n",
    "        for gbk_file in glob.glob(random_gcf+'*.gbff'):\n",
    "            print('Parsing gbk file of', random_gcf, 'for organism and isolation source.')\n",
    "            with open(gbk_file):\n",
    "                for record in SeqIO.parse(gbk_file,'gb'):    \n",
    "                    features = record.features[0]\n",
    "                    \n",
    "                    #parse organism if annotated, handle if not\n",
    "                    if 'organism' in record.features[0].qualifiers:\n",
    "                        organism = record.features[0].qualifiers['organism']\n",
    "                    else:\n",
    "                        organism = ['organism not annotated']\n",
    "\n",
    "                    #parse isolation source if annotated, handle if not\n",
    "                    if 'isolation_source' in record.features[0].qualifiers:\n",
    "                        isolation_source = [entry.lower() for entry in record.features[0].qualifiers['isolation_source']]\n",
    "                        record.features[0].qualifiers['isolation_source']\n",
    "                    else:\n",
    "                        isolation_source = ['isolation_source not annotated']\n",
    "        \n",
    "        \n",
    "                    # if predefined search words are found in isolation source, increment a counter\n",
    "                    #download_indicator = 0\n",
    "                    wordlist = []\n",
    "                    for word in pos_isolation_source_filter:\n",
    "                        if word in isolation_source[0]:\n",
    "                            #download_indicator+=1\n",
    "                            wordlist.append(word)\n",
    "                    \n",
    "\n",
    "                    # if isolation source condition is met...\n",
    "                    if len(wordlist) >= 1:\n",
    "                        #...add row with required information to the summary table...  \n",
    "                        for i in record.features:\n",
    "                            for j in product_accessions:\n",
    "                                if i.qualifiers.get('protein_id') == [j]:\n",
    "                                    #print(i.extract(record.seq))\n",
    "                                    print('Add row for', j, 'in', record.description, 'to summary table')\n",
    "                                    summary_dict['dir'].append('+')\n",
    "                                    summary_dict['assembly'].append(random_gcf)\n",
    "                                    summary_dict['isolation_source'].append(isolation_source[0])\n",
    "                                    summary_dict['organism'].append(organism[0])\n",
    "                                    summary_dict['description'].append(record.description)\n",
    "                                    summary_dict['protein_id'].append(j)\n",
    "                                    summary_dict['sequence'].append(i.qualifiers.get('translation'))\n",
    "                                    print('Adding nucleotide sequence of', j, 'to TP file.')\n",
    "                                    with open(BGC_path+'/'+BGC_type+'_tp_genes.fasta', 'a') as fasta_file:\n",
    "                                        fasta_file.write('>'+random_gcf+'_'+organism[0]+'_'+j+'\\n'+str(i.extract(record.seq))+'\\n')\n",
    "\n",
    "\n",
    "\n",
    "                                \n",
    "        #...and download the fasta file for the corresponding genome and increment the counter.\n",
    "        if len(wordlist) >= 1:\n",
    "            os.chdir(feature_tables_path)\n",
    "            print(wordlist, 'in desired isolation source list.')\n",
    "            print('Downloading fasta of', random_gcf, 'to positive genomes directory')\n",
    "            !/home/blankma/.local/bin/ncbi-genome-download bacteria,archaea -s refseq -F fasta -l complete -A \"{random_gcf}\" --flat-output --output-folder \"{pos_genomes_path}\"\n",
    "            !gunzip -f \"{pos_genomes_path}\"/\"{random_gcf}\"*\n",
    "            pos_genomes_counter+=1\n",
    "            wordlist = []\n",
    "        else:\n",
    "            print(random_gcf, 'positive for', search_term, 'but', isolation_source[0], 'not in desired list.')\n",
    "            print('Removing gbk file and feature table of', random_gcf)\n",
    "            !rm \"{pos_gbk_path}\"/\"{random_gcf}\"*\n",
    "            !rm \"{feature_tables_path}\"/\"{random_gcf}\"*\n",
    "            wordlist = []\n",
    "\n",
    "\n",
    "    # if search term is found in genome and the set amount of positive genomes has been reached\n",
    "    elif len(search_names) != 0 and pos_genomes_counter == amount_pos_genomes:\n",
    "        print(random_gcf, 'is positive for', search_term+',', 'but specified number of positive genomes reached.')\n",
    "        print('Removing feature table of', random_gcf)\n",
    "        !rm \"{feature_tables_path}\"/\"{random_gcf}\"*\n",
    "\n",
    "        \n",
    "        \n",
    "        \n",
    "##### FAIL? #####\n",
    "        \n",
    "    elif neg_genomes_counter == amount_neg_genomes and pos_genomes_counter == amount_pos_genomes:\n",
    "        print(\"This shouldn't be happening\", '\\n')   \n",
    "        \n",
    "\n",
    "print('Done')\n",
    "\n",
    "              \n",
    "              \n",
    "              \n",
    "summary_df = pd.DataFrame(summary_dict)\n",
    "summary_df.to_csv(BGC_path+'/summary.tsv', index=False, sep='\\t')"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3.8.2 (gimkl-2020a)",
   "language": "python",
   "name": "python382-gimkl-2020a"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
