{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "In addition to the original script, 2_generate_TP, this script does a blast search of selected_tp_genes against temp_neg_genomes and filters out all genomes that have a hit. In addition to 2.1_generate_TP, this script also does an hmm search, using an alignment of selected_tp_genes as query against negative genomes. It combines results of blast and hmm searches and removes all samples with hits from the pool of neg_genomes to choose from. One further parameter is required to specify an hmm evalue cutoff for samples to remove. Ideally, this would be 0, but I decided to keep a cutoff, as the original negative genomes from the sugimoto paper also have positive hits for an oxyN hmm.\n",
    "\n",
    "\n",
    "This script generates the 2 remaining files required for input:\n",
    "- True positive genes (nucleotide fasta)\n",
    "- protein alignment (amino acid fasta)\n",
    "IMPORTANTLY, these 2 files are created from non-overlapping sequences!\n",
    "\n",
    "\n",
    "And also moves the corresponding files to new directories so that coverage tables can be generated from them.\n",
    "\n",
    "In the first cell specify:\n",
    "- BGC type (This name must stay constant throughout the scripts)\n",
    "- select_neg_genomes, i.e. the amount of negative genomes to be transferred to the neg_genomes directory\n",
    "- select_pos_genomes, i.e. the amount of positive genomes to be transferred to the pos_genomes directory and to generate the tp_genes file from (the surplus amount will be used to generate the protein alignment from)\n",
    "- pos_isolation_source_filter, if these terms are found in the isolation_source column of the positive samples in the summary file, they will be scored higher in a scoring column, i.e. samples from a known and desired isolation source will be used preferentially.\n",
    "- neg_isolation_source_filter, accordingly\n",
    "- avoid_list. These terms are scored with a 0, end at the bottom of the table, and will be picked last. This is useful when an uncommon gene is searched for and more, and/or more tenuous isolation sources have been allowed during download. These are generally words that contain one of the search terms, e.g. 'sea' in 'diseased'."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Modify in such a way that TP genes are used as query against all individual negative genomes. Negative genomes are only moved from temp directory to neg_genomes directory if the blast search comes back negative"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "BGC_type = 'nitrile_hydratase_alpha'\n",
    "select_neg_genomes = 140\n",
    "select_pos_genomes = 10\n",
    "hmm_evalue_cutoff = 3\n",
    "\n",
    "avoid_list = ['', 'isolation_source not annotated', 'diseased', 'mice', 'spice', 'septicemic', 'research', 'crevice']\n",
    "#these are identical to first script, but don't have to be\n",
    "pos_isolation_source_filter =  ['marine', 'sea', 'sponge', 'ocean', 'porifera', 'seafloor','sediment', 'water', 'tidal', 'coral', 'reef', 'coast', 'ship', 'fish', 'aquaculture', 'atlantic', 'pacific', 'mediterranean', 'baltic', 'pond', 'river', 'ice', 'carribean', 'lake', 'fjord', 'marina', 'hydro', 'algal', 'algae']\n",
    "neg_isolation_source_filter = ['marine', 'sea', 'sponge', 'ocean', 'porifera', 'seafloor', 'sediment', 'water', 'tidal', 'coral', 'reef', 'coast', 'ship', 'fish', 'aquaculture', 'atlantic', 'pacific', 'mediterranean', 'baltic', 'pond', 'river', 'ice', 'carribean', 'lake', 'fjord', 'marina', 'hydro', 'algal', 'algae']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "from os import listdir, mkdir\n",
    "from os.path import isfile, join\n",
    "from pathlib import Path\n",
    "import pandas as pd\n",
    "from pandas.errors import EmptyDataError\n",
    "import random\n",
    "import glob\n",
    "import warnings\n",
    "from Bio import SearchIO"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Making /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "Making /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/pos_genomes\n",
      "Making /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast\n",
      "Making /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases\n",
      "Making /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/results\n",
      "Making /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm\n",
      "Making /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases\n",
      "Making /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results\n"
     ]
    }
   ],
   "source": [
    "def makedir(dirpath):\n",
    "    if os.path.isdir(dirpath):\n",
    "        print(dirpath,'exists already')\n",
    "    else:\n",
    "        print('Making', dirpath)    \n",
    "        os.mkdir(dirpath)\n",
    "\n",
    "        \n",
    "# Defining paths for required directory structure for input and output files relative to parent directory\n",
    "#parent_dir='/media/manu/RiPP_Prioritiser/'\n",
    "#will make directories relative to the path the notebook was opened in\n",
    "parent_dir= !echo $(pwd)\n",
    "BGC_path=os.path.join(parent_dir[0], BGC_type)\n",
    "neg_genomes_path=os.path.join(BGC_path, 'base_genomes/temp_neg_genomes')\n",
    "pos_genomes_path=os.path.join(BGC_path, 'base_genomes/temp_pos_genomes')\n",
    "output_neg_path=os.path.join(BGC_path, 'base_genomes/neg_genomes')\n",
    "output_pos_path=os.path.join(BGC_path, 'base_genomes/pos_genomes')\n",
    "#qc-paths. Put all in one directory eventually\n",
    "neg_blast_path=os.path.join(BGC_path, 'base_genomes/neg_blast')\n",
    "neg_blast_db_path=os.path.join(BGC_path, 'base_genomes/neg_blast/databases')\n",
    "neg_blast_results_path=os.path.join(BGC_path, 'base_genomes/neg_blast/results')\n",
    "neg_hmm_path=os.path.join(BGC_path, 'base_genomes/neg_hmm')\n",
    "neg_hmm_db_path=os.path.join(BGC_path, 'base_genomes/neg_hmm/databases')\n",
    "neg_hmm_results_path=os.path.join(BGC_path, 'base_genomes/neg_hmm/results')\n",
    "\n",
    "\n",
    "# Calling function to make directories if they don't exist yet\n",
    "makedir(output_neg_path)\n",
    "makedir(output_pos_path)\n",
    "makedir(neg_blast_path)\n",
    "makedir(neg_blast_db_path)\n",
    "makedir(neg_blast_results_path)\n",
    "makedir(neg_hmm_path)\n",
    "makedir(neg_hmm_db_path)\n",
    "makedir(neg_hmm_results_path)\n",
    "\n",
    "os.chdir(BGC_path)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Generating a report file for this script\n",
    "with open(BGC_path+'/'+'report_2_generate_tp.txt', 'w') as f:\n",
    "    f.write('Output directory is: '+BGC_path+'\\n')\n",
    "    f.write('\\nBGC_type = '+BGC_type)\n",
    "    f.write('\\nselect_neg_genomes = '+str(select_neg_genomes))\n",
    "    f.write('\\nselect_pos_genomes = '+str(select_pos_genomes))\n",
    "    f.write('\\nhmm_evalue_cutoff = '+str(hmm_evalue_cutoff))\n",
    "    f.write('\\navoid_list = '+str(avoid_list))\n",
    "    f.write('\\nneg_isolation_source_filter = '+str(neg_isolation_source_filter))\n",
    "    f.write('\\npos_isolation_source_filter = '+str(pos_isolation_source_filter)+'\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "# load summary table into data frame () output from 1.)\n",
    "summary_file = pd.read_csv('summary.tsv', sep='\\t')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Change order of tables to prioritize samples that have an isolation source"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "warnings.filterwarnings('ignore')\n",
    "\n",
    "#filter positives and drop all duplicate protein sequences originating from different organisms\n",
    "pos_mask = (summary_file['dir'] == '+')\n",
    "pos_df = summary_file[pos_mask]\n",
    "pos_df.drop_duplicates(subset='protein_id', keep=False, inplace=True)\n",
    "\n",
    "\n",
    "#filter negatives\n",
    "neg_mask = (summary_file['dir'] == '-')\n",
    "neg_df = summary_file[neg_mask]\n",
    "\n",
    "#scoring words in isolation source so as to preferentially pick samples with chosen isolation sources\n",
    "\n",
    "def custom_sorting(source,isolation_source_filter):\n",
    "    score = 1\n",
    "    if isolation_source_filter=='pos':\n",
    "        for word in pos_isolation_source_filter:\n",
    "            if word in source:\n",
    "                score +=1\n",
    "        for word in avoid_list:\n",
    "            if source == word:\n",
    "                score=0\n",
    "    elif isolation_source_filter=='neg':\n",
    "        for word in neg_isolation_source_filter:\n",
    "            if word in source:\n",
    "                score +=1\n",
    "        for word in avoid_list:\n",
    "            if source == word:\n",
    "                score=0\n",
    "    return score\n",
    "\n",
    "\n",
    "pos_df['scoring_column'] = pos_df.apply(lambda x: custom_sorting(x['isolation_source'],'pos'),axis=1)\n",
    "neg_df['scoring_column'] = neg_df.apply(lambda x: custom_sorting(x['isolation_source'],'neg'),axis=1)\n",
    "\n",
    "pos_df.sort_values(by=['scoring_column'], axis=0, ascending=False, inplace=True)\n",
    "neg_df.sort_values(by=['scoring_column'], axis=0, ascending=False, inplace=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "moving positive GCF_002983865.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/pos_genomes\n",
      "moving positive GCF_009363795.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/pos_genomes\n",
      "moving positive GCF_009363595.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/pos_genomes\n",
      "moving positive GCF_011290545.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/pos_genomes\n",
      "moving positive GCF_011040495.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/pos_genomes\n",
      "moving positive GCF_000332275.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/pos_genomes\n",
      "moving positive GCF_000511355.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/pos_genomes\n",
      "moving positive GCF_002215215.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/pos_genomes\n",
      "moving positive GCF_004006175.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/pos_genomes\n",
      "moving positive GCF_001969365.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/pos_genomes\n",
      "generating selected_tp_genes.fasta\n",
      "generating selected_tp_aa.fasta\n",
      "Done\n"
     ]
    }
   ],
   "source": [
    "#Split positive genomes into 2 bins, one goes towards tp-genes and is the pos-genomes used for synthesising metagenomes\n",
    "#the other one constitutes a source of protein sequences for alignment as an input file\n",
    "\n",
    "# Genomes selected in such a way that they are from the top of the pre-sorted pos_df\n",
    "unique_pos_df = pos_df.drop_duplicates(subset='assembly', inplace=False)\n",
    "selected_tp_genomes = list(unique_pos_df.iloc[:,1])[0:select_pos_genomes]\n",
    "remaining_pos_genomes = list(unique_pos_df.iloc[:,1])[select_pos_genomes:]\n",
    "\n",
    "with open(BGC_path+'/'+'report_2_generate_tp.txt', 'a') as f:\n",
    "    f.write('\\nselected_tp_genomes are:\\n')\n",
    "    f.write(str(selected_tp_genomes)+'\\n')\n",
    "\n",
    "#select genomes and isolate GCF number from them, move selected tp genomes to final pos_genomes directory\n",
    "for genome in selected_tp_genomes:\n",
    "    print('moving positive', genome, 'to', output_pos_path)\n",
    "    !mv \"{pos_genomes_path}\"/\"{genome}\"* \"{output_pos_path}\"\n",
    "    \n",
    "#generate dataframe containing all tp-genomes and all the tp-genes contained in it\n",
    "filtered_pos_df = pos_df[pos_df['assembly'].isin(selected_tp_genomes)]\n",
    "remaining_pos_df = pos_df[~pos_df['assembly'].isin(selected_tp_genomes)]\n",
    "\n",
    "#isolate all the headers and transfer them to the selected_tp_genes file\n",
    "full_header_list = []\n",
    "for i in range(0,len(filtered_pos_df)):\n",
    "    full_header=str('>')+filtered_pos_df.iloc[i,1]+str('_')+filtered_pos_df.iloc[i,3]+str('_')+filtered_pos_df.iloc[i,5]\n",
    "    full_header_list.append(full_header)\n",
    "\n",
    "# generate fasta file with selected tp genes found in the selected genomes\n",
    "print('generating selected_tp_genes.fasta')\n",
    "with open(BGC_path+'/'+'report_2_generate_tp.txt', 'a') as f:\n",
    "    f.write('\\nselected_tp_genes in positive genomes are:\\n')\n",
    "tp_gene_counter=0\n",
    "with open(BGC_path+'/'+BGC_type+'_tp_genes.fasta') as fh:\n",
    "    lines=fh.readlines()\n",
    "    for i in range(0,len(lines)):\n",
    "        for j in range(0,len(full_header_list)):\n",
    "            if full_header_list[j] in lines[i]:\n",
    "                tp_gene_counter+=1\n",
    "                with open(BGC_path+'/'+'report_2_generate_tp.txt', 'a') as f:\n",
    "                    f.write(lines[i][1:-1]+'\\n')\n",
    "                with open(BGC_path+'/'+BGC_type+'_selected_tp_genes.fasta', 'a') as outfile:\n",
    "                    outfile.write(lines[i]+lines[i+1])    \n",
    "with open(BGC_path+'/'+'report_2_generate_tp.txt', 'a') as f:\n",
    "    f.write('\\n'+str(len(selected_tp_genomes))+' unique genomes with '+ str(tp_gene_counter)+' unique tp genes.\\n\\n')\n",
    "                    \n",
    "                    \n",
    "# transfer all amino acid sequences that are not part of the tp-genomes to a fasta file\n",
    "print('generating selected_tp_aa.fasta')\n",
    "with open(BGC_path+'/'+'report_2_generate_tp.txt', 'a') as f:\n",
    "    f.write('\\nselected_tp_aa sequences for muscle alignment are:\\n')\n",
    "tp_aa_counter = 0\n",
    "with open(BGC_path+'/'+BGC_type+'_selected_tp_aa.fasta', 'a') as outfile:\n",
    "    for i in range(0,len(remaining_pos_df)):\n",
    "        tp_aa_counter+=1\n",
    "        fasta_header=str('>')+remaining_pos_df.iloc[i,1]+str('_')+remaining_pos_df.iloc[i,3]+str('_')+remaining_pos_df.iloc[i,5]+'\\n'\n",
    "        sequence = remaining_pos_df.iloc[i,6][2:-2]+'\\n'\n",
    "        with open(BGC_path+'/'+'report_2_generate_tp.txt', 'a') as f:\n",
    "            f.write(fasta_header[1:-1]+'\\n')\n",
    "        outfile.write(fasta_header)\n",
    "        outfile.write(sequence)\n",
    "with open(BGC_path+'/'+'report_2_generate_tp.txt', 'a') as f:\n",
    "    f.write('\\n'+str(len(remaining_pos_genomes))+' unique genomes with '+ str(tp_aa_counter)+' unique aa sequences.\\n\\n')\n",
    "\n",
    "    \n",
    "print('Done')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:29\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_000520615.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000520615.1_SOAPdenovo_v1.05_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 64 sequences in 0.0507581 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:29\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_000355675.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000355675.1_ASM35567v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 1 sequences in 0.0281951 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:29\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_013277795.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_013277795.1_ASM1327779v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 36 sequences in 0.0666471 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:29\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_000353285.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000353285.1_ASM35328v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 5 sequences in 0.08165 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:29\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001624595.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001624595.1_ASM162459v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 68 sequences in 0.0442469 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:30\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_013249065.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_013249065.1_ASM1324906v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 63 sequences in 0.038089 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:30\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001529305.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001529305.1_ASM152930v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 82 sequences in 0.044548 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:30\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003050585.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003050585.1_ASM305058v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 30 sequences in 0.0429261 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:30\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001044395.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001044395.1_ASM104439v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 74 sequences in 0.0466158 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:30\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_018199995.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_018199995.1_ASM1819999v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 3 sequences in 0.0457771 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:31\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001854765.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001854765.1_ASM185476v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 138 sequences in 0.0469861 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:31\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003171655.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003171655.1_ASM317165v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 18 sequences in 0.0322499 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:31\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_015864865.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_015864865.1_ASM1586486v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 10 sequences in 0.0241611 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:31\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_009846755.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_009846755.1_ASM984675v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 78 sequences in 0.038543 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:31\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_005938125.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_005938125.1_ASM593812v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 68 sequences in 0.035331 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:32\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_017168275.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_017168275.1_ASM1716827v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 80 sequences in 0.058203 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:32\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003086815.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003086815.1_ASM308681v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 1 sequences in 0.0356598 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:32\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_016837255.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_016837255.1_ASM1683725v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 115 sequences in 0.0305519 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:32\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003015025.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003015025.1_ASM301502v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Adding sequences from FASTA; added 33 sequences in 0.045681 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:33\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_004378355.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004378355.1_ASM437835v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 116 sequences in 0.0368278 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:33\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003050175.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003050175.1_ASM305017v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 53 sequences in 0.044523 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:33\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_002873185.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002873185.1_ASM287318v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 224 sequences in 0.0527561 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:33\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_011806565.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_011806565.1_ASM1180656v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 24 sequences in 0.026427 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:33\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_005771535.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_005771535.1_ASM577153v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 18 sequences in 0.0290151 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:34\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_016461915.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_016461915.1_ASM1646191v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 139 sequences in 0.044831 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:34\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_004803475.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004803475.1_ASM480347v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 15 sequences in 0.0417762 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:34\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_000371805.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000371805.1_ASM37180v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 2 sequences in 0.0111809 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:34\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001730205.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001730205.1_ASM173020v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 65 sequences in 0.057107 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:34\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_014858715.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_014858715.1_ASM1485871v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 10 sequences in 0.0423639 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:35\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_002021755.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002021755.1_ASM202175v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 2 sequences in 0.0459909 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:35\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_000737535.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000737535.1_ASM73753v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 1 sequences in 0.0245171 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:35\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_014076905.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_014076905.1_ASM1407690v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 219 sequences in 0.044858 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:35\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_006246515.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006246515.1_ASM624651v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 26 sequences in 0.0389571 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:35\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_900302475.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_900302475.1_D.aquaemixtaeCECT8626_Spades_Prokka_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 18 sequences in 0.036644 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:36\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_018531475.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_018531475.1_ASM1853147v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 245 sequences in 0.069932 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:36\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001746765.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001746765.1_ASM174676v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 273 sequences in 0.073257 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:36\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_017948225.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_017948225.1_ASM1794822v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Adding sequences from FASTA; added 1 sequences in 0.0276239 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:36\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001015055.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001015055.1_ASM101505v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 136 sequences in 0.0537672 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:36\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_004360145.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004360145.1_ASM436014v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 33 sequences in 0.032711 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:37\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_009362315.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_009362315.1_ASM936231v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 217 sequences in 0.046541 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:37\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_018500205.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_018500205.1_ASM1850020v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 104 sequences in 0.0521059 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:37\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_000474745.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000474745.1_Acht1_1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 99 sequences in 0.0271201 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:37\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_900185035.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_900185035.1_R.arenilitoris_CECT8715_Spades_Prokka_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 11 sequences in 0.0345809 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:37\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_012911815.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_012911815.1_ASM1291181v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 22 sequences in 0.0295401 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:38\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_009857605.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_009857605.1_ASM985760v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 104 sequences in 0.0576298 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:38\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_006243325.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006243325.1_ASM624332v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 225 sequences in 0.0511818 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:38\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_009377985.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_009377985.1_ASM937798v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 28 sequences in 0.026818 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:38\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003970385.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003970385.1_ASM397038v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 215 sequences in 0.047291 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:38\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_005281475.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_005281475.1_ASM528147v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 1 sequences in 0.0434051 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:39\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_006381105.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006381105.1_ASM638110v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 77 sequences in 0.0490081 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:39\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001999205.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001999205.1_ASM199920v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 2 sequences in 0.0346048 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:39\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_000294775.2_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000294775.2_ASM29477v2_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 2 sequences in 0.0499399 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:39\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001951155.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001951155.1_ASM195115v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 1 sequences in 0.0279191 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:39\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_009789075.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_009789075.1_ASM978907v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 22 sequences in 0.0319769 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:40\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_002573675.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002573675.1_ASM257367v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 10 sequences in 0.0525661 seconds.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:40\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_013363245.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_013363245.1_ASM1336324v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 30 sequences in 0.0219841 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:40\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_002893095.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002893095.1_ASM289309v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 36 sequences in 0.0380611 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:40\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_013032145.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_013032145.1_ASM1303214v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 33 sequences in 0.0362942 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:40\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_014164675.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_014164675.1_ASM1416467v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 122 sequences in 0.0355079 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:41\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_006375565.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006375565.1_ASM637556v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 83 sequences in 0.0436311 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:41\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_002882915.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002882915.1_GW456-12-1-14-LB3_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 143 sequences in 0.0619419 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:41\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001888265.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001888265.1_ASM188826v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 92 sequences in 0.0324271 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:41\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_006368485.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006368485.1_ASM636848v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 788 sequences in 0.0688131 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:42\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_004214755.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004214755.1_ASM421475v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 99 sequences in 0.0485928 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:42\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001296255.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001296255.1_ASM129625v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 37 sequences in 0.048183 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:42\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001655635.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001655635.1_ASM165563v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 3 sequences in 0.0475571 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:42\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_902500085.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_902500085.1_BB1436_assembly_illumina_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 152 sequences in 0.0481658 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:42\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001720485.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001720485.1_ASM172048v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 1 sequences in 0.0300591 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:43\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003348465.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003348465.1_ASM334846v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 69 sequences in 0.036901 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:43\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_006374735.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006374735.1_ASM637473v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 533 sequences in 0.063597 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:43\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_007859875.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_007859875.1_ASM785987v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 6 sequences in 0.0813391 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:43\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_018122435.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_018122435.1_ASM1812243v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 61 sequences in 0.0370209 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:43\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003198945.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003198945.1_ASM319894v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 43 sequences in 0.0402272 seconds.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:44\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_004022565.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004022565.1_ASM402256v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 1 sequences in 0.0255692 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:44\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_002127625.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002127625.1_ASM212762v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 106 sequences in 0.0678141 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:44\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_016813735.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_016813735.1_ASM1681373v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 24 sequences in 0.0328848 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:44\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_009904695.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_009904695.1_ASM990469v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 323 sequences in 0.0483758 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:45\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_016812975.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_016812975.1_ASM1681297v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 39 sequences in 0.036736 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:45\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_000968265.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000968265.1_CLOSIBUN125Cv1.0_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 71 sequences in 0.111561 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:45\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_017351805.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_017351805.1_ASM1735180v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 156 sequences in 0.060529 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:45\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001611905.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001611905.1_ASM161190v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 1 sequences in 0.055016 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:45\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_014764705.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_014764705.1_ASM1476470v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 113 sequences in 0.045006 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:46\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001593605.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001593605.1_ASM159360v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 1 sequences in 0.0280478 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:46\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_008386235.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_008386235.1_ASM838623v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 1 sequences in 0.0213571 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:46\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_011393335.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_011393335.1_ASM1139333v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 1 sequences in 0.0261569 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:46\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_014268065.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_014268065.1_ASM1426806v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 215 sequences in 0.055532 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:46\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_005280335.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_005280335.1_ASM528033v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 1 sequences in 0.0214331 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:47\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003664995.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003664995.1_BH458_6_3_2_1_CLC_assembly_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 36 sequences in 0.030678 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:47\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_013726215.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_013726215.1_ASM1372621v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 1 sequences in 0.0326149 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:47\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001494035.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001494035.1_p474D_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 72 sequences in 0.0187199 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:47\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003063435.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003063435.1_ASM306343v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 23 sequences in 0.0347619 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:47\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003925855.2_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003925855.2_ASM392585v2_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Adding sequences from FASTA; added 4 sequences in 0.041625 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:48\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_000593345.2_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000593345.2_vpVPTS2010_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 75 sequences in 0.058589 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:48\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_008632455.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_008632455.1_ASM863245v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 1 sequences in 0.0330241 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:48\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_014856455.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_014856455.1_ASM1485645v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 175 sequences in 0.0466158 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:48\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_002882435.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002882435.1_FW215-L1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 124 sequences in 0.0588269 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:49\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_002507915.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002507915.1_ASM250791v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 55 sequences in 0.039278 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:49\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_006369625.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006369625.1_ASM636962v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 781 sequences in 0.074877 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:49\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_002929435.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002929435.1_ASM292943v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 26 sequences in 0.0378578 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:49\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003199475.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003199475.1_ASM319947v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 78 sequences in 0.0427909 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:49\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003003575.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003003575.1_ASM300357v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 19 sequences in 0.0318279 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:50\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003340555.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003340555.1_ASM334055v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 133 sequences in 0.04426 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:50\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_000489195.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000489195.1_S164_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 541 sequences in 0.0680931 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:50\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_000769825.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000769825.1_ASM76982v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 63 sequences in 0.0388651 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:50\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001833665.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001833665.1_ASM183366v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 49 sequences in 0.0472341 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:50\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_006802465.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006802465.1_ASM680246v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 330 sequences in 0.0487559 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:51\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003296495.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003296495.1_ASM329649v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 444 sequences in 0.0575309 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:51\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001573015.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001573015.1_ASM157301v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 91 sequences in 0.0251601 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:51\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_000737775.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000737775.1_ASM73777v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 89 sequences in 0.0406401 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:51\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_900166155.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_900166155.1_11670_1_22_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 43 sequences in 0.040668 seconds.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:51\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001870285.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001870285.1_ASM187028v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 9 sequences in 0.024852 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:52\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_002156885.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002156885.1_ASM215688v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 203 sequences in 0.058526 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:52\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003348965.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003348965.1_ASM334896v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 80 sequences in 0.034045 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:52\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_013032445.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_013032445.1_ASM1303244v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 82 sequences in 0.0551691 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:52\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003295805.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003295805.1_ASM329580v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 470 sequences in 0.06163 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:52\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_014696555.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_014696555.1_ASM1469655v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 138 sequences in 0.082612 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:53\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_012275045.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_012275045.1_ASM1227504v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 2 sequences in 0.0528569 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:53\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_002192535.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002192535.1_ASM219253v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 5 sequences in 0.0216849 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:53\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_002106585.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002106585.1_ASM210658v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 73 sequences in 0.0396218 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:53\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001832095.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001832095.1_ASM183209v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 29 sequences in 0.039351 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:54\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_000186125.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000186125.1_Bacillus_BT1B_CT2_V1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 24 sequences in 0.046895 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:54\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_005041545.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_005041545.1_ASM504154v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 76 sequences in 0.046679 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:54\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003344765.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003344765.1_ASM334476v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 65 sequences in 0.0379529 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:54\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001707555.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001707555.1_ASM170755v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 26 sequences in 0.0391359 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:54\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_017570045.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_017570045.1_ASM1757004v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 1 sequences in 0.0350308 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:55\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_002106175.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002106175.1_ASM210617v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 70 sequences in 0.043643 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:55\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_018449525.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_018449525.1_ASM1844952v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 30 sequences in 0.048965 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:55\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_000400485.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000400485.1_k0-k1250_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 50 sequences in 0.026485 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:55\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_004023985.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004023985.1_ASM402398v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Adding sequences from FASTA; added 297 sequences in 0.059077 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:55\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003351365.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003351365.1_ASM335136v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 1 sequences in 0.0506411 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:56\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_018282115.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_018282115.1_ASM1828211v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 4 sequences in 0.033469 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:56\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_002216445.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002216445.1_ASM221644v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 93 sequences in 0.0533941 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:56\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_004215335.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004215335.1_ASM421533v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 111 sequences in 0.049005 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:56\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_002881895.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002881895.1_GW704-F2_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 335 sequences in 0.0639899 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:57\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_000979595.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000979595.1_gtlEnvA5udCFS_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 163 sequences in 0.046303 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:57\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_900167585.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_900167585.1_16852_2_94_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 28 sequences in 0.0374341 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:57\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_006517585.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006517585.1_ASM651758v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 159 sequences in 0.0512431 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:57\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_010983895.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_010983895.1_ASM1098389v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 1 sequences in 0.0307488 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:57\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_004745955.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004745955.1_ASM474595v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 137 sequences in 0.047214 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:58\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_002883245.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002883245.1_GW101-3F01_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 127 sequences in 0.060111 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:58\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_002355535.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002355535.1_ASM235553v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 1 sequences in 0.01191 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:58\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_002103335.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002103335.1_ASM210333v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 38 sequences in 0.0354831 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:58\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_004921205.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004921205.1_ASM492120v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 20 sequences in 0.028028 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:58\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_002241775.2_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002241775.2_ASM224177v2_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 61 sequences in 0.044256 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:59\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_012224465.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_012224465.1_ASM1222446v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 1 sequences in 0.0145872 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:59\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_013790265.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_013790265.1_ASM1379026v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 1 sequences in 0.0347681 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:59\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_017696275.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_017696275.1_ASM1769627v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 3 sequences in 0.0145049 seconds.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:59\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003854195.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003854195.1_ASM385419v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 129 sequences in 0.0743492 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:52:59\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_015276975.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_015276975.1_ASM1527697v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 25 sequences in 0.022264 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:00\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_002119585.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002119585.1_ASM211958v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 2 sequences in 0.042078 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:00\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_017349495.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_017349495.1_ASM1734949v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 205 sequences in 0.0713301 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:00\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_015680345.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_015680345.1_ASM1568034v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 57 sequences in 0.047111 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:00\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003096015.2_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003096015.2_ASM309601v2_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 1 sequences in 0.0275741 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:00\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_009790425.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_009790425.1_ASM979042v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 215 sequences in 0.0526111 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:01\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_014526385.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_014526385.1_ASM1452638v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 92 sequences in 0.0522249 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:01\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_009904725.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_009904725.1_ASM990472v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 494 sequences in 0.0669479 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:01\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_006369235.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006369235.1_ASM636923v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 80 sequences in 0.0474548 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:01\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003449015.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003449015.1_ASM344901v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 51 sequences in 0.0451891 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:01\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_016863745.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_016863745.1_ASM1686374v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 107 sequences in 0.050751 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:02\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001507335.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001507335.1_ASM150733v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 16 sequences in 0.0348389 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:02\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003710255.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003710255.1_ASM371025v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 12 sequences in 0.0350032 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:02\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003299525.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003299525.1_ASM329952v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 556 sequences in 0.0604179 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:02\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_900379955.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_900379955.1_ICMP_6278_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 416 sequences in 0.0668032 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:03\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_014893375.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_014893375.1_ASM1489337v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 324 sequences in 0.0579221 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:03\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_000143845.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000143845.1_ASM14384v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 1 sequences in 0.015703 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:03\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_013114355.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_013114355.1_ASM1311435v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Adding sequences from FASTA; added 93 sequences in 0.0509231 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:03\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_010367435.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_010367435.1_ASM1036743v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 74 sequences in 0.020951 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:03\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003574085.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003574085.1_ASM357408v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 224 sequences in 0.038991 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:04\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003049515.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003049515.1_ASM304951v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 71 sequences in 0.0407009 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:04\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_015594835.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_015594835.1_ASM1559483v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 356 sequences in 0.028893 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:04\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003967135.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003967135.1_ASM396713v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 6 sequences in 0.0215859 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:04\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_016603185.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_016603185.1_ASM1660318v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 62 sequences in 0.0190549 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:04\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_017255165.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_017255165.1_ASM1725516v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 23 sequences in 0.032656 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:05\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_006227585.2_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006227585.2_ASM622758v2_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 170 sequences in 0.04636 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:05\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003698745.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003698745.1_ASM369874v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 69 sequences in 0.0655081 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:05\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_004920675.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004920675.1_ASM492067v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 71 sequences in 0.0319171 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:05\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_004345025.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004345025.1_ASM434502v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 1 sequences in 0.023241 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:05\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_004134845.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004134845.1_ASM413484v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 108 sequences in 0.027492 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:06\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_002882455.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002882455.1_FW215-R2_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 138 sequences in 0.054831 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:06\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003295925.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003295925.1_ASM329592v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 468 sequences in 0.0599182 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:06\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_013735435.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_013735435.1_ASM1373543v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 6 sequences in 0.05495 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:06\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001882845.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001882845.1_ASM188284v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 44 sequences in 0.0676892 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:06\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_008642395.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_008642395.1_ASM864239v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 6 sequences in 0.044976 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:07\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_006371235.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006371235.1_ASM637123v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 698 sequences in 0.253173 seconds.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:07\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_005502615.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_005502615.1_ASM550261v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 103 sequences in 0.0479181 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:07\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003265965.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003265965.1_ASM326596v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 324 sequences in 0.0467019 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:08\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_018458215.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_018458215.1_ASM1845821v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 121 sequences in 0.135011 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:08\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_014534785.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_014534785.1_ASM1453478v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 7 sequences in 0.0584671 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:08\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_009176545.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_009176545.1_ASM917654v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 5 sequences in 0.0403371 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:09\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_006370635.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006370635.1_ASM637063v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 219 sequences in 0.050802 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:09\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003839525.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003839525.1_ASM383952v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 51 sequences in 0.060734 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:09\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003427055.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003427055.1_ASM342705v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 2 sequences in 0.0425959 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:09\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_017491835.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_017491835.1_ASM1749183v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 134 sequences in 0.059629 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:09\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001710555.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001710555.1_ASM171055v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 18 sequences in 0.025687 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:10\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_002849795.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002849795.1_ASM284979v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 2 sequences in 0.0294881 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:10\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003294835.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003294835.1_ASM329483v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 435 sequences in 0.056788 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:10\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003034845.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003034845.1_ASM303484v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 41 sequences in 0.032831 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:10\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_004214365.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004214365.1_ASM421436v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 93 sequences in 0.044044 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:10\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_003353065.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003353065.1_ASM335306v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 1 sequences in 0.0311759 seconds.\n",
      "\n",
      "\n",
      "Building a new DB, current time: 07/26/2021 12:53:11\n",
      "New DB name:   /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_blast/databases/GCF_001831805.1_db\n",
      "New DB title:  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001831805.1_ASM183180v1_genomic.fna\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 1000000000B\n",
      "Adding sequences from FASTA; added 28 sequences in 0.041296 seconds.\n",
      "Done\n"
     ]
    }
   ],
   "source": [
    "#Use blast to find tp_gene contamination in negative samples\n",
    "unique_neg_df = neg_df.drop_duplicates(subset='assembly', inplace=False)\n",
    "# gets a list of length of specified amount of neg genomes\n",
    "neg_genomes_list = list(unique_neg_df.iloc[:,1])\n",
    "\n",
    "with open(BGC_path+'/'+'report_2_generate_tp.txt', 'a') as f:\n",
    "    f.write('\\n'+'Using blastn to check negative genomes for contamination'+'\\n')\n",
    "\n",
    "# makes blast databases of all individual neg genomes (easier to keep track of accession numbers than when combining)\n",
    "for genomes in neg_genomes_list:\n",
    "    !makeblastdb -in \"{neg_genomes_path}\"/\"{genomes}\"* -dbtype nucl -out \"{neg_blast_db_path}\"/\"{genomes}\"_db\n",
    "\n",
    "# runs blastn search of all TP genes against all blast databases of negative genomes\n",
    "for genomes in neg_genomes_list:\n",
    "    !blastn -db \"{neg_blast_db_path}\"/\"{genomes}\"_db -query \"{BGC_path}\"/\"{BGC_type}\"_selected_tp_genes.fasta -out \"{neg_blast_results_path}\"/\"{genomes}\".blastout -outfmt \"6 qseqid sseqid pident evalue\"\n",
    "\n",
    "# use pandas to concatenate all blast output tables\n",
    "df_list = []\n",
    "for outfile in os.listdir(neg_blast_results_path):\n",
    "    try:\n",
    "        blast_df = pd.read_csv(neg_blast_results_path+'/'+outfile, sep='\\t', names=['qseqid', 'sseqid', 'pident', 'evalue'], index_col=None)\n",
    "        blast_df['subject_accession'] = '.'.join(outfile.split('.')[0:2])\n",
    "        df_list.append(blast_df)\n",
    "    except EmptyDataError:\n",
    "        continue\n",
    "        \n",
    "# Generate a list of contaminated negative genomes        \n",
    "remove_df = pd.concat(df_list)\n",
    "unique_remove_df = remove_df.drop_duplicates(subset='subject_accession', inplace=False)\n",
    "blast_remove_list = list(unique_remove_df.iloc[:,4])\n",
    "\n",
    "with open(BGC_path+'/'+'report_2_generate_tp.txt', 'a') as f:\n",
    "    f.write('\\n'+'Contamination with tp_seqs:\\n'+str(list(remove_df.iloc[:,0]))+'\\nsequences identified in negative samples:\\n'+str(blast_remove_list)+'\\n\\n')\n",
    "\n",
    "print('Done')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "MUSCLE v3.8.1551 by Robert C. Edgar\n",
      "\n",
      "http://www.drive5.com/muscle\n",
      "This software is donated to the public domain.\n",
      "Please cite: Edgar, R.C. Nucleic Acids Res 32(5), 1792-97.\n",
      "\n",
      "nitrile_hydratase_alpha_selecte 11 seqs, lengths min 597, max 672, avg 627\n",
      "00:00:00    15 MB(-3%)  Iter   1  100.00%  K-mer dist pass 1\n",
      "00:00:00    15 MB(-3%)  Iter   1  100.00%  K-mer dist pass 2\n",
      "00:00:00    22 MB(-4%)  Iter   1  100.00%  Align node       \n",
      "00:00:00    22 MB(-4%)  Iter   1  100.00%  Root alignment\n",
      "00:00:00    22 MB(-4%)  Iter   2  100.00%  Refine tree   \n",
      "00:00:00    22 MB(-4%)  Iter   2  100.00%  Root alignment\n",
      "00:00:00    22 MB(-4%)  Iter   2  100.00%  Root alignment\n",
      "00:00:00    22 MB(-4%)  Iter   3  100.00%  Refine biparts\n",
      "00:00:01    22 MB(-4%)  Iter   4  100.00%  Refine biparts\n",
      "00:00:01    22 MB(-4%)  Iter   5  100.00%  Refine biparts\n",
      "00:00:01    22 MB(-4%)  Iter   5  100.00%  Refine biparts\n",
      "# hmmbuild :: profile HMM construction from multiple sequence alignments\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# input alignment file:             /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_selected_tp_gene_alignment.fasta\n",
      "# output HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# name (the single) HMM:            tp_nucl_aln\n",
      "# input alignment is asserted as:   DNA\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "# idx name                  nseq  alen  mlen     W eff_nseq re/pos description\n",
      "#---- -------------------- ----- ----- ----- ----- -------- ------ -----------\n",
      "1     tp_nucl_aln             11   741   624  1121     5.01  0.620 \n",
      "\n",
      "# CPU time: 0.12u 0.00s 00:00:00.12 Elapsed: 00:00:00.11\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000520615.1_SOAPdenovo_v1.05_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_000520615.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         64  (4638882 residues searched)\n",
      "Passed MSV filter:                        29  (0.453125); expected 1.3 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 1.3 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 1.3 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.3 (0.02)\n",
      "Initial search space (Z):                 64  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.21u 0.01s 00:00:00.22 Elapsed: 00:00:00.12\n",
      "# Mc/sec: 23055.09\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000355675.1_ASM35567v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_000355675.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (3920328 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.07u 0.01s 00:00:00.08 Elapsed: 00:00:00.09\n",
      "# Mc/sec: 26210.68\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_013277795.1_ASM1327779v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_013277795.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         36  (6965470 residues searched)\n",
      "Passed MSV filter:                        11  (0.305556); expected 0.7 (0.02)\n",
      "Passed bias filter:                        2  (0.0555556); expected 0.7 (0.02)\n",
      "Passed Vit filter:                         2  (0.0555556); expected 0.7 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.7 (0.02)\n",
      "Initial search space (Z):                 36  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.90u 0.02s 00:00:00.92 Elapsed: 00:00:00.69\n",
      "# Mc/sec: 6278.98\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000353285.1_ASM35328v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_000353285.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence      Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------      -----------\n",
      "    2.9e-94  305.1   0.0    6.7e-94  303.9   0.0    1.5  1  NZ_KB731324.1  Leptolyngbya boryana PCC 6306 LepboDRAFT_LPC.1\n",
      "    0.00067    6.5   0.0     0.0013    5.6   0.0    1.3  1  NZ_KB731327.1  Leptolyngbya boryana PCC 6306 LepboDRAFT_LPC.4\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_KB731324.1  Leptolyngbya boryana PCC 6306 LepboDRAFT_LPC.1, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 !  303.9   0.0   2.7e-94   6.7e-94      53     602 .. 4654683 4655244 .. 4654668 4655260 .. 0.96\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 303.9 bits;  conditional E-value: 2.7e-94\n",
      "    tp_nucl_aln      53 aaactgctttgcgggtcagggcgctggAaacgcttctgattgaaAAgGGgctgatcgatcccgcagcgctggatgagatcatcgacac 140    \n",
      "                        a a tgct+  cg gt ag gcg+t gA  cg t+ct attga+AA+GGg t at+  +   gca+cg+t ga+ ag t atcga   \n",
      "  NZ_KB731324.1 4654683 ATAGTGCTGCTCGTGTGAGAGCGTTAGAGTCGTTGCTAATTGAGAAAGGGGTCATTACCAGTGCAACGGTTGACAAGGTAATCGAGTT 4654770\n",
      "                        56778999******************************************************************************** PP\n",
      "\n",
      "    tp_nucl_aln     141 ctatgagaacaagatcGgcccgcggAAcGGtGcccacgtgGtgGcgcgggcctGGaccGAcccggagttccgggcgcggctgctggag 228    \n",
      "                        +t tgagaa +a at Gg ccg   AA+GGtGcc a  t Gt Gc    gcctGG++ GA+cc+gagtt+++g +gc  +tgct ga \n",
      "  NZ_KB731324.1 4654771 TTTTGAGAAAGATATGGGACCGTTTAATGGTGCCAAAATTGTTGCAAAAGCCTGGGTTGATCCAGAGTTTAAGCAGCAATTGCTCGAA 4654858\n",
      "                        **************************************************************************************** PP\n",
      "\n",
      "    tp_nucl_aln     229 GAtgccactgcgGcgatcgccgagctgGggttt............tctggacgtcaggggGAacAtAtggtgGcggtggagAAcacgg 304    \n",
      "                        GAt c  c  c+Gcgatcgc g g+tg   ttt             c gg   + a gg GAacA+Atg   Gc gt g gAA+ac g\n",
      "  NZ_KB731324.1 4654859 GATACGCCGTCTGCGATCGCAGTGATGAATTTTccgcgtggcatgGCAGGTGCAGAAGGTGAACACATGAAAGCAGTTGCGAATACTG 4654946\n",
      "                        **************************9888887777777776666677888889999******************************* PP\n",
      "\n",
      "    tp_nucl_aln     305 ccgaggtgcAcAAtatggtcgtctGcAccctgtGctcctGctAcccgtGgcctctgctgGGcctgccgccggcctGGtAcAAatccgc 392    \n",
      "                        ++ga gt cAcAAt+t +t gt tG+Ac+ tgtG tcctGctAccc tGgcct+tg t GG ctgccgcc   ctGGt cAAa   +c\n",
      "  NZ_KB731324.1 4654947 ATGACGTTCACAATCTCATTGTTTGTACGATGTGTTCCTGCTACCCTTGGCCTGTGTTAGGACTGCCGCCCTACTGGTTCAAAGATCC 4655034\n",
      "                        **************************************************************************************** PP\n",
      "\n",
      "    tp_nucl_aln     393 tgcctaccGcgcccGcgcgGtccgcGAaccgcgcagcGtgctggcgGAgtttGGcgtcaccctGcccgatgacatggaggtgcgcGtc 480    \n",
      "                          c t +cG gc  G gc  t cgcGAacc cgca+ Gt  tg + GAgtttGG  t ++c t cccga+   a+ ga +t+cg Gtc\n",
      "  NZ_KB731324.1 4655035 GACGTTTCGAGCGAGAGCAATTCGCGAACCTCGCAAAGTCTTGCAAGAGTTTGGGCTAGACATTCCCGAATCGACTGAAATTCGAGTC 4655122\n",
      "                        **************************************************************************************** PP\n",
      "\n",
      "    tp_nucl_aln     481 tggGAttccAccGccgAggtgcGctatctgGtgatgcccgagcGccctgaagggAccGaaggtctGgacgaggaggcgctggccgcgc 568    \n",
      "                        tggGAt+c A cGcc A++t+cG t   t Gtg t cc ga+cGccc ga gg AccGa+g t+ G+  gaggag +gct gc gc  \n",
      "  NZ_KB731324.1 4655123 TGGGATACGAGCGCCCAAATCCGATGGTTTGTGGTTCCAGAACGCCCAGATGGAACCGAGGATTGGACAGAGGAGCAGCTTGCTGCCA 4655210\n",
      "                        **************************************************************************************** PP\n",
      "\n",
      "    tp_nucl_aln     569 tgGtcAcccgcgAcagcAtGatcGGtacgGggct 602    \n",
      "                        t  t Accc +gA ++ AtG   GG +c G+   \n",
      "  NZ_KB731324.1 4655211 TCATTACCCCTGAATCAATGCAAGGGGCAGCTAA 4655244\n",
      "                        *******************977777666666555 PP\n",
      "\n",
      ">> NZ_KB731327.1  Leptolyngbya boryana PCC 6306 LepboDRAFT_LPC.4, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 !    5.6   0.0   0.00052    0.0013     192     268 ..    2066    2139 ..    2060    2146 .. 0.91\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 5.6 bits;  conditional E-value: 0.00052\n",
      "    tp_nucl_aln  192 ctGGaccGAcccggagttccgggcgcggctgctggagGAtgccactgcgGcgatcgccgagctgGggttttctggac 268 \n",
      "                     ctGGac GAcc     ttc   gcg++++t ct  a  Atgc ++ gcgGcg t+ c g  +tgGggt+t ctgga \n",
      "  NZ_KB731327.1 2066 CTGGACAGACC---TATTCTTTGCGTCCTTTCTAAAAAATGCTGAGGCGGCGGTTTCAGCAATGGGGTATGCTGGAT 2139\n",
      "                     78999999998...56888889999***********************************************99976 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          5  (7262454 residues searched)\n",
      "Passed MSV filter:                         3  (0.6); expected 0.1 (0.02)\n",
      "Passed bias filter:                        2  (0.4); expected 0.1 (0.02)\n",
      "Passed Vit filter:                         2  (0.4); expected 0.1 (0.02)\n",
      "Passed Fwd filter:                         2  (0.4); expected 0.1 (0.02)\n",
      "Initial search space (Z):                  5  [actual number of targets]\n",
      "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
      "# CPU time: 19.05u 0.17s 00:00:19.22 Elapsed: 00:00:19.19\n",
      "# Mc/sec: 236.13\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001624595.1_ASM162459v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001624595.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.33    1.4   0.0       0.58    0.6   0.0    1.2  1  NZ_JSEN01000002.1  Marinomonas sp. SBI22 contig_2, whole geno\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_JSEN01000002.1  Marinomonas sp. SBI22 contig_2, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    0.6   0.0    0.0085      0.58     189     267 ..  299037  299115 ..  299024  299128 .. 0.93\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 0.6 bits;  conditional E-value: 0.0085\n",
      "        tp_nucl_aln    189 ggcctGGaccGAcccggagttccgggcgcggctgctggagGAtgccactgcgGcgatcgccgagctgGggttttctgga 267   \n",
      "                            gc tGG c GA+    a tt+++g c++  ctgct ga GA+ c  +  +gGcg t+gc   gct Gg+ttt+ctgg \n",
      "  NZ_JSEN01000002.1 299037 TGCTTGGTCTGATGATAACTTTAAGTCAGTACTGCTTGAAGACTCTCAAAAGGCGTTTGCAATGCTAGGCTTTACTGGT 299115\n",
      "                           5899***********************************************************************9985 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         68  (5191569 residues searched)\n",
      "Passed MSV filter:                         3  (0.0441176); expected 1.4 (0.02)\n",
      "Passed bias filter:                        3  (0.0441176); expected 1.4 (0.02)\n",
      "Passed Vit filter:                         3  (0.0441176); expected 1.4 (0.02)\n",
      "Passed Fwd filter:                         1  (0.0147059); expected 1.4 (0.02)\n",
      "Initial search space (Z):                 68  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 1.76u 0.01s 00:00:01.77 Elapsed: 00:00:01.69\n",
      "# Mc/sec: 1910.47\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_013249065.1_ASM1324906v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_013249065.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         63  (3888081 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 1.3 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 1.3 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 1.3 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.3 (0.02)\n",
      "Initial search space (Z):                 63  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.09u 0.01s 00:00:00.10 Elapsed: 00:00:00.05\n",
      "# Mc/sec: 41571.43\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001529305.1_ASM152930v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001529305.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         82  (4917586 residues searched)\n",
      "Passed MSV filter:                        15  (0.182927); expected 1.6 (0.02)\n",
      "Passed bias filter:                        1  (0.0121951); expected 1.6 (0.02)\n",
      "Passed Vit filter:                         1  (0.0121951); expected 1.6 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.6 (0.02)\n",
      "Initial search space (Z):                 82  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.18u 0.02s 00:00:00.20 Elapsed: 00:00:00.11\n",
      "# Mc/sec: 26090.50\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003050585.1_ASM305058v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003050585.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         30  (5138901 residues searched)\n",
      "Passed MSV filter:                         1  (0.0333333); expected 0.6 (0.02)\n",
      "Passed bias filter:                        1  (0.0333333); expected 0.6 (0.02)\n",
      "Passed Vit filter:                         1  (0.0333333); expected 0.6 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.6 (0.02)\n",
      "Initial search space (Z):                 30  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.47u 0.02s 00:00:00.49 Elapsed: 00:00:00.43\n",
      "# Mc/sec: 7414.76\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001044395.1_ASM104439v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001044395.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         74  (4721438 residues searched)\n",
      "Passed MSV filter:                        24  (0.324324); expected 1.5 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 1.5 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 1.5 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.5 (0.02)\n",
      "Initial search space (Z):                 74  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.21u 0.01s 00:00:00.22 Elapsed: 00:00:00.12\n",
      "# Mc/sec: 23389.09\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_018199995.1_ASM1819999v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_018199995.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          3  (5310201 residues searched)\n",
      "Passed MSV filter:                         1  (0.333333); expected 0.1 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.1 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.1 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.1 (0.02)\n",
      "Initial search space (Z):                  3  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.20u 0.02s 00:00:00.22 Elapsed: 00:00:00.18\n",
      "# Mc/sec: 17872.78\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001854765.1_ASM185476v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001854765.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        138  (4767843 residues searched)\n",
      "Passed MSV filter:                         1  (0.00724638); expected 2.8 (0.02)\n",
      "Passed bias filter:                        1  (0.00724638); expected 2.8 (0.02)\n",
      "Passed Vit filter:                         1  (0.00724638); expected 2.8 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 2.8 (0.02)\n",
      "Initial search space (Z):                138  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.23u 0.00s 00:00:00.23 Elapsed: 00:00:00.18\n",
      "# Mc/sec: 16442.98\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003171655.1_ASM317165v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003171655.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         18  (3850965 residues searched)\n",
      "Passed MSV filter:                         5  (0.277778); expected 0.4 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.4 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.4 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.4 (0.02)\n",
      "Initial search space (Z):                 18  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.13u 0.00s 00:00:00.13 Elapsed: 00:00:00.09\n",
      "# Mc/sec: 24628.26\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_015864865.1_ASM1586486v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_015864865.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         10  (2934056 residues searched)\n",
      "Passed MSV filter:                         3  (0.3); expected 0.2 (0.02)\n",
      "Passed bias filter:                        1  (0.1); expected 0.2 (0.02)\n",
      "Passed Vit filter:                         1  (0.1); expected 0.2 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.2 (0.02)\n",
      "Initial search space (Z):                 10  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.09u 0.01s 00:00:00.10 Elapsed: 00:00:00.06\n",
      "# Mc/sec: 27746.47\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_009846755.1_ASM984675v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_009846755.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         78  (4179456 residues searched)\n",
      "Passed MSV filter:                        25  (0.320513); expected 1.6 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 1.6 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 1.6 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.6 (0.02)\n",
      "Initial search space (Z):                 78  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.14u 0.00s 00:00:00.14 Elapsed: 00:00:00.07\n",
      "# Mc/sec: 33524.49\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_005938125.1_ASM593812v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_005938125.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         68  (4230810 residues searched)\n",
      "Passed MSV filter:                         4  (0.0588235); expected 1.4 (0.02)\n",
      "Passed bias filter:                        1  (0.0147059); expected 1.4 (0.02)\n",
      "Passed Vit filter:                         1  (0.0147059); expected 1.4 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.4 (0.02)\n",
      "Initial search space (Z):                 68  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.10u 0.00s 00:00:00.10 Elapsed: 00:00:00.05\n",
      "# Mc/sec: 44931.16\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_017168275.1_ASM1716827v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_017168275.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         80  (5448500 residues searched)\n",
      "Passed MSV filter:                        14  (0.175); expected 1.6 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 1.6 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 1.6 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.6 (0.02)\n",
      "Initial search space (Z):                 80  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.24u 0.02s 00:00:00.26 Elapsed: 00:00:00.16\n",
      "# Mc/sec: 20228.60\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003086815.1_ASM308681v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003086815.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (4319112 residues searched)\n",
      "Passed MSV filter:                         1  (1); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.20u 0.03s 00:00:00.23 Elapsed: 00:00:00.23\n",
      "# Mc/sec: 11685.15\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_016837255.1_ASM1683725v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_016837255.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        115  (3159994 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 2.3 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 2.3 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 2.3 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 2.3 (0.02)\n",
      "Initial search space (Z):                115  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.08u 0.00s 00:00:00.08 Elapsed: 00:00:00.05\n",
      "# Mc/sec: 38348.52\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003015025.1_ASM301502v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003015025.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         33  (4926044 residues searched)\n",
      "Passed MSV filter:                        13  (0.393939); expected 0.7 (0.02)\n",
      "Passed bias filter:                        1  (0.030303); expected 0.7 (0.02)\n",
      "Passed Vit filter:                         1  (0.030303); expected 0.7 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.7 (0.02)\n",
      "Initial search space (Z):                 33  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.88u 0.04s 00:00:00.92 Elapsed: 00:00:00.79\n",
      "# Mc/sec: 3880.43\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004378355.1_ASM437835v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_004378355.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.31    2.2   0.0       0.45    1.7   0.0    1.1  1  NZ_SNUB01000079.1  Psychromonas sp. RZ5 RZ5_contig_79, whole \n",
      "        1.8   -0.3   0.0        3.2   -1.1   0.0    1.1  1  NZ_SNUB01000013.1  Psychromonas sp. RZ5 RZ5_contig_13, whole \n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_SNUB01000079.1  Psychromonas sp. RZ5 RZ5_contig_79, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    1.7   0.0    0.0077      0.45     287     344 ..     596     650 ..     591     663 .. 0.88\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 1.7 bits;  conditional E-value: 0.0077\n",
      "        tp_nucl_aln 287 tgGcggtggagAAcacggccgaggtgcAcAAtatggtcgtctGcAccctgtGctcctG 344\n",
      "                        tgG+ggt ga+AA     c g+ gtgc +AAt tggt    tG+Ac    tG   ctG\n",
      "  NZ_SNUB01000079.1 596 TGGTGGTTGAAAAAGTATCAGGAGTGCCTAATTTGGTAAGTTGTACT---TGTAGCTG 650\n",
      "                        6899*****************************************96...44444444 PP\n",
      "\n",
      ">> NZ_SNUB01000013.1  Psychromonas sp. RZ5 RZ5_contig_13, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.1   0.0     0.056       3.2      88     141 ..   62978   63031 ..   62965   63046 .. 0.86\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.1 bits;  conditional E-value: 0.056\n",
      "        tp_nucl_aln    88 ctgattgaaAAgGGgctgatcgatcccgcagcgctggatgagatcatcgacacc 141  \n",
      "                           t +++g  AA+G g tgat gatc c + gcgc  gatga  t  t ga ac+\n",
      "  NZ_SNUB01000013.1 62978 ATTGCGGGTAAAGTGATGATGGATCGCAATGCGCCTGATGACTTATTAGATACT 63031\n",
      "                          5677889999**********************************9999998863 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        116  (3511338 residues searched)\n",
      "Passed MSV filter:                         4  (0.0344828); expected 2.3 (0.02)\n",
      "Passed bias filter:                        2  (0.0172414); expected 2.3 (0.02)\n",
      "Passed Vit filter:                         2  (0.0172414); expected 2.3 (0.02)\n",
      "Passed Fwd filter:                         2  (0.0172414); expected 2.3 (0.02)\n",
      "Initial search space (Z):                116  [actual number of targets]\n",
      "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
      "# CPU time: 0.36u 0.00s 00:00:00.36 Elapsed: 00:00:00.31\n",
      "# Mc/sec: 6958.06\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003050175.1_ASM305017v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003050175.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.19    1.8   0.0       0.32    1.1   0.0    1.2  1  NZ_PIEX01000039.1  Vibrio splendidus strain ZS_82 Contig_39, \n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_PIEX01000039.1  Vibrio splendidus strain ZS_82 Contig_39, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    1.1   0.0     0.006      0.32     425     465 ..     504     544 ..     497     551 .. 0.85\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 1.1 bits;  conditional E-value: 0.006\n",
      "        tp_nucl_aln 425 gcagcGtgctggcgGAgtttGGcgtcaccctGcccgatgac 465\n",
      "                        g++g  tgct  c GA+tttGGc t++++ tGcc  a g++\n",
      "  NZ_PIEX01000039.1 504 GAGGTTTGCTTTCCGAATTTGGCATTGTGATGCCGCAGGGT 544\n",
      "                        556666899999**********************9988765 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         53  (5373700 residues searched)\n",
      "Passed MSV filter:                         1  (0.0188679); expected 1.1 (0.02)\n",
      "Passed bias filter:                        1  (0.0188679); expected 1.1 (0.02)\n",
      "Passed Vit filter:                         1  (0.0188679); expected 1.1 (0.02)\n",
      "Passed Fwd filter:                         1  (0.0188679); expected 1.1 (0.02)\n",
      "Initial search space (Z):                 53  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.15u 0.00s 00:00:00.15 Elapsed: 00:00:00.08\n",
      "# Mc/sec: 40823.89\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002873185.1_ASM287318v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_002873185.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.55    2.4   0.0          1    1.5   0.0    1.3  1  NZ_MCVH01000042.1  Vibrio lentus strain 10N.286.48.A9 10N.286\n",
      "        6.3   -1.1   0.0         20   -2.8   0.0    1.3  1  NZ_MCVH01000181.1  Vibrio lentus strain 10N.286.48.A9 10N.286\n",
      "        6.5   -1.2   0.0         13   -2.2   0.0    1.2  1  NZ_MCVH01000152.1  Vibrio lentus strain 10N.286.48.A9 10N.286\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_MCVH01000042.1  Vibrio lentus strain 10N.286.48.A9 10N.286.48.A9_contig_135, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    1.5   0.0     0.014         1      72     116 ..    1033    1077 ..    1022    1086 .. 0.92\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 1.5 bits;  conditional E-value: 0.014\n",
      "        tp_nucl_aln   72 ggcgctggAaacgcttctgattgaaAAgGGgctgatcgatcccgc 116 \n",
      "                          gcgct gAa+c  ttctg t gaaAA+ G+c   t gat  cgc\n",
      "  NZ_MCVH01000042.1 1033 AGCGCTTGAAGCCTTTCTGCTAGAAAAAAGTCATGTGGATGTCGC 1077\n",
      "                         589**********************************99988876 PP\n",
      "\n",
      ">> NZ_MCVH01000181.1  Vibrio lentus strain 10N.286.48.A9 10N.286.48.A9_contig_6, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.8   0.0      0.26        20     119     220 ..   58824   58928 ..   58803   58971 .. 0.78\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.8 bits;  conditional E-value: 0.26\n",
      "        tp_nucl_aln   119 cgctggatgagatcatcgacacctatgagaacaagatcGgcccgcggAAcGGtGcccacgtgGtgGcgcg..ggcctGGa.ccGAccc 203  \n",
      "                          c++t g +  g tca  g   cct  g gaaca  +t Ggc c  g AA GG Gcc    t GtgGc+     gc  GGa +cGA+c \n",
      "  NZ_MCVH01000181.1 58824 CCGTTGGCTTGTTCAAAGCGCCCTTGGCGAACACTGTTGGCTCCGGTAAAGGCGCCTTTCTCGTGGCCAAacAGCTCGGAtTCGATCA 58911\n",
      "                          4444444444555555666778999999**************************************9875334777788635899999 PP\n",
      "\n",
      "        tp_nucl_aln   204 ggagttccgggcgcggc 220  \n",
      "                          +g  tt++ggg+ +g++\n",
      "  NZ_MCVH01000181.1 58912 AGTCTTTAGGGATTGCT 58928\n",
      "                          99999999999877654 PP\n",
      "\n",
      ">> NZ_MCVH01000152.1  Vibrio lentus strain 10N.286.48.A9 10N.286.48.A9_contig_33, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.2   0.0      0.18        13     380     402 ..    2961    2983 ..    2957    2987 .. 0.93\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.2 bits;  conditional E-value: 0.18\n",
      "        tp_nucl_aln  380 GGtAcAAatccgctgcctaccGc 402 \n",
      "                         GGtAcAAa  cgc+ ccta+cGc\n",
      "  NZ_MCVH01000152.1 2961 GGTACAAAAACGCACCCTATCGC 2983\n",
      "                         89*******************97 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        224  (5120478 residues searched)\n",
      "Passed MSV filter:                         5  (0.0223214); expected 4.5 (0.02)\n",
      "Passed bias filter:                        5  (0.0223214); expected 4.5 (0.02)\n",
      "Passed Vit filter:                         5  (0.0223214); expected 4.5 (0.02)\n",
      "Passed Fwd filter:                         3  (0.0133929); expected 4.5 (0.02)\n",
      "Initial search space (Z):                224  [actual number of targets]\n",
      "Domain search space  (domZ):               3  [number of targets reported over threshold]\n",
      "# CPU time: 0.80u 0.01s 00:00:00.81 Elapsed: 00:00:00.59\n",
      "# Mc/sec: 5411.42\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_011806565.1_ASM1180656v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_011806565.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         24  (3337074 residues searched)\n",
      "Passed MSV filter:                         8  (0.333333); expected 0.5 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.5 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.5 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.5 (0.02)\n",
      "Initial search space (Z):                 24  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.10u 0.01s 00:00:00.11 Elapsed: 00:00:00.06\n",
      "# Mc/sec: 32414.04\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_005771535.1_ASM577153v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_005771535.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.49   -1.1   0.0       0.87   -1.9   0.0    1.1  1  NZ_VANU01000012.1  Halarcobacter arenosus strain CAU 1517 con\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_VANU01000012.1  Halarcobacter arenosus strain CAU 1517 contig12, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.9   0.0     0.048      0.87     270     293 ..    1449    1472 ..    1424    1476 .. 0.81\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.9 bits;  conditional E-value: 0.048\n",
      "        tp_nucl_aln  270 tcaggggGAacAtAtggtgGcggt 293 \n",
      "                         t agg gGAacAtAtggtgG+  t\n",
      "  NZ_VANU01000012.1 1449 TTAGGTGGAACATATGGTGGTTCT 1472\n",
      "                         568999*************98766 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         18  (3134893 residues searched)\n",
      "Passed MSV filter:                         1  (0.0555556); expected 0.4 (0.02)\n",
      "Passed bias filter:                        1  (0.0555556); expected 0.4 (0.02)\n",
      "Passed Vit filter:                         1  (0.0555556); expected 0.4 (0.02)\n",
      "Passed Fwd filter:                         1  (0.0555556); expected 0.4 (0.02)\n",
      "Initial search space (Z):                 18  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.10u 0.01s 00:00:00.11 Elapsed: 00:00:00.06\n",
      "# Mc/sec: 29493.99\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_016461915.1_ASM1646191v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_016461915.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence             Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------             -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.18    3.3   0.0       0.48    1.9   0.0    1.6  2  NZ_JAEMOQ010000107.1  Marinobacter salarius strain NP2017 NOD\n",
      "       0.28    2.6   0.0       0.47    1.9   0.0    1.2  1  NZ_JAEMOQ010000113.1  Marinobacter salarius strain NP2017 NOD\n",
      "        1.6    0.1   0.0        3.8   -1.1   0.0    1.3  1  NZ_JAEMOQ010000131.1  Marinobacter salarius strain NP2017 NOD\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_JAEMOQ010000107.1  Marinobacter salarius strain NP2017 NODE_113_length_4416_cov_11.695426, whole genome shotgun se\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    1.9   0.0      0.01      0.48      80     137 ..     864     921 ..     803     935 .. 0.89\n",
      "   2 ?   -4.4   0.0      0.79        37     224     251 ..    2737    2764 ..    2735    2768 .. 0.89\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 1.9 bits;  conditional E-value: 0.01\n",
      "           tp_nucl_aln  80 AaacgcttctgattgaaAAgGGgctgatcgatcccgcagcgctggatgagatcatcga 137\n",
      "                           A +  ct  tga tg   A GGgctg tcg + c g agcgct g tga+atcatcg \n",
      "  NZ_JAEMOQ010000107.1 864 AGGAACTCATGAATGGCGATGGGCTGCTCGGCACGGTAGCGCTTGGTGAAATCATCGC 921\n",
      "                           45667889999999999999***********************************985 PP\n",
      "\n",
      "  == domain 2  score: -4.4 bits;  conditional E-value: 0.79\n",
      "           tp_nucl_aln  224 tggagGAtgccactgcgGcgatcgccga 251 \n",
      "                            tg +gGAtg+ a+ gcgGc+at gccg \n",
      "  NZ_JAEMOQ010000107.1 2737 TGCCGGATGGAAAGGCGGCTATGGCCGC 2764\n",
      "                            78899*******************9986 PP\n",
      "\n",
      ">> NZ_JAEMOQ010000113.1  Marinobacter salarius strain NP2017 NODE_119_length_963_cov_10.855659, whole genome shotgun seq\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    1.9   0.0      0.01      0.47     514     557 ..     955     998 ..     942    1008 .. 0.90\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 1.9 bits;  conditional E-value: 0.01\n",
      "           tp_nucl_aln 514 atgcccgagcGccctgaagggAccGaaggtctGgacgaggaggc 557\n",
      "                           +t cc g  cG cct a gg A cGa g  + Gga+g  ga g+\n",
      "  NZ_JAEMOQ010000113.1 955 TTTCCTGCTCGGCCTAACGGAATCGATGCGAGGGATGGTGACGA 998\n",
      "                           6889999*****************************99998876 PP\n",
      "\n",
      ">> NZ_JAEMOQ010000131.1  Marinobacter salarius strain NP2017 NODE_143_length_2047_cov_11.197362, whole genome shotgun se\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.1   0.0     0.082       3.8     523     574 ..    1784    1835 ..    1778    1849 .. 0.87\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.1 bits;  conditional E-value: 0.082\n",
      "           tp_nucl_aln  523 cGccctgaagggAccGaaggtctGgacgaggaggcgctggccgcgctgGtcA 574 \n",
      "                            cGccc+g a  +A  G  gg ctGga+ a ga gc ctg ccgc  t  tcA\n",
      "  NZ_JAEMOQ010000131.1 1784 CGCCCCGGAATCAAAGCCGGGCTGGATAACGACGCCCTGCCCGCCTTCATCA 1835\n",
      "                            678888888888999999***********************99998887776 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        139  (4359230 residues searched)\n",
      "Passed MSV filter:                        27  (0.194245); expected 2.8 (0.02)\n",
      "Passed bias filter:                        3  (0.0215827); expected 2.8 (0.02)\n",
      "Passed Vit filter:                         3  (0.0215827); expected 2.8 (0.02)\n",
      "Passed Fwd filter:                         3  (0.0215827); expected 2.8 (0.02)\n",
      "Initial search space (Z):                139  [actual number of targets]\n",
      "Domain search space  (domZ):               3  [number of targets reported over threshold]\n",
      "# CPU time: 0.16u 0.00s 00:00:00.16 Elapsed: 00:00:00.09\n",
      "# Mc/sec: 29328.35\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004803475.1_ASM480347v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_004803475.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         15  (4076386 residues searched)\n",
      "Passed MSV filter:                         3  (0.2); expected 0.3 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.3 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.3 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.3 (0.02)\n",
      "Initial search space (Z):                 15  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.15u 0.01s 00:00:00.16 Elapsed: 00:00:00.09\n",
      "# Mc/sec: 28049.94\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000371805.1_ASM37180v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_000371805.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          2  (1249385 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  2  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.05u 0.00s 00:00:00.05 Elapsed: 00:00:00.05\n",
      "# Mc/sec: 14075.07\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001730205.1_ASM173020v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001730205.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         65  (4786889 residues searched)\n",
      "Passed MSV filter:                        21  (0.323077); expected 1.3 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 1.3 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 1.3 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.3 (0.02)\n",
      "Initial search space (Z):                 65  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.16u 0.01s 00:00:00.17 Elapsed: 00:00:00.09\n",
      "# Mc/sec: 31975.50\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_014858715.1_ASM1485871v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_014858715.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         10  (5441065 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.2 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.2 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.2 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.2 (0.02)\n",
      "Initial search space (Z):                 10  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.13u 0.02s 00:00:00.15 Elapsed: 00:00:00.08\n",
      "# Mc/sec: 38012.36\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002021755.1_ASM202175v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_002021755.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          2  (5890205 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  2  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.13u 0.00s 00:00:00.13 Elapsed: 00:00:00.08\n",
      "# Mc/sec: 43455.56\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000737535.1_ASM73753v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_000737535.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (2789000 residues searched)\n",
      "Passed MSV filter:                         1  (1); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.15u 0.02s 00:00:00.17 Elapsed: 00:00:00.16\n",
      "# Mc/sec: 10772.95\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_014076905.1_ASM1407690v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_014076905.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence             Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------             -----------\n",
      "  ------ inclusion threshold ------\n",
      "      0.082    5.0   0.0       0.12    4.5   0.0    1.2  1  NZ_JACGUW010000133.1  Colwellia sp. MB02u-7 NODE_965324_lengt\n",
      "        1.5    0.8   0.0        2.8   -0.0   0.0    1.2  1  NZ_JACGUW010000053.1  Colwellia sp. MB02u-7 NODE_949272_lengt\n",
      "        1.9    0.6   0.0        2.3    0.3   0.0    1.1  1  NZ_JACGUW010000014.1  Colwellia sp. MB02u-7 NODE_898706_lengt\n",
      "        5.3   -0.9   0.0        7.7   -1.5   0.0    1.1  1  NZ_JACGUW010000060.1  Colwellia sp. MB02u-7 NODE_951092_lengt\n",
      "        5.7   -1.0   0.0        8.2   -1.6   0.0    1.1  1  NZ_JACGUW010000123.1  Colwellia sp. MB02u-7 NODE_964088_lengt\n",
      "        7.9   -1.5   0.0         14   -2.3   0.0    1.1  1  NZ_JACGUW010000077.1  Colwellia sp. MB02u-7 NODE_956410_lengt\n",
      "          8   -1.5   0.0         12   -2.1   0.0    1.1  1  NZ_JACGUW010000034.1  Colwellia sp. MB02u-7 NODE_931926_lengt\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_JACGUW010000133.1  Colwellia sp. MB02u-7 NODE_965324_length_1378_cov_36.2, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    4.5   0.0    0.0039      0.12     426     464 ..     506     544 ..     495     553 .. 0.87\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 4.5 bits;  conditional E-value: 0.0039\n",
      "           tp_nucl_aln 426 cagcGtgctggcgGAgtttGGcgtcaccctGcccgatga 464\n",
      "                           ++g+ t ct gc+GA+tt+GGc tca+c tGcc+ a+g+\n",
      "  NZ_JACGUW010000133.1 506 AGGGTTACTCGCTGAATTCGGCATCATCATGCCTAAAGG 544\n",
      "                           55666899*************************999876 PP\n",
      "\n",
      ">> NZ_JACGUW010000053.1  Colwellia sp. MB02u-7 NODE_949272_length_954_cov_15.7, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.0   0.0      0.09       2.8     122     180 ..     349     407 ..     331     414 .. 0.89\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.0 bits;  conditional E-value: 0.09\n",
      "           tp_nucl_aln 122 tggatgagatcatcgacacctatgagaacaagatcGgcccgcggAAcGGtGcccacgtg 180\n",
      "                           t g t +g tc tcg ca +t  g  aacaa  tcG +ccgc  AA+ G Gc    gt \n",
      "  NZ_JACGUW010000053.1 349 TAGCTTCGTTCCTCGCCATTTTAGCCAACAATTTCGCGCCGCTTAATAGGGCGTTAGTT 407\n",
      "                           566667889999***************************************99877765 PP\n",
      "\n",
      ">> NZ_JACGUW010000014.1  Colwellia sp. MB02u-7 NODE_898706_length_212_cov_11.3, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    0.3   0.0     0.073       2.3     121     174 ..     155     208 ..     134     211 .. 0.87\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 0.3 bits;  conditional E-value: 0.073\n",
      "           tp_nucl_aln 121 ctggatgagatcatcgacacctatgagaacaagatcGgcccgcggAAcGGtGcc 174\n",
      "                           ct g t +g tc tcg +a +t  g  aacaa  tcG +ccgc  AA+ G Gc \n",
      "  NZ_JACGUW010000014.1 155 CTAGCTTCGTTCCTCGCAATTTTAGCCAACAATTTCGCGCCGCTTAATAGGGCG 208\n",
      "                           556666778899999999**************************9999888875 PP\n",
      "\n",
      ">> NZ_JACGUW010000060.1  Colwellia sp. MB02u-7 NODE_951092_length_515_cov_13.8, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.5   0.0      0.25       7.7     122     174 ..     110     162 ..      92     169 .. 0.89\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.5 bits;  conditional E-value: 0.25\n",
      "           tp_nucl_aln 122 tggatgagatcatcgacacctatgagaacaagatcGgcccgcggAAcGGtGcc 174\n",
      "                           t g t +g tc tcg ca +t  g  aacaa  tcG +ccgc  AA+ G Gc \n",
      "  NZ_JACGUW010000060.1 110 TAGCTTCGTTCCTCGCCATTTTAGCCAACAATTTCGCGCCGCTTAATAGGGCG 162\n",
      "                           5666677889999********************************99998885 PP\n",
      "\n",
      ">> NZ_JACGUW010000123.1  Colwellia sp. MB02u-7 NODE_964088_length_386_cov_13.2, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.6   0.0      0.26       8.2     122     174 ..     216     268 ..     197     275 .. 0.88\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.6 bits;  conditional E-value: 0.26\n",
      "           tp_nucl_aln 122 tggatgagatcatcgacacctatgagaacaagatcGgcccgcggAAcGGtGcc 174\n",
      "                           t g t +g tc tcg  a +t  g  aacaa  tcG +ccgc  AA+ G Gc \n",
      "  NZ_JACGUW010000123.1 216 TAGCTTCGTTCCTCGCTATTTTAGCCAACAATTTCGCGCCGCTTAATAGGGCG 268\n",
      "                           56666678899999999999***********************9999888875 PP\n",
      "\n",
      ">> NZ_JACGUW010000077.1  Colwellia sp. MB02u-7 NODE_956410_length_774_cov_19.5, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.3   0.0      0.45        14     122     174 ..      69     121 ..      51     128 .. 0.88\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.3 bits;  conditional E-value: 0.45\n",
      "           tp_nucl_aln 122 tggatgagatcatcgacacctatgagaacaagatcGgcccgcggAAcGGtGcc 174\n",
      "                           t g t +g tc tcg +a +t  g  aacaa  tcG +ccgc  AA+ G Gc \n",
      "  NZ_JACGUW010000077.1  69 TAGCTTCGTTCCTCGCAATTTTAGCCAACAATTTCGCGCCGCTTAATAGGGCG 121\n",
      "                           5666667889999999*****************************99998886 PP\n",
      "\n",
      ">> NZ_JACGUW010000034.1  Colwellia sp. MB02u-7 NODE_931926_length_796_cov_14.4, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.1   0.0      0.39        12     122     174 ..     553     605 ..     535     611 .. 0.88\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.1 bits;  conditional E-value: 0.39\n",
      "           tp_nucl_aln 122 tggatgagatcatcgacacctatgagaacaagatcGgcccgcggAAcGGtGcc 174\n",
      "                           t g t +g tc tcg +a +t  g  aacaa  tcG +ccgc  AA+ G Gc \n",
      "  NZ_JACGUW010000034.1 553 TAGCTTCGTTCCTCGCAATTTTAGCCAACAATTTCGCGCCGCTTAATAGGGCG 605\n",
      "                           56666678899999********************************9999886 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        219  (4507587 residues searched)\n",
      "Passed MSV filter:                         9  (0.0410959); expected 4.4 (0.02)\n",
      "Passed bias filter:                        9  (0.0410959); expected 4.4 (0.02)\n",
      "Passed Vit filter:                         9  (0.0410959); expected 4.4 (0.02)\n",
      "Passed Fwd filter:                         7  (0.0319635); expected 4.4 (0.02)\n",
      "Initial search space (Z):                219  [actual number of targets]\n",
      "Domain search space  (domZ):               7  [number of targets reported over threshold]\n",
      "# CPU time: 0.13u 0.01s 00:00:00.14 Elapsed: 00:00:00.07\n",
      "# Mc/sec: 38489.73\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006246515.1_ASM624651v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_006246515.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         26  (4414981 residues searched)\n",
      "Passed MSV filter:                         8  (0.307692); expected 0.5 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.5 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.5 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.5 (0.02)\n",
      "Initial search space (Z):                 26  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.23u 0.02s 00:00:00.25 Elapsed: 00:00:00.13\n",
      "# Mc/sec: 21064.29\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_900302475.1_D.aquaemixtaeCECT8626_Spades_Prokka_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_900302475.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         18  (4235016 residues searched)\n",
      "Passed MSV filter:                        10  (0.555556); expected 0.4 (0.02)\n",
      "Passed bias filter:                        1  (0.0555556); expected 0.4 (0.02)\n",
      "Passed Vit filter:                         1  (0.0555556); expected 0.4 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.4 (0.02)\n",
      "Initial search space (Z):                 18  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 3.67u 0.04s 00:00:03.71 Elapsed: 00:00:03.61\n",
      "# Mc/sec: 731.16\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_018531475.1_ASM1853147v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_018531475.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        245  (6949548 residues searched)\n",
      "Passed MSV filter:                       124  (0.506122); expected 4.9 (0.02)\n",
      "Passed bias filter:                        1  (0.00408163); expected 4.9 (0.02)\n",
      "Passed Vit filter:                         1  (0.00408163); expected 4.9 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 4.9 (0.02)\n",
      "Initial search space (Z):                245  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.42u 0.03s 00:00:00.44 Elapsed: 00:00:00.22\n",
      "# Mc/sec: 19172.75\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001746765.1_ASM174676v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001746765.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.34    3.3   0.2        1.7    1.0   0.1    2.1  2  NZ_MJBQ01000229.1  Klebsiella pneumoniae strain KP171 scaffol\n",
      "       0.41    3.0   0.0       0.72    2.2   0.0    1.2  1  NZ_MJBQ01000063.1  Klebsiella pneumoniae strain KP171 scaffol\n",
      "        0.7    2.3   0.0        1.2    1.5   0.0    1.2  1  NZ_MJBQ01000101.1  Klebsiella pneumoniae strain KP171 scaffol\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_MJBQ01000229.1  Klebsiella pneumoniae strain KP171 scaffold_59, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    1.0   0.1     0.018       1.7     276     318 ..   20195   20237 ..   20178   20241 .. 0.86\n",
      "   2 ?   -2.5   0.0      0.21        19     525     585 ..   24021   24078 ..   24004   24079 .. 0.82\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 1.0 bits;  conditional E-value: 0.018\n",
      "        tp_nucl_aln   276 gGAacAtAtggtgGcggtggagAAcacggccgaggtgcAcAAt 318  \n",
      "                          gGAa AtAtg   G gg gg +AAcacg c g ggtgcA AAt\n",
      "  NZ_MJBQ01000229.1 20195 GGAAAATATGAAAGAGGGGGTAAACACGTCAGCGGTGCAAAAT 20237\n",
      "                          68999**********************************9997 PP\n",
      "\n",
      "  == domain 2  score: -2.5 bits;  conditional E-value: 0.21\n",
      "        tp_nucl_aln   525 ccctgaagggAccGaaggtctGgacgaggaggcgctggccgcgctgGtcAcccgcgAcagc 585  \n",
      "                          ccc ga  g+A  Ga gg c G+     g  gc+ctgg +gc+ctgG cA  c cgA  +c\n",
      "  NZ_MJBQ01000229.1 24021 CCCGGACAGCAAAGACGGCCAGATG---GTCGCTCTGGTGGCTCTGGCCATTCACGATGCC 24078\n",
      "                          4778888999999999999998755...566789****************99999988766 PP\n",
      "\n",
      ">> NZ_MJBQ01000063.1  Klebsiella pneumoniae strain KP171 scaffold_154, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    2.2   0.0    0.0079      0.72     168     246 ..    2784    2862 ..    2768    2886 .. 0.88\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 2.2 bits;  conditional E-value: 0.0079\n",
      "        tp_nucl_aln  168 cGGtGcccacgtgGtgGcgcgggcctGGaccGAcccggagttccgggcgcggctgctggagGAtgccactgcgGcgatc 246 \n",
      "                         cGGtGc c  g gGtg  +   g ctGG cc A+ cgg g t+ + gc c+  t+ctgga GAtgcc++tg++G +atc\n",
      "  NZ_MJBQ01000063.1 2784 CGGTGCGCTTGAGGTGATCAATGACTGGTCCTATGCGGTGGTTGATGCTCCAGTACTGGACGATGCCGATGATGATATC 2862\n",
      "                         689999999999**********************************************************999876654 PP\n",
      "\n",
      ">> NZ_MJBQ01000101.1  Klebsiella pneumoniae strain KP171 scaffold_189, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    1.5   0.0     0.013       1.2      68     100 ..    3649    3681 ..    3615    3694 .. 0.84\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 1.5 bits;  conditional E-value: 0.013\n",
      "        tp_nucl_aln   68 tcagggcgctggAaacgcttctgattgaaAAgG 100 \n",
      "                         t +++gcgctggA a gct+ctg+ ++a+AAgG\n",
      "  NZ_MJBQ01000101.1 3649 TTCATGCGCTGGAGAAGCTGCTGGAGCAGAAGG 3681\n",
      "                         67899*************************997 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        273  (5517975 residues searched)\n",
      "Passed MSV filter:                        43  (0.157509); expected 5.5 (0.02)\n",
      "Passed bias filter:                        6  (0.021978); expected 5.5 (0.02)\n",
      "Passed Vit filter:                         6  (0.021978); expected 5.5 (0.02)\n",
      "Passed Fwd filter:                         3  (0.010989); expected 5.5 (0.02)\n",
      "Initial search space (Z):                273  [actual number of targets]\n",
      "Domain search space  (domZ):               3  [number of targets reported over threshold]\n",
      "# CPU time: 0.30u 0.01s 00:00:00.31 Elapsed: 00:00:00.18\n",
      "# Mc/sec: 18851.19\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_017948225.1_ASM1794822v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_017948225.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (3942273 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.10u 0.01s 00:00:00.11 Elapsed: 00:00:00.11\n",
      "# Mc/sec: 21830.30\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001015055.1_ASM101505v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001015055.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        136  (5693818 residues searched)\n",
      "Passed MSV filter:                         3  (0.0220588); expected 2.7 (0.02)\n",
      "Passed bias filter:                        3  (0.0220588); expected 2.7 (0.02)\n",
      "Passed Vit filter:                         3  (0.0220588); expected 2.7 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 2.7 (0.02)\n",
      "Initial search space (Z):                136  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.17u 0.01s 00:00:00.18 Elapsed: 00:00:00.11\n",
      "# Mc/sec: 30318.99\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004360145.1_ASM436014v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_004360145.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         33  (3133413 residues searched)\n",
      "Passed MSV filter:                        14  (0.424242); expected 0.7 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.7 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.7 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.7 (0.02)\n",
      "Initial search space (Z):                 33  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.14u 0.01s 00:00:00.15 Elapsed: 00:00:00.08\n",
      "# Mc/sec: 22301.35\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_009362315.1_ASM936231v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_009362315.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        217  (4292473 residues searched)\n",
      "Passed MSV filter:                       101  (0.465438); expected 4.3 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 4.3 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 4.3 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 4.3 (0.02)\n",
      "Initial search space (Z):                217  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.16u 0.01s 00:00:00.17 Elapsed: 00:00:00.09\n",
      "# Mc/sec: 29649.57\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_018500205.1_ASM1850020v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_018500205.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence             Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------             -----------\n",
      "  ------ inclusion threshold ------\n",
      "          2   -0.6   0.0        4.1   -1.6   0.0    1.2  1  NZ_JAHDAY010000074.1  Raoultella ornithinolytica strain LAM1 \n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_JAHDAY010000074.1  Raoultella ornithinolytica strain LAM1 NODE_74_length_4582_cov_81.9387, whole genome shotgun se\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.6   0.0      0.04       4.1     274     322 ..    3695    3743 ..    3682    3750 .. 0.89\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.6 bits;  conditional E-value: 0.04\n",
      "           tp_nucl_aln  274 gggGAacAtAtggtgGcggtggagAAcacggccgaggtgcAcAAtatgg 322 \n",
      "                            gg GAa AtAtg  gGc    ga AA+a ggcc a g gcA  At+tgg\n",
      "  NZ_JAHDAY010000074.1 3695 GGTGAATATATGAAGGCCAGAGATAATATGGCCTATGCGCAATATCTGG 3743\n",
      "                            899***********************************99987877776 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        104  (6013240 residues searched)\n",
      "Passed MSV filter:                        22  (0.211538); expected 2.1 (0.02)\n",
      "Passed bias filter:                        1  (0.00961538); expected 2.1 (0.02)\n",
      "Passed Vit filter:                         1  (0.00961538); expected 2.1 (0.02)\n",
      "Passed Fwd filter:                         1  (0.00961538); expected 2.1 (0.02)\n",
      "Initial search space (Z):                104  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.22u 0.02s 00:00:00.24 Elapsed: 00:00:00.12\n",
      "# Mc/sec: 29193.89\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000474745.1_Acht1_1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_000474745.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.13    3.3   0.0       0.36    1.8   0.0    1.6  2  NZ_ASJR01000002.1  Chitinivibrio alkaliphilus ACht1 contig000\n",
      "          2   -0.6   0.0        3.2   -1.3   0.0    1.2  1  NZ_ASJR01000096.1  Chitinivibrio alkaliphilus ACht1 contig000\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_ASJR01000002.1  Chitinivibrio alkaliphilus ACht1 contig00002, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    1.8   0.0    0.0072      0.36     240     310 ..   38464   38534 ..   38437   38540 .. 0.90\n",
      "   2 ?   -3.1   0.0      0.22        11     281     322 ..   62780   62833 ..   62780   62836 .. 0.71\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 1.8 bits;  conditional E-value: 0.0072\n",
      "        tp_nucl_aln   240 gGcgatcgccgagctgGggttttctggacgtcaggggGAacAtAtggtgGcggtggagAAcacggccgagg 310  \n",
      "                          gGc+  +gc ga  tgGg  +t  tg  cgtca gggGA c tAt    G ggtgg gAA+ac g++g+gg\n",
      "  NZ_ASJR01000002.1 38464 GGCTTCTGCAGAATTGGGAGATCTTGTTCGTCATGGGGAGCGTATCTATGAGGTGGTGAATACTGATGGGG 38534\n",
      "                          567778899******************************************************99888765 PP\n",
      "\n",
      "  == domain 2  score: -3.1 bits;  conditional E-value: 0.22\n",
      "        tp_nucl_aln   281 AtAtggtgGcggtggagAAcacggccgagg............tgcAcAAtatgg 322  \n",
      "                          AtAtggt+Gcggt+ga+AA+a g++ g+gg             gcA+AA+at +\n",
      "  NZ_ASJR01000002.1 62780 ATATGGTCGCGGTCGAAAATATGCAGGGGGatgcccagttccAGCATAACATAA 62833\n",
      "                          78*******************999988776222222222222467777777765 PP\n",
      "\n",
      ">> NZ_ASJR01000096.1  Chitinivibrio alkaliphilus ACht1 contig00096, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.3   0.0     0.064       3.2     228     263 ..     155     190 ..     124     237 .. 0.62\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.3 bits;  conditional E-value: 0.064\n",
      "        tp_nucl_aln 228 gGAtgccactgcgGcgatcgccgagctgGggttttc 263\n",
      "                         GAtg+c+ctgcgG    +gc gag+  Gg+t +t+\n",
      "  NZ_ASJR01000096.1 155 TGATGGCGCTGCGGAAGCTGCTGAGACAGGCTCCTA 190\n",
      "                        444444444444444444444444444444443333 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         99  (2591328 residues searched)\n",
      "Passed MSV filter:                         4  (0.040404); expected 2.0 (0.02)\n",
      "Passed bias filter:                        4  (0.040404); expected 2.0 (0.02)\n",
      "Passed Vit filter:                         4  (0.040404); expected 2.0 (0.02)\n",
      "Passed Fwd filter:                         2  (0.020202); expected 2.0 (0.02)\n",
      "Initial search space (Z):                 99  [actual number of targets]\n",
      "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
      "# CPU time: 0.82u 0.00s 00:00:00.82 Elapsed: 00:00:00.80\n",
      "# Mc/sec: 2012.60\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_900185035.1_R.arenilitoris_CECT8715_Spades_Prokka_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_900185035.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         11  (3871907 residues searched)\n",
      "Passed MSV filter:                         5  (0.454545); expected 0.2 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.2 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.2 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.2 (0.02)\n",
      "Initial search space (Z):                 11  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.17u 0.02s 00:00:00.19 Elapsed: 00:00:00.11\n",
      "# Mc/sec: 20619.81\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_012911815.1_ASM1291181v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_012911815.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence             Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------             -----------\n",
      "  ------ inclusion threshold ------\n",
      "      0.053    2.4   0.0      0.072    1.9   0.0    1.1  1  NZ_JAATNW010000022.1  Alteromonas ponticola strain MYP5 22, w\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_JAATNW010000022.1  Alteromonas ponticola strain MYP5 22, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    1.9   0.0    0.0033     0.072      67     110 ..     435     476 ..     371     483 .. 0.84\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 1.9 bits;  conditional E-value: 0.0033\n",
      "           tp_nucl_aln  67 gtcagggcgctggAaacgcttctgattgaaAAgGGgctgatcga 110\n",
      "                            t a+ gcgctg Aaa   t+ tgat+g  AAg  +c gat+ga\n",
      "  NZ_JAATNW010000022.1 435 ATTAAAGCGCTGAAAAGCGTGTTGATCGGCAAG--TCCGATTGA 476\n",
      "                           467899*************************95..777777666 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         22  (3602801 residues searched)\n",
      "Passed MSV filter:                         3  (0.136364); expected 0.4 (0.02)\n",
      "Passed bias filter:                        3  (0.136364); expected 0.4 (0.02)\n",
      "Passed Vit filter:                         3  (0.136364); expected 0.4 (0.02)\n",
      "Passed Fwd filter:                         1  (0.0454545); expected 0.4 (0.02)\n",
      "Initial search space (Z):                 22  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 1.22u 0.02s 00:00:01.24 Elapsed: 00:00:00.74\n",
      "# Mc/sec: 2997.95\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_009857605.1_ASM985760v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_009857605.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        104  (6742924 residues searched)\n",
      "Passed MSV filter:                        37  (0.355769); expected 2.1 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 2.1 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 2.1 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 2.1 (0.02)\n",
      "Initial search space (Z):                104  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.28u 0.01s 00:00:00.29 Elapsed: 00:00:00.14\n",
      "# Mc/sec: 28601.38\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006243325.1_ASM624332v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_006243325.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "      0.052    5.8   0.0      0.093    4.9   0.0    1.3  1  NZ_NKWI01000092.1  Aeromonas salmonicida strain CIP 107036 F3\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_NKWI01000092.1  Aeromonas salmonicida strain CIP 107036 F320_S7_contig_92, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    4.9   0.0   0.00041     0.093     266     309 ..    6608    6651 ..    6558    6659 .. 0.80\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 4.9 bits;  conditional E-value: 0.00041\n",
      "        tp_nucl_aln  266 gacgtcaggggGAacAtAtggtgGcggtggagAAcacggccgag 309 \n",
      "                         g c +c ggg GAacAtAtggtg +    ga AAcacgg  g+g\n",
      "  NZ_NKWI01000092.1 6608 GTCTACTGGGTGAACATATGGTGATTCCAGATAACACGGTAGGG 6651\n",
      "                         5677899******************************9977665 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        225  (4803087 residues searched)\n",
      "Passed MSV filter:                        27  (0.12); expected 4.5 (0.02)\n",
      "Passed bias filter:                        2  (0.00888889); expected 4.5 (0.02)\n",
      "Passed Vit filter:                         2  (0.00888889); expected 4.5 (0.02)\n",
      "Passed Fwd filter:                         1  (0.00444444); expected 4.5 (0.02)\n",
      "Initial search space (Z):                225  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.15u 0.02s 00:00:00.17 Elapsed: 00:00:00.09\n",
      "# Mc/sec: 30473.17\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_009377985.1_ASM937798v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_009377985.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "      0.044    3.0   0.0       0.12    1.6   0.0    1.6  1  NZ_RSAO01000014.1  Salinivibrio sp. VYel5 NODE_14_length_7350\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_RSAO01000014.1  Salinivibrio sp. VYel5 NODE_14_length_73507_cov_206.027623, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    1.6   0.0    0.0043      0.12     566     609 ..   51658   51701 ..   51635   51715 .. 0.77\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 1.6 bits;  conditional E-value: 0.0043\n",
      "        tp_nucl_aln   566 cgctgGtcAcccgcgAcagcAtGatcGGtacgGggctggcgcag 609  \n",
      "                          c+ tgGtcA +cgcgA a+ AtGatcG +acgG++  g +gca+\n",
      "  NZ_RSAO01000014.1 51658 CTTTGGTCATGCGCGAGACAATGATCGCCACGGCAAAGTTGCAC 51701\n",
      "                          56799***************************999877664444 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         28  (3301751 residues searched)\n",
      "Passed MSV filter:                         1  (0.0357143); expected 0.6 (0.02)\n",
      "Passed bias filter:                        1  (0.0357143); expected 0.6 (0.02)\n",
      "Passed Vit filter:                         1  (0.0357143); expected 0.6 (0.02)\n",
      "Passed Fwd filter:                         1  (0.0357143); expected 0.6 (0.02)\n",
      "Initial search space (Z):                 28  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.32u 0.01s 00:00:00.33 Elapsed: 00:00:00.27\n",
      "# Mc/sec: 7516.88\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003970385.1_ASM397038v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003970385.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "      0.082    5.0   0.0       0.15    4.2   0.0    1.3  1  NZ_RXZH01000087.1  Vibrio aquaticus strain BEI207 scaffold_91\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_RXZH01000087.1  Vibrio aquaticus strain BEI207 scaffold_91, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    4.2   0.0   0.00069      0.15      96     138 ..     240     282 ..     226     307 .. 0.79\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 4.2 bits;  conditional E-value: 0.00069\n",
      "        tp_nucl_aln  96 aAAgGGgctgatcgatcccgcagcgctggatgagatcatcgac 138\n",
      "                        +AA GGgc gat g tcc gc gc ctg+a g+g t    gac\n",
      "  NZ_RXZH01000087.1 240 GAACGGGCAGATAGGTCCGGCTGCTCTGAAGGCGGTGTATGAC 282\n",
      "                        58899****************************9887544443 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        215  (4451605 residues searched)\n",
      "Passed MSV filter:                         8  (0.0372093); expected 4.3 (0.02)\n",
      "Passed bias filter:                        2  (0.00930233); expected 4.3 (0.02)\n",
      "Passed Vit filter:                         2  (0.00930233); expected 4.3 (0.02)\n",
      "Passed Fwd filter:                         1  (0.00465116); expected 4.3 (0.02)\n",
      "Initial search space (Z):                215  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.10u 0.01s 00:00:00.11 Elapsed: 00:00:00.05\n",
      "# Mc/sec: 49409.63\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_005281475.1_ASM528147v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_005281475.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (6064527 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.12u 0.01s 00:00:00.13 Elapsed: 00:00:00.13\n",
      "# Mc/sec: 27872.99\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006381105.1_ASM638110v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_006381105.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         77  (4583336 residues searched)\n",
      "Passed MSV filter:                         1  (0.012987); expected 1.5 (0.02)\n",
      "Passed bias filter:                        1  (0.012987); expected 1.5 (0.02)\n",
      "Passed Vit filter:                         1  (0.012987); expected 1.5 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.5 (0.02)\n",
      "Initial search space (Z):                 77  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.16u 0.00s 00:00:00.16 Elapsed: 00:00:00.40\n",
      "# Mc/sec: 7126.60\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001999205.1_ASM199920v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001999205.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          2  (4051970 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  2  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.16u 0.01s 00:00:00.17 Elapsed: 00:00:01.66\n",
      "# Mc/sec: 1522.66\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000294775.2_ASM29477v2_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_000294775.2_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          2  (5848607 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  2  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.13u 0.02s 00:00:00.15 Elapsed: 00:00:00.15\n",
      "# Mc/sec: 23412.57\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001951155.1_ASM195115v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001951155.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (4007868 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.10u 0.01s 00:00:00.11 Elapsed: 00:00:00.11\n",
      "# Mc/sec: 22132.67\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_009789075.1_ASM978907v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_009789075.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         22  (3888276 residues searched)\n",
      "Passed MSV filter:                        11  (0.5); expected 0.4 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.4 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.4 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.4 (0.02)\n",
      "Initial search space (Z):                 22  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.18u 0.02s 00:00:00.20 Elapsed: 00:00:00.12\n",
      "# Mc/sec: 20196.25\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002573675.1_ASM257367v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_002573675.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         10  (6444826 residues searched)\n",
      "Passed MSV filter:                         8  (0.8); expected 0.2 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.2 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.2 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.2 (0.02)\n",
      "Initial search space (Z):                 10  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.34u 0.04s 00:00:00.38 Elapsed: 00:00:00.26\n",
      "# Mc/sec: 15044.14\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_013363245.1_ASM1336324v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_013363245.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         30  (2212041 residues searched)\n",
      "Passed MSV filter:                         1  (0.0333333); expected 0.6 (0.02)\n",
      "Passed bias filter:                        1  (0.0333333); expected 0.6 (0.02)\n",
      "Passed Vit filter:                         1  (0.0333333); expected 0.6 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.6 (0.02)\n",
      "Initial search space (Z):                 30  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.12u 0.01s 00:00:00.13 Elapsed: 00:00:00.57\n",
      "# Mc/sec: 2389.87\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002893095.1_ASM289309v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_002893095.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         36  (4428876 residues searched)\n",
      "Passed MSV filter:                        14  (0.388889); expected 0.7 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.7 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.7 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.7 (0.02)\n",
      "Initial search space (Z):                 36  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.25u 0.01s 00:00:00.26 Elapsed: 00:00:00.14\n",
      "# Mc/sec: 19034.44\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_013032145.1_ASM1303214v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_013032145.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "    1.1e-95  312.5   4.3    4.3e-95  310.6   4.3    2.0  1  NZ_WQBL01000006.1  Ruegeria arenilitoris strain HKCCD8755 NOD\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_WQBL01000006.1  Ruegeria arenilitoris strain HKCCD8755 NODE_6, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 !  310.6   4.3   1.3e-96   4.3e-95      34     604 ..  171829  172399 ..  171817  172422 .. 0.96\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 310.6 bits;  conditional E-value: 1.3e-96\n",
      "        tp_nucl_aln     34 cacgctcttttgccgccggaaactgctttgcgggtcagggcgctggAaacgcttctgattgaaAAgGGgctgatcgatcccgcagc 119   \n",
      "                           cac+ctc+t+tgcc  c ga +c gct+tgcgggt a+g c +t gA a  ct    +t gaa  gGG ct  tcg  cc g+a+ \n",
      "  NZ_WQBL01000006.1 171829 CACACTCATCTGCCATCCGATCCAGCTCTGCGGGTAAAGACATTAGAGAGCCTCGCAGTAGAACGGGGACTCGTCGCGCCGGAAAG 171914\n",
      "                           78899999*********************************************9999999999999******************** PP\n",
      "\n",
      "        tp_nucl_aln    120 gctggatgagatcatcgacacctatgagaacaagatcGgcccgcggAAcGGtGcccacgtgGtgGcgcgggcctGGaccGAcccgg 205   \n",
      "                             tgga+g+     tcga  c tat    a +a at Gg ccg  gA  GG Gccc+ gt GtgGc cg  c tGG +cGA+cc+g\n",
      "  NZ_WQBL01000006.1 171915 TTTGGACGCTTGGGTCGAGGCGTATTCCGAGCATATTGGACCGATGAGGGGCGCCCGGGTCGTGGCTCGATCGTGGCTCGATCCAG 172000\n",
      "                           ************************************************************************************** PP\n",
      "\n",
      "        tp_nucl_aln    206 agttccgggcgcggctgctggagGAtgccactgcgGcgatcgccgagctgGggttttctggacgtcaggggGAacAtAtggtgGcg 291   \n",
      "                           a tt+cg+g g   ct ct gag Atgc +c g+gGcgatc    ag t G  t+t + gg+     g   G acAt t   gGcg\n",
      "  NZ_WQBL01000006.1 172001 ACTTTCGCGGGAAACTTCTTGAGAATGCTGCCGAGGCGATCAAGCAGTTTGATTATCAAGGCGACGCGACAGCACATCTAAAGGCG 172086\n",
      "                           ******************************************999999**99999988888766555555555888888999**** PP\n",
      "\n",
      "        tp_nucl_aln    292 gtggagAAcacggccgaggtgcAcAAtatggtcgtctGcAccctgtGctcctGctAcccgtGgcctctgctgGGcctgccgccggc 377   \n",
      "                            t+gagAA+ac g+c     gcAcAA+atggtcgt tGcAc+ctgtGctc+tGctA+cc t   c+ t ctgGG +tg c ccggc\n",
      "  NZ_WQBL01000006.1 172087 ATCGAGAATACCGACACCCAGCACAACATGGTCGTGTGCACGCTGTGCTCTTGCTATCCCTTTTCAATCCTGGGGATGTCACCGGC 172172\n",
      "                           ************************************************************************************** PP\n",
      "\n",
      "        tp_nucl_aln    378 ctGGtAcAAatccgctgcctaccGcgcccGcgcgGtccgcGAaccgcgcagcGtgctggcgGAgtttGGcgtcaccctGcccgatg 463   \n",
      "                            tGGtAcAAatc  c g  taccGcgcccGcgc Gt cg GA cc cg+a+ Gtgct gc GAgtttGG gt  ++ct cc+ga g\n",
      "  NZ_WQBL01000006.1 172173 GTGGTACAAATCGACCGAATACCGCGCCCGCGCCGTTCGAGAGCCTCGAAAAGTGCTTGCAGAGTTTGGTGTGCATCTCCCTGAGG 172258\n",
      "                           ************************************************************************************** PP\n",
      "\n",
      "        tp_nucl_aln    464 acatggaggtgcgcGtctggGAttccAccGccgAggtgcGctatctgGtgatgcccgagcGccctgaagggAccGaaggtctGgac 549   \n",
      "                           ac t+g +gt+cg GtctggGA+++cAc GccgAg t+cGctat t Gt  tgcc+ a+cG cc+ ++gg AccGa gg + G+++\n",
      "  NZ_WQBL01000006.1 172259 ACGTCGGCGTCCGAGTCTGGGACAGCACGGCCGAGCTCCGCTATATCGTTGTGCCGCAACGGCCCTCGGGAACCGATGGGTGGAGT 172344\n",
      "                           ************************************************************************************** PP\n",
      "\n",
      "        tp_nucl_aln    550 gaggaggcgctggccgcgctgGtcAcccgcgAcagcAtGatcGGtacgGggctgg 604   \n",
      "                           gaggag   ctggc gc ct GtcAcccg  Ac++ AtGatcGG ac G gc  g\n",
      "  NZ_WQBL01000006.1 172345 GAGGAGCGCCTGGCAGCCCTTGTCACCCGAAACTCAATGATCGGGACAGAGCGAG 172399\n",
      "                           ********************************************99999777655 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         33  (4563248 residues searched)\n",
      "Passed MSV filter:                        12  (0.363636); expected 0.7 (0.02)\n",
      "Passed bias filter:                        1  (0.030303); expected 0.7 (0.02)\n",
      "Passed Vit filter:                         1  (0.030303); expected 0.7 (0.02)\n",
      "Passed Fwd filter:                         1  (0.030303); expected 0.7 (0.02)\n",
      "Initial search space (Z):                 33  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.75u 0.02s 00:00:00.77 Elapsed: 00:00:00.70\n",
      "# Mc/sec: 4017.68\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_014164675.1_ASM1416467v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_014164675.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        122  (3573456 residues searched)\n",
      "Passed MSV filter:                        48  (0.393443); expected 2.4 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 2.4 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 2.4 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 2.4 (0.02)\n",
      "Initial search space (Z):                122  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.20u 0.01s 00:00:00.21 Elapsed: 00:00:00.38\n",
      "# Mc/sec: 5851.88\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006375565.1_ASM637556v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_006375565.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "        2.3   -1.1   0.0          5   -2.2   0.0    1.2  1  NZ_NNOY01000030.1  Vibrio parahaemolyticus strain B2_7 B2_7_c\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_NNOY01000030.1  Vibrio parahaemolyticus strain B2_7 B2_7_contig030, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.2   0.0      0.06         5     108     153 ..   20053   20098 ..   20039   20102 .. 0.83\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.2 bits;  conditional E-value: 0.06\n",
      "        tp_nucl_aln   108 cgatcccgcagcgctggatgagatcatcgacacctatgagaacaag 153  \n",
      "                           gat ccgc+gc ct+gatg+g t atc a a ct +  +  caa \n",
      "  NZ_NNOY01000030.1 20053 AGATACCGCCGCACTCGATGCGTTGATCAATAGCTTCTCAGGCAAT 20098\n",
      "                          47899***********************999988887766666665 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         83  (4967259 residues searched)\n",
      "Passed MSV filter:                         3  (0.0361446); expected 1.7 (0.02)\n",
      "Passed bias filter:                        3  (0.0361446); expected 1.7 (0.02)\n",
      "Passed Vit filter:                         3  (0.0361446); expected 1.7 (0.02)\n",
      "Passed Fwd filter:                         1  (0.0120482); expected 1.7 (0.02)\n",
      "Initial search space (Z):                 83  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.35u 0.01s 00:00:00.36 Elapsed: 00:00:00.30\n",
      "# Mc/sec: 10136.96\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002882915.1_GW456-12-1-14-LB3_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_002882915.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.61    1.6   0.0        1.7    0.0   0.0    1.5  2  NZ_POEI01000135.1  Pseudomonas sp. GW456-12-1-14-LB3 NODE_136\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_POEI01000135.1  Pseudomonas sp. GW456-12-1-14-LB3 NODE_136_length_253_cov_0.829545, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    0.0   0.0     0.012       1.7     120     153 ..      31      64 ..      15      77 .. 0.84\n",
      "   2 ?   -2.6   0.0     0.076        11     439     462 ..     215     238 ..     205     247 .. 0.79\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 0.0 bits;  conditional E-value: 0.012\n",
      "        tp_nucl_aln 120 gctggatgagatcatcgacacctatgagaacaag 153\n",
      "                        gct g +g+ at atcgaca c at  g ac+ag\n",
      "  NZ_POEI01000135.1  31 GCTTGCCGCCATGATCGACAACCATCTGCACCAG 64 \n",
      "                        678899999************9999999998876 PP\n",
      "\n",
      "  == domain 2  score: -2.6 bits;  conditional E-value: 0.076\n",
      "        tp_nucl_aln 439 GAgtttGGcgtcaccctGcccgat 462\n",
      "                        GAg ttGGcgtca+cctG cc   \n",
      "  NZ_POEI01000135.1 215 GAGGTTGGCGTCATCCTGTCCTTG 238\n",
      "                        677899999999999999887555 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        143  (6603863 residues searched)\n",
      "Passed MSV filter:                        32  (0.223776); expected 2.9 (0.02)\n",
      "Passed bias filter:                        2  (0.013986); expected 2.9 (0.02)\n",
      "Passed Vit filter:                         2  (0.013986); expected 2.9 (0.02)\n",
      "Passed Fwd filter:                         1  (0.00699301); expected 2.9 (0.02)\n",
      "Initial search space (Z):                143  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.23u 0.01s 00:00:00.24 Elapsed: 00:00:00.12\n",
      "# Mc/sec: 33995.13\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001888265.1_ASM188826v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001888265.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         92  (3492240 residues searched)\n",
      "Passed MSV filter:                         2  (0.0217391); expected 1.8 (0.02)\n",
      "Passed bias filter:                        2  (0.0217391); expected 1.8 (0.02)\n",
      "Passed Vit filter:                         2  (0.0217391); expected 1.8 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.8 (0.02)\n",
      "Initial search space (Z):                 92  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.24u 0.02s 00:00:00.26 Elapsed: 00:00:00.31\n",
      "# Mc/sec: 7018.34\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006368485.1_ASM636848v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_006368485.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "        0.9    3.4   0.0        1.6    2.6   0.0    1.2  1  NZ_NNCX01000110.1  Vibrio parahaemolyticus strain G1_7 G1_7_c\n",
      "       0.93    3.4   0.0          2    2.3   0.0    1.4  1  NZ_NNCX01000259.1  Vibrio parahaemolyticus strain G1_7 G1_7_c\n",
      "        7.2    0.5   0.0         17   -0.7   0.0    1.3  1  NZ_NNCX01000380.1  Vibrio parahaemolyticus strain G1_7 G1_7_c\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_NNCX01000110.1  Vibrio parahaemolyticus strain G1_7 G1_7_contig110, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    2.6   0.0    0.0061       1.6      70     139 ..   11343   11412 ..   11321   11426 .. 0.85\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 2.6 bits;  conditional E-value: 0.0061\n",
      "        tp_nucl_aln    70 agggcgctggAaacgcttctgattgaaAAgGGgctgatcgatcccgcagcgctggatgagatcatcgaca 139  \n",
      "                          +++gc +tg Aa+ g tt tg t+ aa A+GG ctgat+gat  c ++g + t+ga+ga atc  cg ca\n",
      "  NZ_NNCX01000110.1 11343 CATGCTTTGCAAGAGTTTTTGCTCAAAGAAGGCCTGATTGATATCAACGGCATCGACGACATCTACGCCA 11412\n",
      "                          678999*************************************999999888999988887665444443 PP\n",
      "\n",
      ">> NZ_NNCX01000259.1  Vibrio parahaemolyticus strain G1_7 G1_7_contig259, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    2.3   0.0    0.0077         2     423     463 ..    6197    6237 ..    6179    6271 .. 0.84\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 2.3 bits;  conditional E-value: 0.0077\n",
      "        tp_nucl_aln  423 gcgcagcGtgctggcgGAgtttGGcgtcaccctGcccgatg 463 \n",
      "                         gc c+gcGt ct gc+GA+tt+GGcgt +++ tG  +g +g\n",
      "  NZ_NNCX01000259.1 6197 GCACGGCGTTCTTGCTGAATTCGGCGTAGAGATGATTGGCG 6237\n",
      "                         566899**********************9988775555555 PP\n",
      "\n",
      ">> NZ_NNCX01000380.1  Vibrio parahaemolyticus strain G1_7 G1_7_contig380, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.7   0.0     0.064        17     402     447 ..    2683    2731 ..    2681    2751 .. 0.80\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.7 bits;  conditional E-value: 0.064\n",
      "        tp_nucl_aln  402 cgcccGcgcgGtccgcGAaccgcgc...agcGtgctggcgGAgtttGGc 447 \n",
      "                          +c cG  cgGt cg+GAaccg      ++ G gctg +  A+tttGGc\n",
      "  NZ_NNCX01000380.1 2683 GTCGCGTTCGGTACGTGAACCGACGgatGATGAGCTGCAAAAATTTGGC 2731\n",
      "                         68999***************99764111456789999999999999997 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        788  (4956607 residues searched)\n",
      "Passed MSV filter:                        13  (0.0164975); expected 15.8 (0.02)\n",
      "Passed bias filter:                       13  (0.0164975); expected 15.8 (0.02)\n",
      "Passed Vit filter:                        13  (0.0164975); expected 15.8 (0.02)\n",
      "Passed Fwd filter:                         5  (0.00634518); expected 15.8 (0.02)\n",
      "Initial search space (Z):                788  [actual number of targets]\n",
      "Domain search space  (domZ):               3  [number of targets reported over threshold]\n",
      "# CPU time: 0.59u 0.01s 00:00:00.60 Elapsed: 00:00:00.30\n",
      "# Mc/sec: 10024.61\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004214755.1_ASM421475v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_004214755.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.29    2.1   0.0          2   -0.7   0.0    1.7  2  NZ_RBZJ01000037.1  Vibrio vulnificus strain VA-WGS-18039 NODE\n",
      "        1.3   -0.0   0.0        4.4   -1.8   0.0    1.5  1  NZ_RBZJ01000004.1  Vibrio vulnificus strain VA-WGS-18039 NODE\n",
      "        3.3   -1.4   0.0        5.7   -2.2   0.0    1.1  1  NZ_RBZJ01000084.1  Vibrio vulnificus strain VA-WGS-18039 NODE\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_RBZJ01000037.1  Vibrio vulnificus strain VA-WGS-18039 NODE_37_length_7341_cov_55.643471, whole genome shotgun sequ\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.0   0.0     0.076       2.5      48     101 ..    4092    4149 ..    4053    4180 .. 0.73\n",
      "   2 ?   -0.7   0.0     0.061         2      37     102 ..    4426    4488 ..    4398    4518 .. 0.75\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.0 bits;  conditional E-value: 0.076\n",
      "        tp_nucl_aln   48 gccggaaactgctttgcgggtcagggcgctg....gAaacgcttctgattgaaAAgGG 101 \n",
      "                            gga + +gc ttgc  g  ++ g  ctg    gAa+ gcttct a +gaaAAgG \n",
      "  NZ_RBZJ01000037.1 4092 AAGGGATCGCGCATTGCTTGCTCAAGTACTGtgctGAAGAGCTTCTCAACGAAAAGGT 4149\n",
      "                         334599999*****************9999777779**************99988763 PP\n",
      "\n",
      "  == domain 2  score: -0.7 bits;  conditional E-value: 0.061\n",
      "        tp_nucl_aln   37 gctcttttgccgccggaaactgctttgcgggtcagggcgctggAaacgcttctgattgaaAAgGGg 102 \n",
      "                         gc c+++tgcc+      a+tgct+tg   g   ++gc ctggAa+  cttc  a+ g +AA+G  \n",
      "  NZ_RBZJ01000037.1 4426 GCACAGCTGCCC---AGCATTGCTGTGGATGCTGATGCTCTGGAAGTTCTTCATACAGCGAAAGAC 4488\n",
      "                         444444444444...778999**********************************99999999843 PP\n",
      "\n",
      ">> NZ_RBZJ01000004.1  Vibrio vulnificus strain VA-WGS-18039 NODE_4_length_311869_cov_50.933349, whole genome shotgun seq\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.8   0.0      0.13       4.4     415     457 ..  209196  209238 ..  209185  209257 .. 0.79\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.8 bits;  conditional E-value: 0.13\n",
      "        tp_nucl_aln    415 cgcGAaccgcgcagcGtgctggcgGAgtttGGcgtcaccctGc 457   \n",
      "                           c +GAa  gc c+g Gtgct gcgGA+tt+GGcgt ++  tG \n",
      "  NZ_RBZJ01000004.1 209196 CTTGAAAAGCACGGTGTGCTAGCGGAATTCGGCGTAGAAATGA 209238\n",
      "                           5566666688899**********************99877665 PP\n",
      "\n",
      ">> NZ_RBZJ01000084.1  Vibrio vulnificus strain VA-WGS-18039 NODE_84_length_739_cov_1.050654, whole genome shotgun sequen\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.2   0.0      0.17       5.7      77     119 ..     661     703 ..     646     725 .. 0.85\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.2 bits;  conditional E-value: 0.17\n",
      "        tp_nucl_aln  77 tggAaacgcttctgattgaaAAgGGgctgatcgatcccgcagc 119\n",
      "                        t gA +cgct+ct at+g +AA+G g   at+ga c  g+ gc\n",
      "  NZ_RBZJ01000084.1 661 TTGATGCGCTGCTAATGGTGAAAGAGAAAATTGAGCAAGATGC 703\n",
      "                        66888999****************9999999999887777665 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         99  (4988278 residues searched)\n",
      "Passed MSV filter:                         3  (0.030303); expected 2.0 (0.02)\n",
      "Passed bias filter:                        3  (0.030303); expected 2.0 (0.02)\n",
      "Passed Vit filter:                         3  (0.030303); expected 2.0 (0.02)\n",
      "Passed Fwd filter:                         3  (0.030303); expected 2.0 (0.02)\n",
      "Initial search space (Z):                 99  [actual number of targets]\n",
      "Domain search space  (domZ):               3  [number of targets reported over threshold]\n",
      "# CPU time: 1.07u 0.03s 00:00:01.10 Elapsed: 00:00:01.00\n",
      "# Mc/sec: 3082.54\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001296255.1_ASM129625v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001296255.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         37  (5723552 residues searched)\n",
      "Passed MSV filter:                        20  (0.540541); expected 0.7 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.7 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.7 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.7 (0.02)\n",
      "Initial search space (Z):                 37  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.25u 0.02s 00:00:00.27 Elapsed: 00:00:00.15\n",
      "# Mc/sec: 22728.80\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001655635.1_ASM165563v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001655635.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          3  (5684538 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.1 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.1 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.1 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.1 (0.02)\n",
      "Initial search space (Z):                  3  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.13u 0.02s 00:00:00.15 Elapsed: 00:00:00.10\n",
      "# Mc/sec: 32934.67\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_902500085.1_BB1436_assembly_illumina_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_902500085.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence             Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------             -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.72    1.4   0.1       0.98    1.0   0.1    1.1  1  NZ_CABVOE010000143.1  Escherichia coli isolate BB1436, whole \n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_CABVOE010000143.1  Escherichia coli isolate BB1436, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    1.0   0.1    0.0064      0.98     340     377 ..     114     156 ..     110     165 .. 0.75\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 1.0 bits;  conditional E-value: 0.0064\n",
      "           tp_nucl_aln 340 tcctGctAcc......cgtGgcctctgctgGGcctgccgccggc 377\n",
      "                           tcctG+tAcc      c t gc t+tg t GGcctgccgcc gc\n",
      "  NZ_CABVOE010000143.1 114 TCCTGTTACCgttcgaCTT-GCATGTGTTAGGCCTGCCGCCAGC 156\n",
      "                           6788888888322222222.45678999*************986 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        152  (5273139 residues searched)\n",
      "Passed MSV filter:                        13  (0.0855263); expected 3.0 (0.02)\n",
      "Passed bias filter:                        1  (0.00657895); expected 3.0 (0.02)\n",
      "Passed Vit filter:                         1  (0.00657895); expected 3.0 (0.02)\n",
      "Passed Fwd filter:                         1  (0.00657895); expected 3.0 (0.02)\n",
      "Initial search space (Z):                152  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.14u 0.02s 00:00:00.16 Elapsed: 00:00:00.08\n",
      "# Mc/sec: 36920.97\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001720485.1_ASM172048v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001720485.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (4176400 residues searched)\n",
      "Passed MSV filter:                         1  (1); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.24u 0.03s 00:00:00.27 Elapsed: 00:00:00.93\n",
      "# Mc/sec: 2778.55\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003348465.1_ASM334846v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003348465.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.21    2.1   0.0       0.33    1.4   0.0    1.2  1  NZ_QEEO01000040.1  Vibrio cholerae strain BD05 BD5_S5_L001_R1\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_QEEO01000040.1  Vibrio cholerae strain BD05 BD5_S5_L001_R1_001contig_40_consensus, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    1.4   0.0    0.0047      0.33     260     300 ..     822     862 ..     788     875 .. 0.83\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 1.4 bits;  conditional E-value: 0.0047\n",
      "        tp_nucl_aln 260 tttctggacgtcaggggGAacAtAtggtgGcggtggagAAc 300\n",
      "                        t t+ gga  t   g gGAacAtAt  tgGcg tgga AA+\n",
      "  NZ_QEEO01000040.1 822 TATAAGGAGTTTTAGTGGAACATATTTTGGCGCTGGACAAT 862\n",
      "                        44556667777778899********************9996 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         69  (4040187 residues searched)\n",
      "Passed MSV filter:                         3  (0.0434783); expected 1.4 (0.02)\n",
      "Passed bias filter:                        3  (0.0434783); expected 1.4 (0.02)\n",
      "Passed Vit filter:                         3  (0.0434783); expected 1.4 (0.02)\n",
      "Passed Fwd filter:                         1  (0.0144928); expected 1.4 (0.02)\n",
      "Initial search space (Z):                 69  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.78u 0.01s 00:00:00.79 Elapsed: 00:00:00.73\n",
      "# Mc/sec: 3430.81\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006374735.1_ASM637473v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_006374735.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.85    3.0   0.0        1.4    2.3   0.0    1.3  1  NZ_NNNL01000277.1  Vibrio parahaemolyticus strain B6_8 B6_8_c\n",
      "        2.2    1.6   0.0         22   -1.7   0.0    1.9  2  NZ_NNNL01000075.1  Vibrio parahaemolyticus strain B6_8 B6_8_c\n",
      "        6.2    0.1   0.0         14   -1.1   0.0    1.3  1  NZ_NNNL01000020.1  Vibrio parahaemolyticus strain B6_8 B6_8_c\n",
      "        7.4   -0.1   0.0         11   -0.7   0.0    1.1  1  NZ_NNNL01000424.1  Vibrio parahaemolyticus strain B6_8 B6_8_c\n",
      "        8.5   -0.3   0.0         13   -1.0   0.0    1.1  1  NZ_NNNL01000423.1  Vibrio parahaemolyticus strain B6_8 B6_8_c\n",
      "        9.5   -0.5   0.0         22   -1.7   0.0    1.3  1  NZ_NNNL01000216.1  Vibrio parahaemolyticus strain B6_8 B6_8_c\n",
      "        9.9   -0.5   0.0         20   -1.5   0.0    1.2  1  NZ_NNNL01000146.1  Vibrio parahaemolyticus strain B6_8 B6_8_c\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_NNNL01000277.1  Vibrio parahaemolyticus strain B6_8 B6_8_contig277, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    2.3   0.0     0.018       1.4       7      42 ..    3184    3225 ..    3178    3270 .. 0.69\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 2.3 bits;  conditional E-value: 0.018\n",
      "        tp_nucl_aln    7 cAtGatca......tgccgaccacgaccacccgcacgctctt 42  \n",
      "                         cAtGatca      t  +ga+cacgaccaccc ca+g+tc+t\n",
      "  NZ_NNNL01000277.1 3184 CATGATCAccatcaTCATGATCACGACCACCCACATGATCAT 3225\n",
      "                         666766550000005566777888888888888888777665 PP\n",
      "\n",
      ">> NZ_NNNL01000075.1  Vibrio parahaemolyticus strain B6_8 B6_8_contig075, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.0   0.0      0.36        27      78     132 ..    4721    4777 ..    4692    4791 .. 0.83\n",
      "   2 ?   -1.7   0.0      0.28        22     182     243 ..   15631   15692 ..   15622   15734 .. 0.72\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.0 bits;  conditional E-value: 0.36\n",
      "        tp_nucl_aln   78 ggAaacgcttctgattgaaAAgGGgctgatcgatc..ccgcagcgctggatgagatc 132 \n",
      "                         ggAaa    t tgat+g aAA GG  t at+gatc  c  +a+cgc  gatga+at+\n",
      "  NZ_NNNL01000075.1 4721 GGAAAGTGCTTTGATCGGAAATGGCATCATTGATCatCGCAAACGCCTGATGAAATT 4777\n",
      "                         57888888899**********************984457788999999999999875 PP\n",
      "\n",
      "  == domain 2  score: -1.7 bits;  conditional E-value: 0.28\n",
      "        tp_nucl_aln   182 tgGcgcgggcctGGaccGAcccggagttccgggcgcggctgctggagGAtgccactgcgGcg 243  \n",
      "                          t Gc cg g +tGG   GA+ c+gagtt+    c++  c g t g+ GA+g+ +  g+ G  \n",
      "  NZ_NNNL01000075.1 15631 TTGCTCGCGTATGGCATGATGCCGAGTTTGTTACAGTTCCGGTCGCAGACGGTGGCGAAGGC 15692\n",
      "                          6899**********************************************998877765533 PP\n",
      "\n",
      ">> NZ_NNNL01000020.1  Vibrio parahaemolyticus strain B6_8 B6_8_contig020, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.1   0.0      0.18        14     423     457 ..   72718   72752 ..   72699   72788 .. 0.84\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.1 bits;  conditional E-value: 0.18\n",
      "        tp_nucl_aln   423 gcgcagcGtgctggcgGAgtttGGcgtcaccctGc 457  \n",
      "                          gc c+gcGt ct gc+GA+tt+GGcgt +++ tG \n",
      "  NZ_NNNL01000020.1 72718 GCACGGCGTTCTTGCTGAATTCGGCGTAGAGATGA 72752\n",
      "                          566899*********************99987764 PP\n",
      "\n",
      ">> NZ_NNNL01000424.1  Vibrio parahaemolyticus strain B6_8 B6_8_contig424, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.7   0.0      0.15        11      22      87 ..     181     246 ..     168     257 .. 0.88\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.7 bits;  conditional E-value: 0.15\n",
      "        tp_nucl_aln  22 cacgaccacccgcacgctcttttgccgccggaaactgctttgcgggtcagggcgctggAaacgctt 87 \n",
      "                         acgac ac+c  +c++tc+ttt c +   ga++ t c tt   gg  ag gc  tggAa+c ctt\n",
      "  NZ_NNNL01000424.1 181 TACGACTACGCCATCAATCATTTACTCAGTGAGCATCCGTTCAAGGAAAGCGCAGTGGAAGCCCTT 246\n",
      "                        689999999999999999***999999999******************************999886 PP\n",
      "\n",
      ">> NZ_NNNL01000423.1  Vibrio parahaemolyticus strain B6_8 B6_8_contig423, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.0   0.0      0.18        13     568     589 ..      67      88 ..      57     120 .. 0.86\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.0 bits;  conditional E-value: 0.18\n",
      "        tp_nucl_aln 568 ctgGtcAcccgcgAcagcAtGa 589\n",
      "                         tgGt Ac cg+gA agcAtGa\n",
      "  NZ_NNNL01000423.1  67 ATGGTAACTCGTGAAAGCATGA 88 \n",
      "                        589*****************98 PP\n",
      "\n",
      ">> NZ_NNNL01000216.1  Vibrio parahaemolyticus strain B6_8 B6_8_contig216, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.7   0.0      0.29        22      97     143 ..    1575    1612 ..    1562    1624 .. 0.82\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.7 bits;  conditional E-value: 0.29\n",
      "        tp_nucl_aln   97 AAgGGgctgatcgatcccgcagcgctggatgagatcatcgacaccta 143 \n",
      "                         AA          g + ccg++gcgc g+a+g+ at+ t gacac+ a\n",
      "  NZ_NNNL01000216.1 1575 AAC---------GGCACCGACGCGCCGAACGCTATTTTAGACACTCA 1612\n",
      "                         444.........45899*************************99876 PP\n",
      "\n",
      ">> NZ_NNNL01000146.1  Vibrio parahaemolyticus strain B6_8 B6_8_contig146, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.5   0.0      0.26        20     520     561 ..   15670   15711 ..   15667   15718 .. 0.87\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.5 bits;  conditional E-value: 0.26\n",
      "        tp_nucl_aln   520 gagcGccctgaagggAccGaaggtctGgacgaggaggcgctg 561  \n",
      "                          g++cG cc gaagg+A cG  gg + Ggacga ga   ++tg\n",
      "  NZ_NNNL01000146.1 15670 GTACGTCCAGAAGGCAGCGTCGGCAAGGACGATGATATTTTG 15711\n",
      "                          5789******************************98766554 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        533  (5150638 residues searched)\n",
      "Passed MSV filter:                        14  (0.0262664); expected 10.7 (0.02)\n",
      "Passed bias filter:                       14  (0.0262664); expected 10.7 (0.02)\n",
      "Passed Vit filter:                        14  (0.0262664); expected 10.7 (0.02)\n",
      "Passed Fwd filter:                         7  (0.0131332); expected 10.7 (0.02)\n",
      "Initial search space (Z):                533  [actual number of targets]\n",
      "Domain search space  (domZ):               7  [number of targets reported over threshold]\n",
      "# CPU time: 0.94u 0.03s 00:00:00.97 Elapsed: 00:00:00.51\n",
      "# Mc/sec: 6259.55\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_007859875.1_ASM785987v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_007859875.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          6  (7137949 residues searched)\n",
      "Passed MSV filter:                         2  (0.333333); expected 0.1 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.1 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.1 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.1 (0.02)\n",
      "Initial search space (Z):                  6  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.29u 0.04s 00:00:00.32 Elapsed: 00:00:00.32\n",
      "# Mc/sec: 13773.03\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_018122435.1_ASM1812243v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_018122435.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         61  (4099751 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 1.2 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 1.2 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 1.2 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.2 (0.02)\n",
      "Initial search space (Z):                 61  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.10u 0.00s 00:00:00.10 Elapsed: 00:00:00.05\n",
      "# Mc/sec: 49333.05\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003198945.1_ASM319894v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003198945.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         43  (4654393 residues searched)\n",
      "Passed MSV filter:                         6  (0.139535); expected 0.9 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.9 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.9 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.9 (0.02)\n",
      "Initial search space (Z):                 43  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.11u 0.01s 00:00:00.12 Elapsed: 00:00:00.07\n",
      "# Mc/sec: 39190.49\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004022565.1_ASM402256v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_004022565.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (3548703 residues searched)\n",
      "Passed MSV filter:                         1  (1); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.16u 0.03s 00:00:00.19 Elapsed: 00:00:00.18\n",
      "# Mc/sec: 12266.14\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002127625.1_ASM212762v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_002127625.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        106  (6265111 residues searched)\n",
      "Passed MSV filter:                        35  (0.330189); expected 2.1 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 2.1 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 2.1 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 2.1 (0.02)\n",
      "Initial search space (Z):                106  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.18u 0.01s 00:00:00.19 Elapsed: 00:00:00.09\n",
      "# Mc/sec: 40100.91\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_016813735.1_ASM1681373v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_016813735.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         24  (3983916 residues searched)\n",
      "Passed MSV filter:                         6  (0.25); expected 0.5 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.5 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.5 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.5 (0.02)\n",
      "Initial search space (Z):                 24  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.15u 0.01s 00:00:00.16 Elapsed: 00:00:00.10\n",
      "# Mc/sec: 24651.70\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_009904695.1_ASM990469v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_009904695.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.71    2.5   0.0        1.2    1.8   0.0    1.2  1  NZ_WXWQ01000298.1  Vibrio sp. V40_P2S30T141 scaffold55_size17\n",
      "        3.2    0.4   0.0        5.4   -0.4   0.0    1.2  1  NZ_WXWQ01000287.1  Vibrio sp. V40_P2S30T141 scaffold176_size6\n",
      "        3.7    0.2   0.0        7.7   -0.9   0.0    1.3  1  NZ_WXWQ01000124.1  Vibrio sp. V40_P2S30T141 scaffold269_size3\n",
      "        5.5   -0.4   0.0         10   -1.3   0.0    1.1  1  NZ_WXWQ01000125.1  Vibrio sp. V40_P2S30T141 scaffold153_size8\n",
      "        6.1   -0.6   0.2         16   -2.0   0.1    1.3  2  NZ_WXWQ01000197.1  Vibrio sp. V40_P2S30T141 scaffold307_size2\n",
      "        8.1   -1.0   0.0         22   -2.4   0.0    1.3  1  NZ_WXWQ01000021.1  Vibrio sp. V40_P2S30T141 scaffold111_size1\n",
      "        9.2   -1.2   0.0         20   -2.2   0.0    1.2  1  NZ_WXWQ01000292.1  Vibrio sp. V40_P2S30T141 scaffold129_size9\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_WXWQ01000298.1  Vibrio sp. V40_P2S30T141 scaffold55_size17730, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    1.8   0.0     0.026       1.2      92     143 ..   10440   10483 ..   10364   10494 .. 0.77\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 1.8 bits;  conditional E-value: 0.026\n",
      "        tp_nucl_aln    92 ttgaaAAgGGgctgatcgatcccgcagcgctggatgagatcatcgacaccta 143  \n",
      "                          tt+a AAgGG        atccc +agcg tg+atgag t+ tcgacacc a\n",
      "  NZ_WXWQ01000298.1 10440 TTCACAAGGG--------ATCCCTAAGCGTTGAATGAGTTTTTCGACACCAA 10483\n",
      "                          5678899998........9****************************99865 PP\n",
      "\n",
      ">> NZ_WXWQ01000287.1  Vibrio sp. V40_P2S30T141 scaffold176_size6433, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.4   0.0      0.12       5.4     275     311 ..    3813    3849 ..    3806    3865 .. 0.87\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.4 bits;  conditional E-value: 0.12\n",
      "        tp_nucl_aln  275 ggGAacAtAtggtgGcggtggagAAcacggccgaggt 311 \n",
      "                         g+ AacA+Atg t Gcg t  agAAcac g+c a  t\n",
      "  NZ_WXWQ01000287.1 3813 GCCAACACATGATTGCGATTCAGAACACCGACAAATT 3849\n",
      "                         5559************************998876555 PP\n",
      "\n",
      ">> NZ_WXWQ01000124.1  Vibrio sp. V40_P2S30T141 scaffold269_size3287, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.9   0.0      0.17       7.7     576     594 ..    3034    3052 ..    2998    3070 .. 0.87\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.9 bits;  conditional E-value: 0.17\n",
      "        tp_nucl_aln  576 ccgcgAcagcAtGatcGGt 594 \n",
      "                         +cgcgAcagcAtGatcGG+\n",
      "  NZ_WXWQ01000124.1 3034 GCGCGACAGCATGATCGGC 3052\n",
      "                         589**************97 PP\n",
      "\n",
      ">> NZ_WXWQ01000125.1  Vibrio sp. V40_P2S30T141 scaffold153_size8158, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.3   0.0      0.22        10     421     452 ..    2052    2083 ..    2043    2101 .. 0.89\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.3 bits;  conditional E-value: 0.22\n",
      "        tp_nucl_aln  421 ccgcgcagcGtgctggcgGAgtttGGcgtcac 452 \n",
      "                         ccgc  a+ Gt ctg  +GAgtttGG gtc++\n",
      "  NZ_WXWQ01000125.1 2052 CCGCTTAATGTACTGATTGAGTTTGGTGTCGA 2083\n",
      "                         78888899********************9875 PP\n",
      "\n",
      ">> NZ_WXWQ01000197.1  Vibrio sp. V40_P2S30T141 scaffold307_size2420, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.0   0.1      0.36        16       7      51 ..     333     377 ..     326     391 .. 0.82\n",
      "   2 ?   -4.4   0.0       1.9        88       5      49 ..     712     756 ..     710     763 .. 0.84\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.0 bits;  conditional E-value: 0.36\n",
      "        tp_nucl_aln   7 cAtGatcatgccgaccacgaccacccgcacgctcttttgccgccg 51 \n",
      "                        cA Ga c +  cg++ ac accacc++cacg+t + +t c g + \n",
      "  NZ_WXWQ01000197.1 333 CACGAGCGCCACGGTAACCACCACCATCACGATTACCTTCTGGAC 377\n",
      "                        8999*************************9999888777766444 PP\n",
      "\n",
      "  == domain 2  score: -4.4 bits;  conditional E-value: 1.9\n",
      "        tp_nucl_aln   5 cccAtGatcatgccgaccacgaccacccgcacgctcttttgccgc 49 \n",
      "                        c+cA G   at  cgac acga   + ++cacgctc+ t gc+gc\n",
      "  NZ_WXWQ01000197.1 712 CTCACGGCGATCACGACGACGAGACTTATCACGCTCAATCGCAGC 756\n",
      "                        667888888888889999999999999999999999999998876 PP\n",
      "\n",
      ">> NZ_WXWQ01000021.1  Vibrio sp. V40_P2S30T141 scaffold111_size11003, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.4   0.0      0.47        22     207     269 ..    2452    2514 ..    2446    2525 .. 0.88\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.4 bits;  conditional E-value: 0.47\n",
      "        tp_nucl_aln  207 gttccgggcgcggctgctggagGAtgccactgcgGcgatcgccgagctgGggttttctggacg 269 \n",
      "                         gt +c+ gc + g tg tgg gGAtg c c  ++Gcg t g c   +tgGg+ + ++tgg+cg\n",
      "  NZ_WXWQ01000021.1 2452 GTATCATGCTTTGGTGATGGTGGATGACTCCCATGCGGTGGGCTTTATGGGCGAAAATGGCCG 2514\n",
      "                         567788888899999************************************998888888765 PP\n",
      "\n",
      ">> NZ_WXWQ01000292.1  Vibrio sp. V40_P2S30T141 scaffold129_size9615, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.2   0.0      0.42        20     569     606 ..    6014    6051 ..    6008    6068 .. 0.85\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.2 bits;  conditional E-value: 0.42\n",
      "        tp_nucl_aln  569 tgGtcAcccgcgAcagcAtGatcGGtacgGggctggcg 606 \n",
      "                         tgGt Accc cg c+gcA Gat GGt+  Gggct  cg\n",
      "  NZ_WXWQ01000292.1 6014 TGGTTACCCTCGGCTGCACGATAGGTGTTGGGCTCTCG 6051\n",
      "                         79****************************99987664 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        323  (3492478 residues searched)\n",
      "Passed MSV filter:                        13  (0.0402477); expected 6.5 (0.02)\n",
      "Passed bias filter:                       13  (0.0402477); expected 6.5 (0.02)\n",
      "Passed Vit filter:                        13  (0.0402477); expected 6.5 (0.02)\n",
      "Passed Fwd filter:                         7  (0.0216718); expected 6.5 (0.02)\n",
      "Initial search space (Z):                323  [actual number of targets]\n",
      "Domain search space  (domZ):               7  [number of targets reported over threshold]\n",
      "# CPU time: 0.40u 0.01s 00:00:00.41 Elapsed: 00:00:00.25\n",
      "# Mc/sec: 8593.43\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_016812975.1_ASM1681297v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_016812975.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         39  (4652699 residues searched)\n",
      "Passed MSV filter:                        11  (0.282051); expected 0.8 (0.02)\n",
      "Passed bias filter:                        1  (0.025641); expected 0.8 (0.02)\n",
      "Passed Vit filter:                         1  (0.025641); expected 0.8 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.8 (0.02)\n",
      "Initial search space (Z):                 39  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 1.02u 0.04s 00:00:01.06 Elapsed: 00:00:00.93\n",
      "# Mc/sec: 3113.75\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000968265.1_CLOSIBUN125Cv1.0_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_000968265.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         71  (4596888 residues searched)\n",
      "Passed MSV filter:                         1  (0.0140845); expected 1.4 (0.02)\n",
      "Passed bias filter:                        1  (0.0140845); expected 1.4 (0.02)\n",
      "Passed Vit filter:                         1  (0.0140845); expected 1.4 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.4 (0.02)\n",
      "Initial search space (Z):                 71  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.10u 0.00s 00:00:00.10 Elapsed: 00:00:00.05\n",
      "# Mc/sec: 55730.25\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_017351805.1_ASM1735180v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_017351805.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence             Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------             -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.36    2.5   0.0       0.64    1.6   0.0    1.4  1  NZ_JACNDX010000144.1  Pseudomonas sp. FW215-T2 SRX4123280_con\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_JACNDX010000144.1  Pseudomonas sp. FW215-T2 SRX4123280_contig_144, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    1.6   0.0    0.0041      0.64      71     102 ..     222     253 ..     145     273 .. 0.83\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 1.6 bits;  conditional E-value: 0.0041\n",
      "           tp_nucl_aln  71 gggcgctggAaacgcttctgattgaaAAgGGg 102\n",
      "                            ggc + ggAaac ct+ctga+++  AA GG \n",
      "  NZ_JACNDX010000144.1 222 CGGCTTCGGAAACTCTGCTGACCCTCAACGGC 253\n",
      "                           45788899***************999999885 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        156  (6380199 residues searched)\n",
      "Passed MSV filter:                        41  (0.262821); expected 3.1 (0.02)\n",
      "Passed bias filter:                        1  (0.00641026); expected 3.1 (0.02)\n",
      "Passed Vit filter:                         1  (0.00641026); expected 3.1 (0.02)\n",
      "Passed Fwd filter:                         1  (0.00641026); expected 3.1 (0.02)\n",
      "Initial search space (Z):                156  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.20u 0.01s 00:00:00.21 Elapsed: 00:00:00.10\n",
      "# Mc/sec: 37822.81\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001611905.1_ASM161190v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001611905.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (6520772 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.13u 0.00s 00:00:00.13 Elapsed: 00:00:00.14\n",
      "# Mc/sec: 28260.19\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_014764705.1_ASM1476470v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_014764705.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        113  (4861582 residues searched)\n",
      "Passed MSV filter:                        28  (0.247788); expected 2.3 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 2.3 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 2.3 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 2.3 (0.02)\n",
      "Initial search space (Z):                113  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.14u 0.01s 00:00:00.15 Elapsed: 00:00:00.07\n",
      "# Mc/sec: 39481.91\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001593605.1_ASM159360v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001593605.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (3882610 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.09u 0.00s 00:00:00.09 Elapsed: 00:00:00.08\n",
      "# Mc/sec: 27312.48\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_008386235.1_ASM838623v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_008386235.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (2647823 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.06u 0.01s 00:00:00.06 Elapsed: 00:00:00.06\n",
      "# Mc/sec: 26334.27\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_011393335.1_ASM1139333v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_011393335.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (3671901 residues searched)\n",
      "Passed MSV filter:                         1  (1); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.17u 0.03s 00:00:00.20 Elapsed: 00:00:00.20\n",
      "# Mc/sec: 11318.16\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_014268065.1_ASM1426806v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_014268065.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence             Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------             -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.22    3.6   0.0        1.1    1.2   0.0    1.9  2  NZ_JACNYN010000153.1  Escherichia coli strain UPMP2130 contig\n",
      "        1.6    0.8   0.1        3.9   -0.5   0.1    1.4  1  NZ_JACNYN010000071.1  Escherichia coli strain UPMP2130 contig\n",
      "        2.1    0.4   0.0        4.7   -0.8   0.0    1.3  1  NZ_JACNYN010000098.1  Escherichia coli strain UPMP2130 contig\n",
      "        2.2    0.3   0.0        4.4   -0.7   0.0    1.3  1  NZ_JACNYN010000214.1  Escherichia coli strain UPMP2130 contig\n",
      "        3.9   -0.5   0.0         12   -2.2   0.0    1.3  1  NZ_JACNYN010000004.1  Escherichia coli strain UPMP2130 contig\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_JACNYN010000153.1  Escherichia coli strain UPMP2130 contig00416, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.4   0.0      0.17       7.4      73     142 ..     550     618 ..     525     626 .. 0.80\n",
      "   2 ?    1.2   0.0     0.026       1.1      87     149 ..    2955    3028 ..    2940    3036 .. 0.67\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.4 bits;  conditional E-value: 0.17\n",
      "           tp_nucl_aln  73 gcgctggAaacgcttctgattgaaAAgGGgctgatcgatcccgcagcgctggatgagatcatcgacacct 142\n",
      "                            c ctgg aacg ttctg+ttga AAgG +   a cg    c + gcg+ gga  a  tc tcga  c t\n",
      "  NZ_JACNYN010000153.1 550 TCACTGGTAACG-TTCTGGTTGATAAGGATAAAACCGCAGACAATGCGGCGGAATATTTCTTCGATCCAT 618\n",
      "                           588888888876.789*************99999999999999**********99999999999976655 PP\n",
      "\n",
      "  == domain 2  score: 1.2 bits;  conditional E-value: 0.026\n",
      "           tp_nucl_aln   87 tctgattgaaAAgGGgctgatcgat...............cccgcagcgctggatgagatcatcgacacctatgagaa 149 \n",
      "                             ct    ga A gGG+ tgatcg t                 cgc++cg+ ggatg+g t atcga+a c a+g  a \n",
      "  NZ_JACNYN010000153.1 2955 ACT----GACACGGGTATGATCGGTatgcaactcgatgccAACGCCACGGCGGATGCGGTAATCGAAAACAACGGCAC 3028\n",
      "                            444....44556788888888765411111111111111145789999*******************99988876655 PP\n",
      "\n",
      ">> NZ_JACNYN010000071.1  Escherichia coli strain UPMP2130 contig00150, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.5   0.1     0.091       3.9     216     297 ..    8712    8814 ..    8697    8823 .. 0.62\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.5 bits;  conditional E-value: 0.091\n",
      "           tp_nucl_aln  216 gcggctgctggagGAtgccactgcgGcgatcgccgagctgGggttt....................tctggacgtcag......ggg 276 \n",
      "                            gcg +t  t ga GA g+c+c  +gG   t+gcc  gctgG   t                     t    +cgtcag      gg+\n",
      "  NZ_JACNYN010000071.1 8712 GCGATTTATCGAAGAAGGCGCGCAGG---TTGCCACGCTGGAACTGtcggcggcaaaagtcgccagT--CTGCGTCAGcgatttGGC 8793\n",
      "                            55566666666666666666666655...45666666666666554223333332222222222111..11233444455555589* PP\n",
      "\n",
      "           tp_nucl_aln  277 GAacAtAtggtgGcggtggag 297 \n",
      "                            GAacAtAt  tgGcggtgga+\n",
      "  NZ_JACNYN010000071.1 8794 GAACATATTCTGGCGGTGGAA 8814\n",
      "                            ******************985 PP\n",
      "\n",
      ">> NZ_JACNYN010000098.1  Escherichia coli strain UPMP2130 contig00210, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.8   0.0      0.11       4.7     269     321 ..    6205    6252 ..    6196    6258 .. 0.86\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.8 bits;  conditional E-value: 0.11\n",
      "           tp_nucl_aln  269 gtcaggggGAacAtAtggtgGcggtggagAAcacggccgaggtgcAcAAtatg 321 \n",
      "                            gtcag g     AtAt gtgGcg tgga+AAc c + +ga  tg AcAA +t \n",
      "  NZ_JACNYN010000098.1 6205 GTCAGCG-----ATATTGTGGCGCTGGAAAACGCCCTTGATATGTACAAACTC 6252\n",
      "                            5555554.....899********************************998776 PP\n",
      "\n",
      ">> NZ_JACNYN010000214.1  Escherichia coli strain UPMP2130 contig00729, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.7   0.0       0.1       4.4     212     273 ..      73     134 ..      66     147 .. 0.82\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.7 bits;  conditional E-value: 0.1\n",
      "           tp_nucl_aln 212 gggcgcggctgctggagGAtgccactgcgGcgatcgccgagctgGggttttctggacgtcag 273\n",
      "                           ++g++++++t ctgg+ G +gc actg+ Gcga c cc agctg gg +++c+g a    a \n",
      "  NZ_JACNYN010000214.1  73 ACGAATCCTTCCTGGCTGGCGCGACTGAAGCGACCACCAAGCTGTGGGACACCGTAATGGAA 134\n",
      "                           5678899999****************************999998777777766665555555 PP\n",
      "\n",
      ">> NZ_JACNYN010000004.1  Escherichia coli strain UPMP2130 contig00006, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.2   0.0      0.28        12      85     149 ..   85433   85497 ..   85370   85501 .. 0.81\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.2 bits;  conditional E-value: 0.28\n",
      "           tp_nucl_aln    85 cttctgattgaaAAgGGgctgatcgatcccgcagcgctggatgagatcatcgacacctatgagaa 149  \n",
      "                              t+ t +++g  AAgG g tgat ga+c c ++gc c ggat a  t  tcgacac+  +ga+a \n",
      "  NZ_JACNYN010000004.1 85433 ATGATTGCCGGTAAGGTGATGATGGACCGCAACGCACCGGATTATCTGCTCGACACTGCCGAAAG 85497\n",
      "                             57778889999*******************************************99877776665 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        215  (4601339 residues searched)\n",
      "Passed MSV filter:                        28  (0.130233); expected 4.3 (0.02)\n",
      "Passed bias filter:                       12  (0.055814); expected 4.3 (0.02)\n",
      "Passed Vit filter:                        12  (0.055814); expected 4.3 (0.02)\n",
      "Passed Fwd filter:                         5  (0.0232558); expected 4.3 (0.02)\n",
      "Initial search space (Z):                215  [actual number of targets]\n",
      "Domain search space  (domZ):               5  [number of targets reported over threshold]\n",
      "# CPU time: 0.81u 0.01s 00:00:00.82 Elapsed: 00:00:00.47\n",
      "# Mc/sec: 6107.57\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_005280335.1_ASM528033v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_005280335.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (2848891 residues searched)\n",
      "Passed MSV filter:                         1  (1); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.13u 0.02s 00:00:00.15 Elapsed: 00:00:00.15\n",
      "# Mc/sec: 11144.00\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003664995.1_BH458_6_3_2_1_CLC_assembly_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003664995.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.76   -0.7   0.0        1.9   -2.1   0.0    1.3  1  NZ_JFHL01000015.1  Mesotoga sp. BH458_6_3_2_1 contig_16, whol\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_JFHL01000015.1  Mesotoga sp. BH458_6_3_2_1 contig_16, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.1   0.0     0.053       1.9     270     318 ..   76772   76820 ..   76750   76823 .. 0.85\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.1 bits;  conditional E-value: 0.053\n",
      "        tp_nucl_aln   270 tcaggggGAacAtAtggtgGcggtggagAAcacggccgaggtgcAcAAt 318  \n",
      "                          t aggg GAacA At   gGc  t   gAAca g   g+ g gcAcAA+\n",
      "  NZ_JFHL01000015.1 76772 TGAGGGAGAACAAATAAAGGCCCTTATGAACAGGTTGGGTGCGCACAAC 76820\n",
      "                          459********************************************96 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         36  (3234409 residues searched)\n",
      "Passed MSV filter:                         2  (0.0555556); expected 0.7 (0.02)\n",
      "Passed bias filter:                        2  (0.0555556); expected 0.7 (0.02)\n",
      "Passed Vit filter:                         2  (0.0555556); expected 0.7 (0.02)\n",
      "Passed Fwd filter:                         1  (0.0277778); expected 0.7 (0.02)\n",
      "Initial search space (Z):                 36  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.56u 0.00s 00:00:00.56 Elapsed: 00:00:00.52\n",
      "# Mc/sec: 3853.24\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_013726215.1_ASM1372621v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_013726215.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (4818023 residues searched)\n",
      "Passed MSV filter:                         1  (1); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.24u 0.02s 00:00:00.26 Elapsed: 00:00:00.26\n",
      "# Mc/sec: 11403.88\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001494035.1_p474D_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001494035.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         72  (1734238 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 1.4 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 1.4 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 1.4 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.4 (0.02)\n",
      "Initial search space (Z):                 72  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.05u 0.01s 00:00:00.06 Elapsed: 00:00:00.04\n",
      "# Mc/sec: 25901.59\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003063435.1_ASM306343v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003063435.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         23  (3923189 residues searched)\n",
      "Passed MSV filter:                         7  (0.304348); expected 0.5 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.5 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.5 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.5 (0.02)\n",
      "Initial search space (Z):                 23  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.14u 0.02s 00:00:00.16 Elapsed: 00:00:00.08\n",
      "# Mc/sec: 30501.51\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003925855.2_ASM392585v2_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003925855.2_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          4  (4826820 residues searched)\n",
      "Passed MSV filter:                         2  (0.5); expected 0.1 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.1 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.1 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.1 (0.02)\n",
      "Initial search space (Z):                  4  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.24u 0.02s 00:00:00.26 Elapsed: 00:00:00.25\n",
      "# Mc/sec: 11928.68\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000593345.2_vpVPTS2010_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_000593345.2_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence      Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------      -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.22    2.1   0.1       0.29    1.7   0.1    1.1  1  NZ_KK082012.1  Vibrio parahaemolyticus VPTS-2010 vpVPTS2010.c\n",
      "        2.5   -1.4   0.0          4   -2.1   0.0    1.1  1  NZ_KK081990.1  Vibrio parahaemolyticus VPTS-2010 vpVPTS2010.c\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_KK082012.1  Vibrio parahaemolyticus VPTS-2010 vpVPTS2010.contig.72, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    1.7   0.1    0.0078      0.29     431     454 ..     111     134 ..      96     152 .. 0.87\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 1.7 bits;  conditional E-value: 0.0078\n",
      "    tp_nucl_aln 431 tgctggcgGAgtttGGcgtcaccc 454\n",
      "                    tgctgg gGA+tt GGcgt a++c\n",
      "  NZ_KK082012.1 111 TGCTGGTGGAATTAGGCGTGATTC 134\n",
      "                    789****************98865 PP\n",
      "\n",
      ">> NZ_KK081990.1  Vibrio parahaemolyticus VPTS-2010 vpVPTS2010.contig.52, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.1   0.0      0.11         4     355     377 ..      25      47 ..      21      56 .. 0.85\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.1 bits;  conditional E-value: 0.11\n",
      "    tp_nucl_aln 355 cctctgctgGGcctgccgccggc 377\n",
      "                    c t+tg t GGcctgccgcc gc\n",
      "  NZ_KK081990.1  25 CATGTGTTAGGCCTGCCGCCAGC 47 \n",
      "                    556788899**********9886 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         75  (5296462 residues searched)\n",
      "Passed MSV filter:                         7  (0.0933333); expected 1.5 (0.02)\n",
      "Passed bias filter:                        6  (0.08); expected 1.5 (0.02)\n",
      "Passed Vit filter:                         6  (0.08); expected 1.5 (0.02)\n",
      "Passed Fwd filter:                         2  (0.0266667); expected 1.5 (0.02)\n",
      "Initial search space (Z):                 75  [actual number of targets]\n",
      "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
      "# CPU time: 0.29u 0.00s 00:00:00.28 Elapsed: 00:00:00.25\n",
      "# Mc/sec: 13201.94\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_008632455.1_ASM863245v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_008632455.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (4554005 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.10u 0.00s 00:00:00.10 Elapsed: 00:00:00.10\n",
      "# Mc/sec: 27723.43\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_014856455.1_ASM1485645v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_014856455.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        175  (5150225 residues searched)\n",
      "Passed MSV filter:                        24  (0.137143); expected 3.5 (0.02)\n",
      "Passed bias filter:                        1  (0.00571429); expected 3.5 (0.02)\n",
      "Passed Vit filter:                         1  (0.00571429); expected 3.5 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 3.5 (0.02)\n",
      "Initial search space (Z):                175  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.23u 0.01s 00:00:00.24 Elapsed: 00:00:00.14\n",
      "# Mc/sec: 22248.78\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002882435.1_FW215-L1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_002882435.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        124  (6523743 residues searched)\n",
      "Passed MSV filter:                        26  (0.209677); expected 2.5 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 2.5 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 2.5 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 2.5 (0.02)\n",
      "Initial search space (Z):                124  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.19u 0.01s 00:00:00.20 Elapsed: 00:00:00.10\n",
      "# Mc/sec: 37399.10\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002507915.1_ASM250791v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_002507915.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         55  (4860445 residues searched)\n",
      "Passed MSV filter:                         6  (0.109091); expected 1.1 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 1.1 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 1.1 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.1 (0.02)\n",
      "Initial search space (Z):                 55  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.13u 0.00s 00:00:00.13 Elapsed: 00:00:00.07\n",
      "# Mc/sec: 40518.12\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006369625.1_ASM636962v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_006369625.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.27    5.2   0.0       0.37    4.7   0.0    1.1  1  NZ_NNEQ01000764.1  Vibrio parahaemolyticus strain F5_9 F5_9_c\n",
      "       0.35    4.8   0.0       0.56    4.1   0.0    1.1  1  NZ_NNEQ01000043.1  Vibrio parahaemolyticus strain F5_9 F5_9_c\n",
      "        2.1    2.2   0.0        3.5    1.5   0.0    1.2  1  NZ_NNEQ01000220.1  Vibrio parahaemolyticus strain F5_9 F5_9_c\n",
      "        4.4    1.1   0.0        6.2    0.7   0.0    1.1  1  NZ_NNEQ01000603.1  Vibrio parahaemolyticus strain F5_9 F5_9_c\n",
      "        4.8    1.0   0.0         11   -0.2   0.0    1.4  2  NZ_NNEQ01000373.1  Vibrio parahaemolyticus strain F5_9 F5_9_c\n",
      "          5    1.0   0.0        6.8    0.5   0.0    1.1  1  NZ_NNEQ01000727.1  Vibrio parahaemolyticus strain F5_9 F5_9_c\n",
      "          7    0.5   0.0         14   -0.5   0.0    1.3  1  NZ_NNEQ01000255.1  Vibrio parahaemolyticus strain F5_9 F5_9_c\n",
      "        8.8    0.2   0.0         20   -1.0   0.0    1.3  1  NZ_NNEQ01000270.1  Vibrio parahaemolyticus strain F5_9 F5_9_c\n",
      "        9.4    0.1   0.0         13   -0.4   0.0    1.1  1  NZ_NNEQ01000657.1  Vibrio parahaemolyticus strain F5_9 F5_9_c\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_NNEQ01000764.1  Vibrio parahaemolyticus strain F5_9 F5_9_contig767, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    4.7   0.0    0.0043      0.37     541     581 ..      47      88 ..      29      94 .. 0.81\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 4.7 bits;  conditional E-value: 0.0043\n",
      "        tp_nucl_aln 541 ggtctGgacgaggaggcgctggccgcgctg.GtcAcccgcgA 581\n",
      "                         g ctGg+cg ggag ++ctgg cgc  tg GtcAcc gc A\n",
      "  NZ_NNEQ01000764.1  47 CGACTGGGCGTGGAGCATCTGGTCGCATTGgGTCACCAGCAA 88 \n",
      "                        5789**********************9996368888876665 PP\n",
      "\n",
      ">> NZ_NNEQ01000043.1  Vibrio parahaemolyticus strain F5_9 F5_9_contig043, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    4.1   0.0    0.0064      0.56      70     139 ..     785     854 ..     763     868 .. 0.84\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 4.1 bits;  conditional E-value: 0.0064\n",
      "        tp_nucl_aln  70 agggcgctggAaacgcttctgattgaaAAgGGgctgatcgatcccgcagcgctggatgagatcatcgaca 139\n",
      "                        +++gc +tg Aa+ g tt tg t+ aa A+GG +tgat+gat  c ++g + t+ga+ga atc  cg ca\n",
      "  NZ_NNEQ01000043.1 785 CATGCTTTGCAAGAGTTTTTGCTCAAAGAAGGCTTGATTGATATCAACGGCATCGACGACATCTACGCCA 854\n",
      "                        678999*************************************999998888999988887665444443 PP\n",
      "\n",
      ">> NZ_NNEQ01000220.1  Vibrio parahaemolyticus strain F5_9 F5_9_contig220, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    1.5   0.0      0.04       3.5       6      42 ..    2735    2777 ..    2730    2811 .. 0.66\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 1.5 bits;  conditional E-value: 0.04\n",
      "        tp_nucl_aln    6 ccAtGatca......tgccgaccacgaccacccgcacgctctt 42  \n",
      "                          cAtGatca      t  +ga+cacgaccaccc ca+g+tc+t\n",
      "  NZ_NNEQ01000220.1 2735 GCATGATCAccatcaTCATGATCACGACCACCCACATGATCAT 2777\n",
      "                         5667766550000005566777888888888888888777665 PP\n",
      "\n",
      ">> NZ_NNEQ01000603.1  Vibrio parahaemolyticus strain F5_9 F5_9_contig604, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    0.7   0.0     0.071       6.2     293     327 ..     414     448 ..     399     450 .. 0.90\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 0.7 bits;  conditional E-value: 0.071\n",
      "        tp_nucl_aln 293 tggagAAcacggccgaggtgcAcAAtatggtcgtc 327\n",
      "                        t+ga+AA+ cg++ gagg   A AAtatg+tcgt \n",
      "  NZ_NNEQ01000603.1 414 TCGAAAATGCGCAGGAGGCTGAAAATATGATCGTA 448\n",
      "                        789*****************************986 PP\n",
      "\n",
      ">> NZ_NNEQ01000373.1  Vibrio parahaemolyticus strain F5_9 F5_9_contig373, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.2   0.0      0.13        11     116     161 ..     443     488 ..     408     494 .. 0.88\n",
      "   2 ?   -4.5   0.0       2.7   2.3e+02      87     134 ..     864     911 ..     849     915 .. 0.74\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.2 bits;  conditional E-value: 0.13\n",
      "        tp_nucl_aln 116 cagcgctggatgagatcatcgacacctatgagaacaagatcGgccc 161\n",
      "                        cag gct gatg g   at g ca  tatgag a +ag+t Ggccc\n",
      "  NZ_NNEQ01000373.1 443 CAGAGCTAGATGTGGAAATAGCCAAATATGAGTATGAGGTTGGCCC 488\n",
      "                        55668999999999999**************************998 PP\n",
      "\n",
      "  == domain 2  score: -4.5 bits;  conditional E-value: 2.7\n",
      "        tp_nucl_aln  87 tctgattgaaAAgGGgctgatcgatcccgcagcgctggatgagatcat 134\n",
      "                        tct a+ +  AA GGg tg t+ga   c  + cgct  atg++ t at\n",
      "  NZ_NNEQ01000373.1 864 TCTTACACGCAATGGGGTGGTTGAAAACATCTCGCTTTATGCACTAAT 911\n",
      "                        566777777888888888888888888888888888777777777666 PP\n",
      "\n",
      ">> NZ_NNEQ01000727.1  Vibrio parahaemolyticus strain F5_9 F5_9_contig728, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    0.5   0.0     0.078       6.8     117     149 ..       4      36 ..       2      51 .. 0.87\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 0.5 bits;  conditional E-value: 0.078\n",
      "        tp_nucl_aln 117 agcgctggatgagatcatcgacacctatgagaa 149\n",
      "                         +cg+t ga+ + at+ t ga+acct tgag  \n",
      "  NZ_NNEQ01000727.1   4 GACGGTAGACACCATTGTGGAAACCTTTGAGTT 36 \n",
      "                        57999**********************999865 PP\n",
      "\n",
      ">> NZ_NNEQ01000255.1  Vibrio parahaemolyticus strain F5_9 F5_9_contig255, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.5   0.0      0.16        14      97     143 ..    1696    1733 ..    1678    1745 .. 0.83\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.5 bits;  conditional E-value: 0.16\n",
      "        tp_nucl_aln   97 AAgGGgctgatcgatcccgcagcgctggatgagatcatcgacaccta 143 \n",
      "                         AA          g + ccg++gcgc g+a+g+ at+ t gacac+ a\n",
      "  NZ_NNEQ01000255.1 1696 AAC---------GGCACCGACGCGCCGAACGCCATTTTAGACACTCA 1733\n",
      "                         444.........45899*************************99876 PP\n",
      "\n",
      ">> NZ_NNEQ01000270.1  Vibrio parahaemolyticus strain F5_9 F5_9_contig270, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.0   0.0      0.23        20     402     447 ..    3102    3150 ..    3100    3170 .. 0.80\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.0 bits;  conditional E-value: 0.23\n",
      "        tp_nucl_aln  402 cgcccGcgcgGtccgcGAaccgcgc...agcGtgctggcgGAgtttGGc 447 \n",
      "                          +c cG  cgGt cg+GAaccg      ++ G gctg +  A+tttGGc\n",
      "  NZ_NNEQ01000270.1 3102 GTCGCGTTCGGTACGTGAACCGACGgatGATGAGCTGCAAAAATTTGGC 3150\n",
      "                         68999***************99764111456789999999999999997 PP\n",
      "\n",
      ">> NZ_NNEQ01000657.1  Vibrio parahaemolyticus strain F5_9 F5_9_contig658, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.4   0.0      0.16        13      68     108 ..     304     343 ..     268     355 .. 0.84\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.4 bits;  conditional E-value: 0.16\n",
      "        tp_nucl_aln  68 tcagggcgctggAaacgcttctgattgaaAAgGGgctgatc 108\n",
      "                        t a+ gc +t  Aa+c  tt t + +gaaAAgGGg  g t+\n",
      "  NZ_NNEQ01000657.1 304 TGAAAGCTTTTCAAGCAGTTGTTGACGAAAAGGGGA-GGTT 343\n",
      "                        56889999***********99999999999999864.4444 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        781  (5009497 residues searched)\n",
      "Passed MSV filter:                        22  (0.028169); expected 15.6 (0.02)\n",
      "Passed bias filter:                       22  (0.028169); expected 15.6 (0.02)\n",
      "Passed Vit filter:                        22  (0.028169); expected 15.6 (0.02)\n",
      "Passed Fwd filter:                        13  (0.0166453); expected 15.6 (0.02)\n",
      "Initial search space (Z):                781  [actual number of targets]\n",
      "Domain search space  (domZ):               9  [number of targets reported over threshold]\n",
      "# CPU time: 0.47u 0.00s 00:00:00.47 Elapsed: 00:00:00.25\n",
      "# Mc/sec: 12459.26\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002929435.1_ASM292943v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_002929435.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         26  (3890718 residues searched)\n",
      "Passed MSV filter:                         3  (0.115385); expected 0.5 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.5 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.5 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.5 (0.02)\n",
      "Initial search space (Z):                 26  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.18u 0.01s 00:00:00.19 Elapsed: 00:00:00.11\n",
      "# Mc/sec: 21925.67\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003199475.1_ASM319947v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003199475.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.75    0.4   0.0        1.8   -0.9   0.0    1.3  2  NZ_QJFV01000077.1  Salmonella enterica subsp. enterica strain\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_QJFV01000077.1  Salmonella enterica subsp. enterica strain CFSAN008904 SRR1567236_NODE_77_ID_153, whole genome sho\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.9   0.0     0.023       1.8     385     419 ..      34      65 ..      33      73 .. 0.81\n",
      "   2 ?   -4.2   0.0      0.24        18     275     295 ..     360     380 ..     350     382 .. 0.86\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.9 bits;  conditional E-value: 0.023\n",
      "        tp_nucl_aln 385 AAatccgctgcctaccGcgcccGcgcgGtccgcGA 419\n",
      "                        AA+tc g + cct   GcgcccGcgcgGt+ gcGA\n",
      "  NZ_QJFV01000077.1  34 AAGTCAGTATCCT---GCGCCCGCGCGGTGTGCGA 65 \n",
      "                        5666666666666...899**********999998 PP\n",
      "\n",
      "  == domain 2  score: -4.2 bits;  conditional E-value: 0.24\n",
      "        tp_nucl_aln 275 ggGAacAtAtggtgGcggtgg 295\n",
      "                        g+GAa AtAtg +gGcg tgg\n",
      "  NZ_QJFV01000077.1 360 GCGAATATATGCGGGCGCTGG 380\n",
      "                        689999999*99999998876 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         78  (4803231 residues searched)\n",
      "Passed MSV filter:                        10  (0.128205); expected 1.6 (0.02)\n",
      "Passed bias filter:                        3  (0.0384615); expected 1.6 (0.02)\n",
      "Passed Vit filter:                         3  (0.0384615); expected 1.6 (0.02)\n",
      "Passed Fwd filter:                         1  (0.0128205); expected 1.6 (0.02)\n",
      "Initial search space (Z):                 78  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.16u 0.00s 00:00:00.16 Elapsed: 00:00:00.09\n",
      "# Mc/sec: 30862.69\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003003575.1_ASM300357v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003003575.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.39   -0.7   0.0        1.3   -2.4   0.0    1.3  1  NZ_PVWC01000005.1  Bacillus halotolerans strain DGL6 NODE_6_l\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_PVWC01000005.1  Bacillus halotolerans strain DGL6 NODE_6_length_337382_cov_40.553, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.4   0.0     0.066       1.3     246     291 ..    2403    2448 ..    2382    2454 .. 0.90\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.4 bits;  conditional E-value: 0.066\n",
      "        tp_nucl_aln  246 cgccgagctgGggttttctggacgtcaggggGAacAtAtggtgGcg 291 \n",
      "                         cgccg  ctgGg+tttt t   c  cag g GAacAtAt gtgGc \n",
      "  NZ_PVWC01000005.1 2403 CGCCGGCCTGGGCTTTTGTTTTCAGCAGTGTGAACATATTGTGGCT 2448\n",
      "                         689999**********************************999986 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         19  (3739092 residues searched)\n",
      "Passed MSV filter:                         2  (0.105263); expected 0.4 (0.02)\n",
      "Passed bias filter:                        2  (0.105263); expected 0.4 (0.02)\n",
      "Passed Vit filter:                         2  (0.105263); expected 0.4 (0.02)\n",
      "Passed Fwd filter:                         1  (0.0526316); expected 0.4 (0.02)\n",
      "Initial search space (Z):                 19  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 1.73u 0.02s 00:00:01.75 Elapsed: 00:00:01.09\n",
      "# Mc/sec: 2140.06\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003340555.1_ASM334055v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003340555.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        133  (4729406 residues searched)\n",
      "Passed MSV filter:                        50  (0.37594); expected 2.7 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 2.7 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 2.7 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 2.7 (0.02)\n",
      "Initial search space (Z):                133  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.21u 0.01s 00:00:00.22 Elapsed: 00:00:00.11\n",
      "# Mc/sec: 26502.17\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000489195.1_S164_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_000489195.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.84    3.0   0.0        9.7   -0.5   0.0    2.2  4  NZ_AWHP01000083.1  Vibrio parahaemolyticus S164 Contig83, who\n",
      "        2.4    1.5   0.0        5.5    0.3   0.0    1.4  1  NZ_AWHP01000013.1  Vibrio parahaemolyticus S164 Contig13, who\n",
      "        5.1    0.4   0.0          8   -0.2   0.0    1.1  1  NZ_AWHP01000399.1  Vibrio parahaemolyticus S164 Contig399, wh\n",
      "          7   -0.0   0.0         11   -0.7   0.0    1.1  1  NZ_AWHP01000378.1  Vibrio parahaemolyticus S164 Contig378, wh\n",
      "        7.9   -0.2   0.0         15   -1.1   0.0    1.2  1  NZ_AWHP01000242.1  Vibrio parahaemolyticus S164 Contig242, wh\n",
      "        8.7   -0.3   0.0         23   -1.7   0.0    1.3  2  NZ_AWHP01000235.1  Vibrio parahaemolyticus S164 Contig235, wh\n",
      "        8.8   -0.4   0.0         27   -2.0   0.0    1.3  2  NZ_AWHP01000123.1  Vibrio parahaemolyticus S164 Contig123, wh\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_AWHP01000083.1  Vibrio parahaemolyticus S164 Contig83, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.3   0.0      0.23        17      78     132 ..    8403    8459 ..    8381    8475 .. 0.86\n",
      "   2 ?   -3.9   0.0       1.4   1.1e+02     211     295 ..   17392   17479 ..   17381   17486 .. 0.75\n",
      "   3 ?   -0.5   0.0      0.13       9.7     182     280 ..   19313   19411 ..   19302   19418 .. 0.71\n",
      "   4 ?   -4.2   0.0       1.7   1.3e+02     177     248 ..   19651   19718 ..   19589   19727 .. 0.80\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.3 bits;  conditional E-value: 0.23\n",
      "        tp_nucl_aln   78 ggAaacgcttctgattgaaAAgGGgctgatcgatc..ccgcagcgctggatgagatc 132 \n",
      "                         ggAaa    t tgat+g aAA GG  t at+gatc  c  +a+cgc  gatga+at+\n",
      "  NZ_AWHP01000083.1 8403 GGAAAGTGCTTTGATCGGAAATGGCATCATTGATCatCGCAAACGCCTGATGAAATT 8459\n",
      "                         57888888889**********************984457788999999999999875 PP\n",
      "\n",
      "  == domain 2  score: -3.9 bits;  conditional E-value: 1.4\n",
      "        tp_nucl_aln   211 cgggcgcggctgctggagGA........tgccactgcgGcgatcgccgagctgGggttttctggacgtcaggggGAacAtAtggtgGc 290  \n",
      "                          cg  +g+ +ctg+t ga GA        tgcc  tgc+G++a cgccg     Gg t+tt      g ca gg GAacA At  t G \n",
      "  NZ_AWHP01000083.1 17392 CGTAAGTACCTGTTCGATGAacttcgttTGCCTGTGCTGTTAGCGCCGTTGGCGGATATTT-----GGCAAGGTGAACAAATTATTGA 17474\n",
      "                          5566666777888888888777666666789999999**********99999999998875.....679**********999777765 PP\n",
      "\n",
      "        tp_nucl_aln   291 ggtgg 295  \n",
      "                           g +g\n",
      "  NZ_AWHP01000083.1 17475 TGGCG 17479\n",
      "                          55444 PP\n",
      "\n",
      "  == domain 3  score: -0.5 bits;  conditional E-value: 0.13\n",
      "        tp_nucl_aln   182 tgGcgcgggcctGGaccGAcccggagttccgggcgcggctgctggagGAtgccactgcgGcgatcgccgagctgGggttt.....tct 264  \n",
      "                          t Gc cg g +tGG   GA+ c+gagtt+    cg+  c g t g+ GA+g+ +  g+ G  a       g t     tt      +t\n",
      "  NZ_AWHP01000083.1 19313 TTGCTCGCGTATGGCATGATGCCGAGTTTGTTACGGTTCCGGTCGCAGACGGAGGCGAAGGCA-----CGGTTCAATCTTtgataGAT 19395\n",
      "                          6899*********************************************99877666554322.....22222222211111111334 PP\n",
      "\n",
      "        tp_nucl_aln   265 ggacgtcaggggGAac 280  \n",
      "                          g a  tca gg+GAac\n",
      "  NZ_AWHP01000083.1 19396 GCAACTCAAGGCGAAC 19411\n",
      "                          4455566666666666 PP\n",
      "\n",
      "  == domain 4  score: -4.2 bits;  conditional E-value: 1.7\n",
      "        tp_nucl_aln   177 cgtgGtgGcgcgggcctGGaccGAcccggagttccgggcgcggctgctggagGAtgccactgcgGcgatcgc 248  \n",
      "                            tgGtgGcg gggc+tG  + GAc  +g gtt  gggc  +g+t ct ga GA  c +   ++Gc at+ c\n",
      "  NZ_AWHP01000083.1 19651 GATGGTGGCGTGGGCATG--TTGAC--CGCGTTGGGGGCTACGTTTCTTGATGAATCTGGCCATGCAATTAC 19718\n",
      "                          4599*************8..78886..5999********************999999988888888777665 PP\n",
      "\n",
      ">> NZ_AWHP01000013.1  Vibrio parahaemolyticus S164 Contig13, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    0.3   0.0     0.071       5.5     423     467 ..   37843   37887 ..   37826   37920 .. 0.83\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 0.3 bits;  conditional E-value: 0.071\n",
      "        tp_nucl_aln   423 gcgcagcGtgctggcgGAgtttGGcgtcaccctGcccgatgacat 467  \n",
      "                          gc c+gcGt ct gc+GA+tt+GGcgt +++ tG  +g +g +a+\n",
      "  NZ_AWHP01000013.1 37843 GCACGGCGTTCTTGCTGAATTCGGCGTAGAGATGATTGGCGCTAC 37887\n",
      "                          566899**********************99988866665555555 PP\n",
      "\n",
      ">> NZ_AWHP01000399.1  Vibrio parahaemolyticus S164 Contig399, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.2   0.0       0.1         8     208     262 ..     502     556 ..     498     569 .. 0.88\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.2 bits;  conditional E-value: 0.1\n",
      "        tp_nucl_aln 208 ttccgggcgcggctgctggagGAtgccactgcgGcgatcgccgagctgGggtttt 262\n",
      "                        tt +g  +++gg+t+ t g+ GAtg+ca+ g +G+     c gag tgG  tttt\n",
      "  NZ_AWHP01000399.1 502 TTGAGTAAAGGGTTAATCGCAGATGGCAAAGTTGTACAATCTGAGGTGGAATTTT 556\n",
      "                        555667788999*************************************999875 PP\n",
      "\n",
      ">> NZ_AWHP01000378.1  Vibrio parahaemolyticus S164 Contig378, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.7   0.0      0.15        11     197     234 ..     869     906 ..     858     935 .. 0.84\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.7 bits;  conditional E-value: 0.15\n",
      "        tp_nucl_aln 197 ccGAcccggagttccgggcgcggctgctggagGAtgcc 234\n",
      "                        +cGA+cc+   ttccg  +g+g ct++tgga GA  cc\n",
      "  NZ_AWHP01000378.1 869 TCGATCCACCTTTCCGTAAGGGACTATTGGATGAAACC 906\n",
      "                        689**************************999985554 PP\n",
      "\n",
      ">> NZ_AWHP01000242.1  Vibrio parahaemolyticus S164 Contig242, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.1   0.0      0.19        15     115     155 ..    3567    3607 ..    3561    3608 .. 0.90\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.1 bits;  conditional E-value: 0.19\n",
      "        tp_nucl_aln  115 gcagcgctggatgagatcatcgacacctatgagaacaagat 155 \n",
      "                         g  gc+ t +a ga atc tcga acct t agaacaag+t\n",
      "  NZ_AWHP01000242.1 3567 GTTGCCTTTAAAGACATCTTCGATACCTTTAAGAACAAGGT 3607\n",
      "                         6678899999***************************9987 PP\n",
      "\n",
      ">> NZ_AWHP01000235.1  Vibrio parahaemolyticus S164 Contig235, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -4.5   0.0       2.1   1.6e+02     113     147 ..    3629    3660 ..    3627    3664 .. 0.89\n",
      "   2 ?   -1.7   0.0       0.3        23     108     160 ..    4041    4093 ..    4035    4101 .. 0.89\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -4.5 bits;  conditional E-value: 2.1\n",
      "        tp_nucl_aln  113 ccgcagcgctggatgagatcatcgacacctatgag 147 \n",
      "                         ccg+agcgct+ga g++   at gaca ctatga+\n",
      "  NZ_AWHP01000235.1 3629 CCGAAGCGCTCGAAGCA---ATGGACAACTATGAA 3660\n",
      "                         7899************9...899*******99975 PP\n",
      "\n",
      "  == domain 2  score: -1.7 bits;  conditional E-value: 0.3\n",
      "        tp_nucl_aln  108 cgatcccgcagcgctggatgagatcatcgacacctatgagaacaagatcGgcc 160 \n",
      "                         cg tc cgcagc ctg a+ga a cat g  a ct  g  aacaa +tcG   \n",
      "  NZ_AWHP01000235.1 4041 CGCTCGCGCAGCTCTGTACGACAACATTGCGAACTTAGCTAACAAAGTCGATT 4093\n",
      "                         5668899***************************************9999765 PP\n",
      "\n",
      ">> NZ_AWHP01000123.1  Vibrio parahaemolyticus S164 Contig123, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -4.4   0.0       1.9   1.4e+02     281     321 ..    8894    8934 ..    8890    8945 .. 0.91\n",
      "   2 ?   -2.0   0.0      0.35        27     451     589 ..   12239   12396 ..   12237   12420 .. 0.73\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -4.4 bits;  conditional E-value: 1.9\n",
      "        tp_nucl_aln  281 AtAtggtgGcggtggagAAcacggccgaggtgcAcAAtatg 321 \n",
      "                         A+At gtgGcg t gagAAc cg   ga  tg AcAA +t \n",
      "  NZ_AWHP01000123.1 8894 ACATTGTGGCGTTAGAGAACGCGTTGGATATGTACAAGCTA 8934\n",
      "                         67999***************************999997775 PP\n",
      "\n",
      "  == domain 2  score: -2.0 bits;  conditional E-value: 0.35\n",
      "        tp_nucl_aln   451 accctGcccgatgacatggaggtgcgcGtctggGAt.tccAccGccgAggtgcGctatctgGtga.....tgcccgagcGccctgaag 532  \n",
      "                          a+c+tGcc gatg   tgg+  tgc  Gtc g GAt +c Ac Gc+g g  gc ct   tgGtga      gc   + cG c + +ag\n",
      "  NZ_AWHP01000123.1 12239 ATCTTGCCAGATGTATTGGCATTGCCTGTCAGTGATgACAACAGCAGTGCGGCACTGATTGGTGAgcaatGGCTAATTCGTCACTCAG 12326\n",
      "                          57899******************************95577788887766666667888999998622222246667789999999999 PP\n",
      "\n",
      "        tp_nucl_aln   533 ggAccGaaggtctGgacgaggaggcg......ctgg.......ccgcgctgGtcAcccgcgAcagcAtGa 589  \n",
      "                           gAc Gaagg   Gg  g gga  cg      ctgg       cc c+ct  t Acc gc A a+cAtGa\n",
      "  NZ_AWHP01000123.1 12327 AGACGGAAGGCGCGGTGGTGGATTCGgcgtggCTGGatctttaCCTCTCTTCTTACCTGCAAAACCATGA 12396\n",
      "                          9999999999887777777666533211111133322222222777888889999999999999999987 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        541  (4985943 residues searched)\n",
      "Passed MSV filter:                        17  (0.0314233); expected 10.8 (0.02)\n",
      "Passed bias filter:                       17  (0.0314233); expected 10.8 (0.02)\n",
      "Passed Vit filter:                        17  (0.0314233); expected 10.8 (0.02)\n",
      "Passed Fwd filter:                         8  (0.0147874); expected 10.8 (0.02)\n",
      "Initial search space (Z):                541  [actual number of targets]\n",
      "Domain search space  (domZ):               7  [number of targets reported over threshold]\n",
      "# CPU time: 0.71u 0.01s 00:00:00.72 Elapsed: 00:00:00.40\n",
      "# Mc/sec: 7760.51\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000769825.1_ASM76982v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_000769825.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         63  (4479972 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 1.3 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 1.3 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 1.3 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.3 (0.02)\n",
      "Initial search space (Z):                 63  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.10u 0.00s 00:00:00.10 Elapsed: 00:00:00.05\n",
      "# Mc/sec: 48282.19\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001833665.1_ASM183366v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001833665.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         49  (4784753 residues searched)\n",
      "Passed MSV filter:                         6  (0.122449); expected 1.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 1.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 1.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.0 (0.02)\n",
      "Initial search space (Z):                 49  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.15u 0.01s 00:00:00.16 Elapsed: 00:00:00.07\n",
      "# Mc/sec: 37345.05\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006802465.1_ASM680246v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_006802465.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.12    5.1   0.0        0.2    4.4   0.0    1.2  1  NZ_VIPA01000057.1  Vibrio cholerae strain A12JL36W25 NODE_64_\n",
      "        1.6    1.3   0.3         14   -1.7   0.0    2.0  2  NZ_VIPA01000195.1  Vibrio cholerae strain A12JL36W25 NODE_275\n",
      "        2.9    0.5   0.1         18   -2.1   0.1    1.7  2  NZ_VIPA01000133.1  Vibrio cholerae strain A12JL36W25 NODE_179\n",
      "        3.2    0.4   0.0        5.7   -0.5   0.0    1.2  1  NZ_VIPA01000023.1  Vibrio cholerae strain A12JL36W25 NODE_29_\n",
      "        4.3   -0.1   0.1         17   -2.0   0.1    1.5  1  NZ_VIPA01000016.1  Vibrio cholerae strain A12JL36W25 NODE_19_\n",
      "        4.5   -0.1   0.0         14   -1.7   0.0    1.4  1  NZ_VIPA01000098.1  Vibrio cholerae strain A12JL36W25 NODE_127\n",
      "        6.4   -0.6   0.0         37   -3.1   0.0    1.3  2  NZ_VIPA01000253.1  Vibrio cholerae strain A12JL36W25 NODE_379\n",
      "          7   -0.7   0.0         26   -2.6   0.0    1.3  2  NZ_VIPA01000060.1  Vibrio cholerae strain A12JL36W25 NODE_74_\n",
      "        7.2   -0.8   0.0         27   -2.7   0.0    1.3  1  NZ_VIPA01000140.1  Vibrio cholerae strain A12JL36W25 NODE_191\n",
      "        9.1   -1.1   0.0         39   -3.2   0.0    1.3  1  NZ_VIPA01000059.1  Vibrio cholerae strain A12JL36W25 NODE_66_\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_VIPA01000057.1  Vibrio cholerae strain A12JL36W25 NODE_64_length_6077_cov_11.348363, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    4.4   0.0    0.0061       0.2      84     161 ..    2028    2113 ..    1932    2117 .. 0.60\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 4.4 bits;  conditional E-value: 0.0061\n",
      "        tp_nucl_aln   84 gcttctgattgaaAAgGGgctgatcgatcccgcagcgctggatgagatcatcgacacctatgag........aacaagatcGgccc 161 \n",
      "                           t  tg++ gaaAA+GG    at+  tc  gc gcg tgga+ a atc t g     t  gag        a c++ga  Ggccc\n",
      "  NZ_VIPA01000057.1 2028 TGTCTTGGCAGAAAAAGGCACCATTAGTCAGGCTGCGATGGACAATATCCTTGCTGAATTAGAGacagcgcaAGCCGGACTGGCCC 2113\n",
      "                         457899******************************************99998776666655441111111145666666666665 PP\n",
      "\n",
      ">> NZ_VIPA01000195.1  Vibrio cholerae strain A12JL36W25 NODE_275_length_40689_cov_14.832412, whole genome shotgun sequen\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.0   0.1      0.52        17     170     264 ..   17336   17425 ..   17320   17435 .. 0.72\n",
      "   2 ?   -1.7   0.0      0.41        14     427     465 ..   36198   36236 ..   36194   36252 .. 0.80\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.0 bits;  conditional E-value: 0.52\n",
      "        tp_nucl_aln   170 GtGcccacgtgGtgGcgcgggcctG.GaccGAcccggagt.tccgggcgcggctgctggagGAtgccactgcgGcgatcgccgagctg 255  \n",
      "                          G Gc     tgGt     g gc tG G +cGA+ c+gagt t+c++ + +gg+tg+t g+ GAt cca+  +gGc  t g   agct \n",
      "  NZ_VIPA01000195.1 17336 GCGC--TGTTGGTT----GAGCGTGaGCTCGATGCAGAGTgTTCACAACTGGTTGTT-GCCGATACCAAACAGGCCTTAGGTCAGCTT 17416\n",
      "                          2222..23345555....555566647789********9945566666667777665.6789*************************9 PP\n",
      "\n",
      "        tp_nucl_aln   256 Gggttttct 264  \n",
      "                          Gg+t tt  \n",
      "  NZ_VIPA01000195.1 17417 GGCTCTTGG 17425\n",
      "                          999877654 PP\n",
      "\n",
      "  == domain 2  score: -1.7 bits;  conditional E-value: 0.41\n",
      "        tp_nucl_aln   427 agcGtgctggcgGAgtttGGcgtcaccctGcccgatgac 465  \n",
      "                          +gcGt  tggc GAgtttGGcgt++++ tG  +g tg  \n",
      "  NZ_VIPA01000195.1 36198 GGCGTATTGGCCGAGTTTGGCGTTGAGATGATTGGTGCA 36236\n",
      "                          6799********************998887666555555 PP\n",
      "\n",
      ">> NZ_VIPA01000133.1  Vibrio cholerae strain A12JL36W25 NODE_179_length_38529_cov_14.474733, whole genome shotgun sequen\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.1   0.1      0.54        18     195     299 ..   23957   24045 ..   23954   24058 .. 0.73\n",
      "   2 ?   -4.5   0.0       2.9        97     281     320 ..   25554   25593 ..   25553   25595 .. 0.92\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.1 bits;  conditional E-value: 0.54\n",
      "        tp_nucl_aln   195 GaccGAcccggagttccgggcgcggctgctggagGAtgccactgcgGcgatcgccgagctgGggttttctggacgtcaggggGAacAt 282  \n",
      "                          G+ccGA ccgg  t cc++g+gc gctgctgg gGAtg       +Gcg t    gagctg    t+ +t g+     ggg+GAacA \n",
      "  NZ_VIPA01000133.1 23957 GGCCGAACCGGTAT-CCACGAGCTGCTGCTGGTGGATGA------TGCGTTGCAGGAGCTGA---TCCATAGCGAGGCGGGCGAACA- 24033\n",
      "                          46889999999886.778999***********9999874......55555555556666654...3445555556667889999985. PP\n",
      "\n",
      "        tp_nucl_aln   283 AtggtgGcggtggagAA 299  \n",
      "                               gGcg tggagAA\n",
      "  NZ_VIPA01000133.1 24034 -----GGCGATGGAGAA 24045\n",
      "                          .....677777777764 PP\n",
      "\n",
      "  == domain 2  score: -4.5 bits;  conditional E-value: 2.9\n",
      "        tp_nucl_aln   281 AtAtggtgGcggtggagAAcacggccgaggtgcAcAAtat 320  \n",
      "                          AtAt gt Gcg tgga+AA+ cg   ga  tg AcAA +t\n",
      "  NZ_VIPA01000133.1 25554 ATATCGTTGCGCTGGAAAATGCGTTGGATATGTACAAGCT 25593\n",
      "                          789****************************999999776 PP\n",
      "\n",
      ">> NZ_VIPA01000023.1  Vibrio cholerae strain A12JL36W25 NODE_29_length_25934_cov_16.730316, whole genome shotgun sequenc\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.5   0.0      0.17       5.7      10      51 ..   22202   22243 ..   22195   22295 .. 0.76\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.5 bits;  conditional E-value: 0.17\n",
      "        tp_nucl_aln    10 Gatcatgccgaccacgaccacccgcacgctcttttgccgccg 51   \n",
      "                          Ga ca+gccgaccacgaccac  gc++g+t tt+tgc   + \n",
      "  NZ_VIPA01000023.1 22202 GACCACGCCGACCACGACCACTGGCTTGATATTCTGCTAAAC 22243\n",
      "                          899****************************99999976443 PP\n",
      "\n",
      ">> NZ_VIPA01000016.1  Vibrio cholerae strain A12JL36W25 NODE_19_length_210234_cov_15.100032, whole genome shotgun sequen\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.0   0.1      0.52        17     536     603 ..   58247   58311 ..   58242   58324 .. 0.80\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.0 bits;  conditional E-value: 0.52\n",
      "        tp_nucl_aln   536 ccGaaggtctGgacgaggaggcgctggccgcgctgGtcAcccgcgAcagcAtGatcGGtacgGggctg 603  \n",
      "                          c Gaag t tGga g  ga gcgctggc+ c+ctgGt      c Aca cAtG  cGG+a  G g   \n",
      "  NZ_VIPA01000016.1 58247 CTGAAGATGTGGAAGGCGAAGCGCTGGCGACTCTGGTTG---TCAACAACATGCGCGGCATCGTGAAA 58311\n",
      "                          5799999****************************9963...34677777777777777766665555 PP\n",
      "\n",
      ">> NZ_VIPA01000098.1  Vibrio cholerae strain A12JL36W25 NODE_127_length_18990_cov_14.195840, whole genome shotgun sequen\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.7   0.0      0.41        14     289     324 ..   12908   12943 ..   12903   12951 .. 0.88\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.7 bits;  conditional E-value: 0.41\n",
      "        tp_nucl_aln   289 GcggtggagAAcacggccgaggtgcAcAAtatggtc 324  \n",
      "                          Gc g  ga AA+ cg ccga gt cA+AA+at gt \n",
      "  NZ_VIPA01000098.1 12908 GCTGAAGATAATGCGACCGACGTTCATAACATCGTT 12943\n",
      "                          566677999***********************9986 PP\n",
      "\n",
      ">> NZ_VIPA01000253.1  Vibrio cholerae strain A12JL36W25 NODE_379_length_13412_cov_15.330973, whole genome shotgun sequen\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -3.2   0.0       1.2        38     165     245 ..    5703    5796 ..    5636    5804 .. 0.70\n",
      "   2 ?   -3.1   0.0       1.1        37     487     515 ..    7320    7348 ..    7316    7356 .. 0.92\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -3.2 bits;  conditional E-value: 1.2\n",
      "        tp_nucl_aln  165 gAAcGGtGcccacgt.gGtgGcgcgggc.ctGG...........accGAcccggagttccgggcgcggctgctggagGAtgccactgcgG 241 \n",
      "                         +AA+GG Gc   c t  Gt Gcgcg g  ctGG           + cGA+cc g gt +cg g+++g  tg t  + GAt  c+ t ++G\n",
      "  NZ_VIPA01000253.1 5703 CAATGGCGCGTTCTTaTGTCGCGCGCGAaCTGGtaaaactcggtGGCGATCCTGTGTATCGTGAAGGTGTGATCACCGATAACGGTAACG 5792\n",
      "                         567777777665544235555555554223444222222222224689***************************999998777777777 PP\n",
      "\n",
      "        tp_nucl_aln  242 cgat 245 \n",
      "                         ++at\n",
      "  NZ_VIPA01000253.1 5793 TTAT 5796\n",
      "                         6655 PP\n",
      "\n",
      "  == domain 2  score: -3.1 bits;  conditional E-value: 1.1\n",
      "        tp_nucl_aln  487 tccAccGccgAggtgcGctatctgGtgat 515 \n",
      "                         +ccAccGc+gA++t+ Gctat tgGt+at\n",
      "  NZ_VIPA01000253.1 7320 ACCACCGCAGAAATCGGCTATGTGGTCAT 7348\n",
      "                         689***********************987 PP\n",
      "\n",
      ">> NZ_VIPA01000060.1  Vibrio cholerae strain A12JL36W25 NODE_74_length_950_cov_35.090527, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -3.2   0.0       1.1        38     385     437 ..     209     253 ..     208     258 .. 0.80\n",
      "   2 ?   -2.6   0.0      0.79        26      87     140 ..     863     912 ..     799     929 .. 0.68\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -3.2 bits;  conditional E-value: 1.1\n",
      "        tp_nucl_aln 385 AAatccgctgcctaccGcgcccGcgcgGtccgcGAaccgcgcagcGtgctggc 437\n",
      "                        AAatc g+t   taccGc+c  Gc cg t cg GA         cGtgctg +\n",
      "  NZ_VIPA01000060.1 209 AAATCTGATTATTACCGCTCAGGCTCGATTCGGGAG--------CGTGCTGAA 253\n",
      "                        6888899999999********999999999998875........577777654 PP\n",
      "\n",
      "  == domain 2  score: -2.6 bits;  conditional E-value: 0.79\n",
      "        tp_nucl_aln  87 tctgattgaaAAgGGgctgatcgatcccgcagcgctggatgagatcatcgacac 140\n",
      "                        t tga+   aAAgG  c g t ga c c    cgctgga+g   t atcga+  \n",
      "  NZ_VIPA01000060.1 863 TTTGACAACAAAGGTACAGGTAGAACAC----CGCTGGACGTTTTAATCGAAGT 912\n",
      "                        3456666666777777888898887765....89999***99999999998754 PP\n",
      "\n",
      ">> NZ_VIPA01000140.1  Vibrio cholerae strain A12JL36W25 NODE_191_length_25058_cov_11.729946, whole genome shotgun sequen\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.7   0.0      0.83        27      59     168 ..   11091   11208 ..   11064   11246 .. 0.76\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.7 bits;  conditional E-value: 0.83\n",
      "        tp_nucl_aln    59 ctttgcgggtcagggcgctggAaacgc......ttctgattgaaAAg...GGgctgatcgatcccgcagcgctggatgagatcatcga 137  \n",
      "                          c+ tgcg  tc   gc c ggAaac        tt tg t+   AA+   G g  gat+ga c  g  +cg   g +ga at+atcg \n",
      "  NZ_VIPA01000140.1 11091 CCATGCGAATCGCCGCACCGGAAACCAgcggtgTTTTG-TGTTTAAAgcaGAGGCGATTGAGCTGGTTACGTGTGTCGACATTATCGC 11177\n",
      "                          55566666666666777777777653222222133222.2222222202222336899999999999****999************** PP\n",
      "\n",
      "        tp_nucl_aln   138 cacctatgagaacaagatcGgcccgcggAAc 168  \n",
      "                          +a c  + a+ ac+agatcG + cg g+AA+\n",
      "  NZ_VIPA01000140.1 11178 AAGCATCCAACACCAGATCGTGTCGTGCAAT 11208\n",
      "                          *************************999985 PP\n",
      "\n",
      ">> NZ_VIPA01000059.1  Vibrio cholerae strain A12JL36W25 NODE_66_length_35868_cov_13.540538, whole genome shotgun sequenc\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -3.2   0.0       1.2        39     350     388 ..   24954   24992 ..   24951   24994 .. 0.91\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -3.2 bits;  conditional E-value: 1.2\n",
      "        tp_nucl_aln   350 cgtGgcctctgctgGGcctgccgccggcctGGtAcAAat 388  \n",
      "                          cgt gcc++ gctgGG ct+c  c g   tGGtA+AAat\n",
      "  NZ_VIPA01000059.1 24954 CGTTGCCAGCGCTGGGGCTTCATCAGTAATGGTATAAAT 24992\n",
      "                          56678999*****************************97 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        330  (4063635 residues searched)\n",
      "Passed MSV filter:                        18  (0.0545455); expected 6.6 (0.02)\n",
      "Passed bias filter:                       17  (0.0515152); expected 6.6 (0.02)\n",
      "Passed Vit filter:                        17  (0.0515152); expected 6.6 (0.02)\n",
      "Passed Fwd filter:                        10  (0.030303); expected 6.6 (0.02)\n",
      "Initial search space (Z):                330  [actual number of targets]\n",
      "Domain search space  (domZ):              10  [number of targets reported over threshold]\n",
      "# CPU time: 1.58u 0.02s 00:00:01.60 Elapsed: 00:00:01.02\n",
      "# Mc/sec: 2468.85\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003296495.1_ASM329649v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003296495.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        444  (4569939 residues searched)\n",
      "Passed MSV filter:                       105  (0.236486); expected 8.9 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 8.9 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 8.9 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 8.9 (0.02)\n",
      "Initial search space (Z):                444  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.15u 0.01s 00:00:00.16 Elapsed: 00:00:00.08\n",
      "# Mc/sec: 35269.46\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001573015.1_ASM157301v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001573015.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         91  (2101776 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 1.8 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 1.8 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 1.8 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.8 (0.02)\n",
      "Initial search space (Z):                 91  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.09u 0.00s 00:00:00.09 Elapsed: 00:00:00.05\n",
      "# Mc/sec: 22364.37\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000737775.1_ASM73777v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_000737775.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         89  (4340594 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 1.8 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 1.8 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 1.8 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.8 (0.02)\n",
      "Initial search space (Z):                 89  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.09u 0.00s 00:00:00.09 Elapsed: 00:00:00.05\n",
      "# Mc/sec: 49221.63\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_900166155.1_11670_1_22_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_900166155.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         43  (4125071 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.9 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.9 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.9 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.9 (0.02)\n",
      "Initial search space (Z):                 43  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.09u 0.01s 00:00:00.10 Elapsed: 00:00:00.05\n",
      "# Mc/sec: 49466.62\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001870285.1_ASM187028v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001870285.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          9  (2270806 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.2 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.2 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.2 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.2 (0.02)\n",
      "Initial search space (Z):                  9  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.06u 0.00s 00:00:00.06 Elapsed: 00:00:00.03\n",
      "# Mc/sec: 39499.89\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002156885.1_ASM215688v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_002156885.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "          7   -1.4   0.0         13   -2.3   0.0    1.1  1  NZ_NCWQ01000118.1  Flavobacterium sp. AJR contig118, whole ge\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_NCWQ01000118.1  Flavobacterium sp. AJR contig118, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.3   0.0     0.062        13      84     132 ..   16959   17007 ..   16951   17022 .. 0.88\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.3 bits;  conditional E-value: 0.062\n",
      "        tp_nucl_aln    84 gcttctgattgaaAAgGGgctgatcgatcccgcagcgctggatgagatc 132  \n",
      "                            t+ t at+ga+AA GG     t+gatc cg+agcgct  a+ a  t+\n",
      "  NZ_NCWQ01000118.1 16959 AATGTTTATCGAGAACGGAAAATTTGATCACGAAGCGCTTTACAACGTT 17007\n",
      "                          5688999*******************************98887776655 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        203  (5396196 residues searched)\n",
      "Passed MSV filter:                         3  (0.0147783); expected 4.1 (0.02)\n",
      "Passed bias filter:                        3  (0.0147783); expected 4.1 (0.02)\n",
      "Passed Vit filter:                         3  (0.0147783); expected 4.1 (0.02)\n",
      "Passed Fwd filter:                         1  (0.00492611); expected 4.1 (0.02)\n",
      "Initial search space (Z):                203  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.27u 0.00s 00:00:00.27 Elapsed: 00:00:00.14\n",
      "# Mc/sec: 23036.08\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003348965.1_ASM334896v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003348965.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.78    0.4   0.0          2   -1.0   0.0    1.4  1  NZ_QEBE01000038.1  Vibrio cholerae strain OYP6G01 Vc_OYP6G01_\n",
      "       0.94    0.1   0.1        3.5   -1.8   0.1    1.5  1  NZ_QEBE01000005.1  Vibrio cholerae strain OYP6G01 Vc_OYP6G01_\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_QEBE01000038.1  Vibrio cholerae strain OYP6G01 Vc_OYP6G01_Contig_38, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.0   0.0     0.051         2     109     157 ..   19176   19224 ..   19166   19228 .. 0.88\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.0 bits;  conditional E-value: 0.051\n",
      "        tp_nucl_aln   109 gatcccgcagcgctggatgagatcatcgacacctatgagaacaagatcG 157  \n",
      "                          ga+ ccgc+gc ctggatg++ tcatc aca ct +     caa atcG\n",
      "  NZ_QEBE01000038.1 19176 GACACCGCCGCACTGGATGCACTCATCAACAGCTTCTCCGGCAATATCG 19224\n",
      "                          7899***************************999998888888888776 PP\n",
      "\n",
      ">> NZ_QEBE01000005.1  Vibrio cholerae strain OYP6G01 Vc_OYP6G01_Contig_5, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.8   0.1     0.086       3.5     536     603 ..   95496   95560 ..   95491   95573 .. 0.80\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.8 bits;  conditional E-value: 0.086\n",
      "        tp_nucl_aln   536 ccGaaggtctGgacgaggaggcgctggccgcgctgGtcAcccgcgAcagcAtGatcGGtacgGggctg 603  \n",
      "                          c Gaag t tGga g  ga gcgctggc+ c+ctgGt      c Aca cAtG  cGG+a  G g   \n",
      "  NZ_QEBE01000005.1 95496 CTGAAGATGTGGAAGGCGAAGCGCTGGCGACTCTGGTTG---TCAACAACATGCGCGGCATCGTGAAA 95560\n",
      "                          5799999****************************9963...34677777777777777766665555 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         80  (3850126 residues searched)\n",
      "Passed MSV filter:                         3  (0.0375); expected 1.6 (0.02)\n",
      "Passed bias filter:                        3  (0.0375); expected 1.6 (0.02)\n",
      "Passed Vit filter:                         3  (0.0375); expected 1.6 (0.02)\n",
      "Passed Fwd filter:                         2  (0.025); expected 1.6 (0.02)\n",
      "Initial search space (Z):                 80  [actual number of targets]\n",
      "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
      "# CPU time: 0.78u 0.01s 00:00:00.79 Elapsed: 00:00:00.74\n",
      "# Mc/sec: 3204.06\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_013032445.1_ASM1303244v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_013032445.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         82  (5243664 residues searched)\n",
      "Passed MSV filter:                        16  (0.195122); expected 1.6 (0.02)\n",
      "Passed bias filter:                        1  (0.0121951); expected 1.6 (0.02)\n",
      "Passed Vit filter:                         1  (0.0121951); expected 1.6 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.6 (0.02)\n",
      "Initial search space (Z):                 82  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.65u 0.00s 00:00:00.65 Elapsed: 00:00:00.56\n",
      "# Mc/sec: 5796.42\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003295805.1_ASM329580v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003295805.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        470  (4533357 residues searched)\n",
      "Passed MSV filter:                       100  (0.212766); expected 9.4 (0.02)\n",
      "Passed bias filter:                        1  (0.00212766); expected 9.4 (0.02)\n",
      "Passed Vit filter:                         1  (0.00212766); expected 9.4 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 9.4 (0.02)\n",
      "Initial search space (Z):                470  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.15u 0.00s 00:00:00.15 Elapsed: 00:00:00.08\n",
      "# Mc/sec: 34112.90\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_014696555.1_ASM1469655v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_014696555.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence             Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------             -----------\n",
      "  ------ inclusion threshold ------\n",
      "          2   -0.2   0.0        3.3   -0.9   0.0    1.1  1  NZ_JACJQH010000121.1  Calothrix parietina FACHB-288 contig121\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_JACJQH010000121.1  Calothrix parietina FACHB-288 contig121, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.9   0.0     0.024       3.3     200     238 ..    5163    5201 ..    5159    5212 .. 0.92\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.9 bits;  conditional E-value: 0.024\n",
      "           tp_nucl_aln  200 AcccggagttccgggcgcggctgctggagGAtgccactg 238 \n",
      "                            Accc+ga tt++ +gc + g+tgct ga GAtg+ ac  \n",
      "  NZ_JACJQH010000121.1 5163 ACCCAGATTTTACCGCCGTGTTGCTCGATGATGGTACGA 5201\n",
      "                            89*******************************999865 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        138  (9242072 residues searched)\n",
      "Passed MSV filter:                         2  (0.0144928); expected 2.8 (0.02)\n",
      "Passed bias filter:                        2  (0.0144928); expected 2.8 (0.02)\n",
      "Passed Vit filter:                         2  (0.0144928); expected 2.8 (0.02)\n",
      "Passed Fwd filter:                         1  (0.00724638); expected 2.8 (0.02)\n",
      "Initial search space (Z):                138  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.29u 0.00s 00:00:00.28 Elapsed: 00:00:00.16\n",
      "# Mc/sec: 35113.55\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_012275045.1_ASM1227504v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_012275045.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          2  (5036404 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  2  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.13u 0.00s 00:00:00.13 Elapsed: 00:00:00.08\n",
      "# Mc/sec: 34957.21\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002192535.1_ASM219253v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_002192535.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.14   -1.1   0.0       0.51   -3.0   0.0    1.3  1  NZ_NAIA01000002.1  Polynucleobacter hirudinilacicola strain M\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_NAIA01000002.1  Polynucleobacter hirudinilacicola strain MWH-EgelM1-30-B4 2590873606, whole genome shotgun sequenc\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -3.0   0.0       0.1      0.51       7      54 ..  168669  168716 ..  168661  168752 .. 0.70\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -3.0 bits;  conditional E-value: 0.1\n",
      "        tp_nucl_aln      7 cAtGatcatgccgaccacgaccacccgcacgctcttttgccgccg.gaa 54    \n",
      "                           cA Gatcat+ cga+  cgaccac++ cacg++ +tt gc  cc  ga \n",
      "  NZ_NAIA01000002.1 168669 CACGATCATTACGATTGCGACCACGACCACGACGATT-GCGACCAcGAC 168716\n",
      "                           6778888888888888888888888888887776653.33333222443 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          5  (2008240 residues searched)\n",
      "Passed MSV filter:                         1  (0.2); expected 0.1 (0.02)\n",
      "Passed bias filter:                        1  (0.2); expected 0.1 (0.02)\n",
      "Passed Vit filter:                         1  (0.2); expected 0.1 (0.02)\n",
      "Passed Fwd filter:                         1  (0.2); expected 0.1 (0.02)\n",
      "Initial search space (Z):                  5  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.91u 0.02s 00:00:00.93 Elapsed: 00:00:00.92\n",
      "# Mc/sec: 1356.74\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002106585.1_ASM210658v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_002106585.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.53    0.8   0.1       0.72    0.3   0.1    1.1  1  NZ_NBQI01000072.1  Salmonella enterica subsp. houtenae serova\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_NBQI01000072.1  Salmonella enterica subsp. houtenae serovar 40:z4,z32:- strain NY_FSL S10-1821 NODE_73_length_292_\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    0.3   0.1    0.0099      0.72     340     377 ..      49      91 ..      43     100 .. 0.75\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 0.3 bits;  conditional E-value: 0.0099\n",
      "        tp_nucl_aln 340 tcctGctAcc......cgtGgcctctgctgGGcctgccgccggc 377\n",
      "                        tcctG+tAcc      c t gc t+tg t GGcctgccgcc gc\n",
      "  NZ_NBQI01000072.1  49 TCCTGTTACCgttcgaCTT-GCATGTGTTAGGCCTGCCGCCAGC 91 \n",
      "                        5788888888333222222.45578999*************986 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         73  (4678868 residues searched)\n",
      "Passed MSV filter:                         9  (0.123288); expected 1.5 (0.02)\n",
      "Passed bias filter:                        2  (0.0273973); expected 1.5 (0.02)\n",
      "Passed Vit filter:                         2  (0.0273973); expected 1.5 (0.02)\n",
      "Passed Fwd filter:                         1  (0.0136986); expected 1.5 (0.02)\n",
      "Initial search space (Z):                 73  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.14u 0.02s 00:00:00.16 Elapsed: 00:00:00.07\n",
      "# Mc/sec: 37732.22\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001832095.1_ASM183209v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001832095.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         29  (4934421 residues searched)\n",
      "Passed MSV filter:                         3  (0.103448); expected 0.6 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.6 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.6 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.6 (0.02)\n",
      "Initial search space (Z):                 29  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.15u 0.02s 00:00:00.17 Elapsed: 00:00:00.09\n",
      "# Mc/sec: 33630.22\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000186125.1_Bacillus_BT1B_CT2_V1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_000186125.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         24  (4369552 residues searched)\n",
      "Passed MSV filter:                         1  (0.0416667); expected 0.5 (0.02)\n",
      "Passed bias filter:                        1  (0.0416667); expected 0.5 (0.02)\n",
      "Passed Vit filter:                         1  (0.0416667); expected 0.5 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.5 (0.02)\n",
      "Initial search space (Z):                 24  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 3.31u 0.03s 00:00:03.34 Elapsed: 00:00:03.31\n",
      "# Mc/sec: 822.25\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_005041545.1_ASM504154v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_005041545.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.15    2.7   0.0       0.26    1.8   0.0    1.2  1  NZ_RRNU01000037.1  Escherichia coli strain FWSEC0397 FWSEC039\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_RRNU01000037.1  Escherichia coli strain FWSEC0397 FWSEC0397_contig00038, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    1.8   0.0    0.0035      0.26      62     108 ..   21682   21728 ..   21663   21748 .. 0.91\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 1.8 bits;  conditional E-value: 0.0035\n",
      "        tp_nucl_aln    62 tgcgggtcagggcgctggAaacgcttctgattgaaAAgGGgctgatc 108  \n",
      "                           gc  gtcag+ cg tggA a   t+ctga+tgaaAA+G g t atc\n",
      "  NZ_RRNU01000037.1 21682 CGCTCGTCAGTACGATGGAGAGTATGCTGACTGAAAAAGAGATTATC 21728\n",
      "                          467889*********************************99998887 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         76  (4911063 residues searched)\n",
      "Passed MSV filter:                         7  (0.0921053); expected 1.5 (0.02)\n",
      "Passed bias filter:                        1  (0.0131579); expected 1.5 (0.02)\n",
      "Passed Vit filter:                         1  (0.0131579); expected 1.5 (0.02)\n",
      "Passed Fwd filter:                         1  (0.0131579); expected 1.5 (0.02)\n",
      "Initial search space (Z):                 76  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.18u 0.02s 00:00:00.20 Elapsed: 00:00:00.14\n",
      "# Mc/sec: 21687.76\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003344765.1_ASM334476v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003344765.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         65  (4233556 residues searched)\n",
      "Passed MSV filter:                        29  (0.446154); expected 1.3 (0.02)\n",
      "Passed bias filter:                        1  (0.0153846); expected 1.3 (0.02)\n",
      "Passed Vit filter:                         1  (0.0153846); expected 1.3 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.3 (0.02)\n",
      "Initial search space (Z):                 65  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.55u 0.01s 00:00:00.56 Elapsed: 00:00:00.46\n",
      "# Mc/sec: 5736.93\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001707555.1_ASM170755v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001707555.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         26  (4580631 residues searched)\n",
      "Passed MSV filter:                         2  (0.0769231); expected 0.5 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.5 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.5 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.5 (0.02)\n",
      "Initial search space (Z):                 26  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.14u 0.01s 00:00:00.15 Elapsed: 00:00:00.07\n",
      "# Mc/sec: 36617.84\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_017570045.1_ASM1757004v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_017570045.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (4971565 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.10u 0.01s 00:00:00.11 Elapsed: 00:00:00.11\n",
      "# Mc/sec: 27663.10\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002106175.1_ASM210617v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_002106175.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         70  (4951112 residues searched)\n",
      "Passed MSV filter:                         6  (0.0857143); expected 1.4 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 1.4 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 1.4 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.4 (0.02)\n",
      "Initial search space (Z):                 70  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.15u 0.01s 00:00:00.16 Elapsed: 00:00:00.08\n",
      "# Mc/sec: 36206.74\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_018449525.1_ASM1844952v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_018449525.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         30  (6038584 residues searched)\n",
      "Passed MSV filter:                        12  (0.4); expected 0.6 (0.02)\n",
      "Passed bias filter:                        1  (0.0333333); expected 0.6 (0.02)\n",
      "Passed Vit filter:                         1  (0.0333333); expected 0.6 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.6 (0.02)\n",
      "Initial search space (Z):                 30  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.44u 0.03s 00:00:00.47 Elapsed: 00:00:00.31\n",
      "# Mc/sec: 11866.24\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000400485.1_k0-k1250_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_000400485.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         50  (2878403 residues searched)\n",
      "Passed MSV filter:                        39  (0.78); expected 1.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 1.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 1.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.0 (0.02)\n",
      "Initial search space (Z):                 50  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.17u 0.00s 00:00:00.17 Elapsed: 00:00:00.10\n",
      "# Mc/sec: 17054.80\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004023985.1_ASM402398v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_004023985.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.91    2.0   0.0        1.3    1.5   0.0    1.1  1  NZ_QKOK01000215.1  Klebsiella pneumoniae strain GEO_49_Up_C N\n",
      "        1.4    1.5   0.0        5.2   -0.5   0.0    1.5  2  NZ_QKOK01000285.1  Klebsiella pneumoniae strain GEO_49_Up_C N\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_QKOK01000215.1  Klebsiella pneumoniae strain GEO_49_Up_C NODE_215_length_708_cov_10.2031, whole genome shotgun seq\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    1.5   0.0    0.0088       1.3       7      39 ..     281     313 ..     262     337 .. 0.81\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 1.5 bits;  conditional E-value: 0.0088\n",
      "        tp_nucl_aln   7 cAtGatcatgccgaccacgaccacccgcacgct 39 \n",
      "                         AtGat atgccg+c acga+ a+cc c c++t\n",
      "  NZ_QKOK01000215.1 281 TATGATGATGCCGGCAACGATAATCCACGCAAT 313\n",
      "                        589**********************99988776 PP\n",
      "\n",
      ">> NZ_QKOK01000285.1  Klebsiella pneumoniae strain GEO_49_Up_C NODE_285_length_246_cov_8.82353, whole genome shotgun seq\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.5   0.0     0.035       5.2     534     573 ..      44      81 ..      16     136 .. 0.61\n",
      "   2 ?   -0.6   0.0     0.038       5.7     542     582 ..     122     156 ..      87     189 .. 0.56\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.5 bits;  conditional E-value: 0.035\n",
      "        tp_nucl_aln 534 gAccGaaggtctGgacgaggaggcgctggccgcgctgGtc 573\n",
      "                        +AccG ag  + Gga+g  g gg +ctgg  gc  t Gtc\n",
      "  NZ_QKOK01000285.1  44 CACCGCAGAAAAGGATG--GCGGTTCTGGTTGCCTTTGTC 81 \n",
      "                        45555555555555543..444555555555555555543 PP\n",
      "\n",
      "  == domain 2  score: -0.6 bits;  conditional E-value: 0.038\n",
      "        tp_nucl_aln 542 gtctGgacgaggaggcgctggccgcgctgGtcAcccgcgAc 582\n",
      "                         tctGgac   g    gctggccg   t  tcA ccg+gAc\n",
      "  NZ_QKOK01000285.1 122 TTCTGGAC---GCCATGCTGGCCG---TTATCATCCGTGAC 156\n",
      "                        22222221...1222233333333...22344555555555 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        297  (6081898 residues searched)\n",
      "Passed MSV filter:                        29  (0.0976431); expected 5.9 (0.02)\n",
      "Passed bias filter:                        4  (0.013468); expected 5.9 (0.02)\n",
      "Passed Vit filter:                         4  (0.013468); expected 5.9 (0.02)\n",
      "Passed Fwd filter:                         2  (0.00673401); expected 5.9 (0.02)\n",
      "Initial search space (Z):                297  [actual number of targets]\n",
      "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
      "# CPU time: 0.19u 0.00s 00:00:00.19 Elapsed: 00:00:00.09\n",
      "# Mc/sec: 41684.30\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003351365.1_ASM335136v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003351365.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (7255264 residues searched)\n",
      "Passed MSV filter:                         1  (1); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.33u 0.05s 00:00:00.38 Elapsed: 00:00:00.39\n",
      "# Mc/sec: 11579.96\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_018282115.1_ASM1828211v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_018282115.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence      Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------      -----------\n",
      "  ------ inclusion threshold ------\n",
      "      0.094   -0.9   0.0       0.26   -2.4   0.0    1.3  1  NZ_CP054307.1  Synechocystis sp. PCC 7338 plasmid pSyn7338-1,\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_CP054307.1  Synechocystis sp. PCC 7338 plasmid pSyn7338-1, complete sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.4   0.0     0.066      0.26     194     255 ..   79599   79660 ..   79595   79686 .. 0.86\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.4 bits;  conditional E-value: 0.066\n",
      "    tp_nucl_aln   194 GGaccGAcccggagttccgggcgcggctgctggagGAtgccactgcgGcgatcgccgagctg 255  \n",
      "                      GG+c GA cc   gt + g+g+ cg+ctg tgg gGAtgc +c  + G +at+g  ga +tg\n",
      "  NZ_CP054307.1 79599 GGGCAGAACCTACGTCTGGCGATCGCCTGGTGGTGGATGCTGCCCAAGGTATTGATGAAATG 79660\n",
      "                      6888999999999*********************************9999999999888885 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          4  (3832803 residues searched)\n",
      "Passed MSV filter:                         1  (0.25); expected 0.1 (0.02)\n",
      "Passed bias filter:                        1  (0.25); expected 0.1 (0.02)\n",
      "Passed Vit filter:                         1  (0.25); expected 0.1 (0.02)\n",
      "Passed Fwd filter:                         1  (0.25); expected 0.1 (0.02)\n",
      "Initial search space (Z):                  4  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.34u 0.01s 00:00:00.35 Elapsed: 00:00:00.29\n",
      "# Mc/sec: 8195.31\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002216445.1_ASM221644v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_002216445.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         93  (6467958 residues searched)\n",
      "Passed MSV filter:                        43  (0.462366); expected 1.9 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 1.9 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 1.9 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.9 (0.02)\n",
      "Initial search space (Z):                 93  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.30u 0.03s 00:00:00.32 Elapsed: 00:00:00.17\n",
      "# Mc/sec: 23640.57\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004215335.1_ASM421533v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_004215335.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.28    2.3   0.0       0.66    1.1   0.0    1.4  1  NZ_RCGB01000043.1  Vibrio vulnificus strain VA-WGS-18024 NODE\n",
      "        1.3    0.1   0.0        2.3   -0.7   0.0    1.2  1  NZ_RCGB01000064.1  Vibrio vulnificus strain VA-WGS-18024 NODE\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_RCGB01000043.1  Vibrio vulnificus strain VA-WGS-18024 NODE_43_length_43121_cov_39.251617, whole genome shotgun seq\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    1.1   0.0     0.012      0.66     414     457 ..    6988    7031 ..    6977    7051 .. 0.79\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 1.1 bits;  conditional E-value: 0.012\n",
      "        tp_nucl_aln  414 ccgcGAaccgcgcagcGtgctggcgGAgtttGGcgtcaccctGc 457 \n",
      "                         +c +GAa  gc c+g Gtgct gcgGA+tt+GGcgt ++  tG \n",
      "  NZ_RCGB01000043.1 6988 GCTTGAAAAGCACGGTGTGCTAGCGGAATTCGGCGTAGAAATGA 7031\n",
      "                         55566666688899**********************99877665 PP\n",
      "\n",
      ">> NZ_RCGB01000064.1  Vibrio vulnificus strain VA-WGS-18024 NODE_64_length_6767_cov_40.166717, whole genome shotgun sequ\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.7   0.0     0.042       2.3     108     150 ..    3029    3071 ..    3023    3078 .. 0.78\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.7 bits;  conditional E-value: 0.042\n",
      "        tp_nucl_aln  108 cgatcccgcagcgctggatgagatcatcgacacctatgagaac 150 \n",
      "                         +ga  c g  +  +tggatgag tcatcga+a +ta+ga aac\n",
      "  NZ_RCGB01000064.1 3029 TGAAGCAGGTAGTGTGGATGAGCTCATCGAAATTTACGACAAC 3071\n",
      "                         5555555555556799**********************99886 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        111  (4941069 residues searched)\n",
      "Passed MSV filter:                         3  (0.027027); expected 2.2 (0.02)\n",
      "Passed bias filter:                        3  (0.027027); expected 2.2 (0.02)\n",
      "Passed Vit filter:                         3  (0.027027); expected 2.2 (0.02)\n",
      "Passed Fwd filter:                         2  (0.018018); expected 2.2 (0.02)\n",
      "Initial search space (Z):                111  [actual number of targets]\n",
      "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
      "# CPU time: 0.35u 0.02s 00:00:00.37 Elapsed: 00:00:00.22\n",
      "# Mc/sec: 13811.75\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002881895.1_GW704-F2_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_002881895.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "      0.077    5.8   0.1      0.091    5.5   0.1    1.1  1  NZ_POEC01000307.1  Pseudomonas sp. GW704-F2 NODE_311_length_2\n",
      "        1.6    1.4   0.0          3    0.5   0.0    1.4  1  NZ_POEC01000123.1  Pseudomonas sp. GW704-F2 NODE_123_length_1\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_POEC01000307.1  Pseudomonas sp. GW704-F2 NODE_311_length_247_cov_0.782353, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    5.5   0.1   0.00054     0.091     212     325 ..      71     185 ..      21     192 .. 0.64\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 5.5 bits;  conditional E-value: 0.00054\n",
      "        tp_nucl_aln 212 gggcgcggctgctggagGAtgccactgcgGcgatcgccgagctgGggttt......tctggacgtcaggggGAacAtAtggtgGcggtggag 297\n",
      "                        +gg+ ++gctg tggag       c g+ Gcg tcgc g     G  t+t       c       caggg GA c +At   +G+gg +   \n",
      "  NZ_POEC01000307.1  71 AGGATGCGCTGATGGAG------TCGGAAGCGTTCGCGGGCGGCGAATATctcatcGCGCTTGCGCAGGGAGACCTCATCCCCGTGGCCCGC 156\n",
      "                        45555555555555432......2233334444444333333333333331111111111223469************************** PP\n",
      "\n",
      "        tp_nucl_aln 298 AAcacggccgaggtgcAcAAtatg.gtcg 325\n",
      "                        AA acgg+    g gcAcAA +tg gtcg\n",
      "  NZ_POEC01000307.1 157 AAAACGGAGATCGCGCACAAGCTGaGTCG 185\n",
      "                        ********************998734555 PP\n",
      "\n",
      ">> NZ_POEC01000123.1  Pseudomonas sp. GW704-F2 NODE_123_length_1506_cov_79.6116, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    0.5   0.0     0.018         3     278     306 ..     497     525 ..     369     534 .. 0.88\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 0.5 bits;  conditional E-value: 0.018\n",
      "        tp_nucl_aln 278 AacAtAtggtgGcggtggagAAcacggcc 306\n",
      "                        A cAtAt  tgGc gt+gagAAca  gcc\n",
      "  NZ_POEC01000123.1 497 AGCATATCCTGGCTGTCGAGAACATTGCC 525\n",
      "                        458999***************99866555 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        335  (6680317 residues searched)\n",
      "Passed MSV filter:                        61  (0.18209); expected 6.7 (0.02)\n",
      "Passed bias filter:                        4  (0.0119403); expected 6.7 (0.02)\n",
      "Passed Vit filter:                         4  (0.0119403); expected 6.7 (0.02)\n",
      "Passed Fwd filter:                         2  (0.00597015); expected 6.7 (0.02)\n",
      "Initial search space (Z):                335  [actual number of targets]\n",
      "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
      "# CPU time: 0.21u 0.04s 00:00:00.25 Elapsed: 00:00:00.12\n",
      "# Mc/sec: 33092.84\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000979595.1_gtlEnvA5udCFS_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_000979595.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "      0.091    4.5   0.1       0.39    2.4   0.0    1.9  2  NZ_JJPJ01000056.1  Methanosarcina mazei strain 3.F.T.1A.2 sca\n",
      "       0.49    2.1   0.0        1.7    0.2   0.0    1.6  2  NZ_JJPJ01000009.1  Methanosarcina mazei strain 3.F.T.1A.2 sca\n",
      "       0.98    1.1   0.0        1.8    0.2   0.0    1.2  1  NZ_JJPJ01000091.1  Methanosarcina mazei strain 3.F.T.1A.2 sca\n",
      "          1    1.0   0.0        1.7    0.2   0.0    1.2  1  NZ_JJPJ01000029.1  Methanosarcina mazei strain 3.F.T.1A.2 sca\n",
      "        3.5   -0.8   0.0        7.3   -1.8   0.0    1.2  1  NZ_JJPJ01000162.1  Methanosarcina mazei strain 3.F.T.1A.2 sca\n",
      "          4   -0.9   0.0        8.1   -2.0   0.0    1.2  1  NZ_JJPJ01000031.1  Methanosarcina mazei strain 3.F.T.1A.2 sca\n",
      "        4.4   -1.1   0.0        7.5   -1.9   0.0    1.1  1  NZ_JJPJ01000076.1  Methanosarcina mazei strain 3.F.T.1A.2 sca\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_JJPJ01000056.1  Methanosarcina mazei strain 3.F.T.1A.2 scaffold162_1_size630-refined, whole genome shotgun sequenc\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.5   0.0      0.26       6.1     155     186 ..     289     322 ..     268     348 .. 0.84\n",
      "   2 ?    2.4   0.0     0.017      0.39     541     581 ..     536     577 ..     518     581 .. 0.81\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.5 bits;  conditional E-value: 0.26\n",
      "        tp_nucl_aln 155 tcGgcccgcggAAc..GGtGcccacgtgGtgGcg 186\n",
      "                        tcG +ccg  +AAc  GGtGcc +cgtgGtgG g\n",
      "  NZ_JJPJ01000056.1 289 TCGCGCCGATCAACtgGGTGCCAGCGTGGTGGTG 322\n",
      "                        5666789999999877999***********9975 PP\n",
      "\n",
      "  == domain 2  score: 2.4 bits;  conditional E-value: 0.017\n",
      "        tp_nucl_aln 541 ggtctGgacgaggaggcgctggccgcgctg.GtcAcccgcgA 581\n",
      "                         g ctGg+cg ggag ++ctgg cgc  tg GtcAcc gc A\n",
      "  NZ_JJPJ01000056.1 536 CGACTGGGCGTGGAGCATCTGGTCGCATTGgGTCACCAGCAA 577\n",
      "                        57899*********************9996368888766655 PP\n",
      "\n",
      ">> NZ_JJPJ01000009.1  Methanosarcina mazei strain 3.F.T.1A.2 scaffold124_1_size4807-refined, whole genome shotgun sequen\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.0   0.0      0.35       8.3       8      88 ..    3515    3604 ..    3513    3668 .. 0.68\n",
      "   2 ?    0.2   0.0     0.075       1.7      45      88 ..    4495    4543 ..    4487    4548 .. 0.80\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.0 bits;  conditional E-value: 0.35\n",
      "        tp_nucl_aln    8 AtGatcatgccgaccacgaccacccgcacgctcttttgccgccggaaactgctttgcggg...........tcagggcgctggAaacgct 86  \n",
      "                         AtGa c t  +gac acga+c +  +c   c+c++  gc   +  a ac gctt    g            t a+ggcgct gAaa g t\n",
      "  NZ_JJPJ01000009.1 3515 ATGAGCCTCTTGACTACGATCTTTTTCGGTCCCAG--GCTAGAAAATACGGCTTCAATGAggaggaatacaTAAAGGCGCTTGAAAAGGT 3602\n",
      "                         67888888888888888888888888887777776..66666667777777777654321111222222227778888888888888877 PP\n",
      "\n",
      "        tp_nucl_aln   87 tc 88  \n",
      "                          c\n",
      "  NZ_JJPJ01000009.1 3603 CC 3604\n",
      "                         66 PP\n",
      "\n",
      "  == domain 2  score: 0.2 bits;  conditional E-value: 0.075\n",
      "        tp_nucl_aln   45 gccgccg.....gaaactgctttgcgggtcagggcgctggAaacgcttc 88  \n",
      "                         gc gc g     ga a++ c++  cg    a+ggcgctggAaacgcttc\n",
      "  NZ_JJPJ01000009.1 4495 GCTGCTGcacgcGATATCACCGAACGTAAGAAGGCGCTGGAAACGCTTC 4543\n",
      "                         4444333333669999999***9***********************986 PP\n",
      "\n",
      ">> NZ_JJPJ01000091.1  Methanosarcina mazei strain 3.F.T.1A.2 scaffold3_1_size91717-refined, whole genome shotgun sequenc\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    0.2   0.0     0.076       1.8      47     107 ..   20363   20423 ..   20344   20429 .. 0.83\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 0.2 bits;  conditional E-value: 0.076\n",
      "        tp_nucl_aln    47 cgccggaaactgctttgcgggtcagggcgctggAaacgcttctgattgaaAAgGGgctgat 107  \n",
      "                             +gga++c g ++tgc ggt +    gctg Aaa  ct+ctg  + aaAAgGG +tgat\n",
      "  NZ_JJPJ01000091.1 20363 TATAGGAGCCGGTCCTGCAGGTTCCATTGCTGCAAAAACTGCTGCGGAAAAAGGGCTTGAT 20423\n",
      "                          4456799************************************977777777777777766 PP\n",
      "\n",
      ">> NZ_JJPJ01000029.1  Methanosarcina mazei strain 3.F.T.1A.2 scaffold36_1_size42830-refined, whole genome shotgun sequen\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    0.2   0.0     0.075       1.7      83     146 ..   33650   33715 ..   33606   33736 .. 0.77\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 0.2 bits;  conditional E-value: 0.075\n",
      "        tp_nucl_aln    83 cgcttctgat..tgaaAAgGGgctgatcgatcccgcagcgctggatgagatcatcgacacctatga 146  \n",
      "                          cg t  tgat  +ga AA+   c gatcga    g+agc+ctg+a g gatcat gaca  ta+  \n",
      "  NZ_JJPJ01000029.1 33650 CGGTCTTGATgcCGATAAACTCCAGATCGAAGATGAAGCCCTGAAGGTGATCATTGACAAATACAC 33715\n",
      "                          4556666655114677777778999**********************************9998754 PP\n",
      "\n",
      ">> NZ_JJPJ01000162.1  Methanosarcina mazei strain 3.F.T.1A.2 scaffold20_1_size87324-refined, whole genome shotgun sequen\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.8   0.0      0.31       7.3     433     467 ..   38806   38840 ..   38792   38846 .. 0.88\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.8 bits;  conditional E-value: 0.31\n",
      "        tp_nucl_aln   433 ctggcgGAgtttGGcgtcaccctGcccgatgacat 467  \n",
      "                           t g  GA+tt+GGc tca+ +tGcc+ga+g+ca+\n",
      "  NZ_JJPJ01000162.1 38806 ATCGGAGAATTCGGCCTCAAATTGCCTGAAGGCAC 38840\n",
      "                          577889************************99876 PP\n",
      "\n",
      ">> NZ_JJPJ01000031.1  Methanosarcina mazei strain 3.F.T.1A.2 scaffold38_1_size33828-refined, whole genome shotgun sequen\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.0   0.0      0.35       8.1      91     141 ..   30403   30453 ..   30331   30472 .. 0.79\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.0 bits;  conditional E-value: 0.35\n",
      "        tp_nucl_aln    91 attgaaAAgGGgctgatcgatcccgcagcgctggatgagatcatcgacacc 141  \n",
      "                          attgaaAAgGG +  at  a   cgc gc +t +atg + t+atc  ca c\n",
      "  NZ_JJPJ01000031.1 30403 ATTGAAAAGGGCTACATAAAAAACGCGGCAGTTAATGTAGTTATCTCCAAC 30453\n",
      "                          6899**************************************998766655 PP\n",
      "\n",
      ">> NZ_JJPJ01000076.1  Methanosarcina mazei strain 3.F.T.1A.2 scaffold129_1_size3200-refined, whole genome shotgun sequen\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.9   0.0      0.32       7.5      77     118 ..    2773    2814 ..    2728    2833 .. 0.89\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.9 bits;  conditional E-value: 0.32\n",
      "        tp_nucl_aln   77 tggAaacgcttctgattgaaAAgGGgctgatcgatcccgcag 118 \n",
      "                         tggAa    tt tgat+gaaAA+G ++  at  a+cccg+a+\n",
      "  NZ_JJPJ01000076.1 2773 TGGAACATATTATGATGGAAAAAGTTTCCATAAACCCCGAAA 2814\n",
      "                         7999999******************9999**99999999865 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        163  (4157971 residues searched)\n",
      "Passed MSV filter:                        11  (0.0674847); expected 3.3 (0.02)\n",
      "Passed bias filter:                       11  (0.0674847); expected 3.3 (0.02)\n",
      "Passed Vit filter:                        11  (0.0674847); expected 3.3 (0.02)\n",
      "Passed Fwd filter:                         7  (0.0429448); expected 3.3 (0.02)\n",
      "Initial search space (Z):                163  [actual number of targets]\n",
      "Domain search space  (domZ):               7  [number of targets reported over threshold]\n",
      "# CPU time: 1.06u 0.03s 00:00:01.09 Elapsed: 00:00:00.56\n",
      "# Mc/sec: 4594.94\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_900167585.1_16852_2_94_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_900167585.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         28  (4098030 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.6 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.6 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.6 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.6 (0.02)\n",
      "Initial search space (Z):                 28  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.09u 0.00s 00:00:00.09 Elapsed: 00:00:00.05\n",
      "# Mc/sec: 48978.04\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006517585.1_ASM651758v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_006517585.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        159  (5323630 residues searched)\n",
      "Passed MSV filter:                        45  (0.283019); expected 3.2 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 3.2 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 3.2 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 3.2 (0.02)\n",
      "Initial search space (Z):                159  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.18u 0.01s 00:00:00.19 Elapsed: 00:00:00.10\n",
      "# Mc/sec: 31329.28\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_010983895.1_ASM1098389v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_010983895.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (4236644 residues searched)\n",
      "Passed MSV filter:                         1  (1); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.20u 0.04s 00:00:00.24 Elapsed: 00:00:00.24\n",
      "# Mc/sec: 10932.56\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004745955.1_ASM474595v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_004745955.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        137  (4971571 residues searched)\n",
      "Passed MSV filter:                        24  (0.175182); expected 2.7 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 2.7 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 2.7 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 2.7 (0.02)\n",
      "Initial search space (Z):                137  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.14u 0.00s 00:00:00.14 Elapsed: 00:00:00.07\n",
      "# Mc/sec: 41608.56\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002883245.1_GW101-3F01_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_002883245.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        127  (7061489 residues searched)\n",
      "Passed MSV filter:                        29  (0.228346); expected 2.5 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 2.5 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 2.5 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 2.5 (0.02)\n",
      "Initial search space (Z):                127  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.25u 0.01s 00:00:00.26 Elapsed: 00:00:00.13\n",
      "# Mc/sec: 32202.56\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002355535.1_ASM235553v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_002355535.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (1622386 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.05u 0.00s 00:00:00.05 Elapsed: 00:00:00.05\n",
      "# Mc/sec: 19435.61\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002103335.1_ASM210333v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_002103335.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         38  (3130626 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.8 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.8 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.8 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.8 (0.02)\n",
      "Initial search space (Z):                 38  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.10u 0.00s 00:00:00.10 Elapsed: 00:00:00.06\n",
      "# Mc/sec: 29176.65\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004921205.1_ASM492120v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_004921205.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         20  (3389277 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.4 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.4 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.4 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.4 (0.02)\n",
      "Initial search space (Z):                 20  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.08u 0.00s 00:00:00.08 Elapsed: 00:00:00.05\n",
      "# Mc/sec: 42156.76\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002241775.2_ASM224177v2_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_002241775.2_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         61  (5216394 residues searched)\n",
      "Passed MSV filter:                         5  (0.0819672); expected 1.2 (0.02)\n",
      "Passed bias filter:                        1  (0.0163934); expected 1.2 (0.02)\n",
      "Passed Vit filter:                         1  (0.0163934); expected 1.2 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.2 (0.02)\n",
      "Initial search space (Z):                 61  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.16u 0.01s 00:00:00.17 Elapsed: 00:00:00.08\n",
      "# Mc/sec: 36226.63\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_012224465.1_ASM1222446v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_012224465.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (2054227 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.06u 0.00s 00:00:00.06 Elapsed: 00:00:00.06\n",
      "# Mc/sec: 19766.69\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_013790265.1_ASM1379026v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_013790265.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (5066655 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.11u 0.01s 00:00:00.12 Elapsed: 00:00:00.13\n",
      "# Mc/sec: 24070.51\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_017696275.1_ASM1769627v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_017696275.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          3  (1672276 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.1 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.1 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.1 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.1 (0.02)\n",
      "Initial search space (Z):                  3  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.05u 0.00s 00:00:00.05 Elapsed: 00:00:00.04\n",
      "# Mc/sec: 22976.50\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003854195.1_ASM385419v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003854195.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        129  (8060335 residues searched)\n",
      "Passed MSV filter:                        58  (0.449612); expected 2.6 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 2.6 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 2.6 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 2.6 (0.02)\n",
      "Initial search space (Z):                129  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.34u 0.01s 00:00:00.35 Elapsed: 00:00:00.18\n",
      "# Mc/sec: 27127.80\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_015276975.1_ASM1527697v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_015276975.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         25  (2850790 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.5 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.5 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.5 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.5 (0.02)\n",
      "Initial search space (Z):                 25  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.09u 0.01s 00:00:00.10 Elapsed: 00:00:00.05\n",
      "# Mc/sec: 31816.57\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002119585.1_ASM211958v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_002119585.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          2  (5389178 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  2  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.11u 0.01s 00:00:00.12 Elapsed: 00:00:00.08\n",
      "# Mc/sec: 39999.47\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_017349495.1_ASM1734949v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_017349495.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        205  (7384011 residues searched)\n",
      "Passed MSV filter:                        52  (0.253659); expected 4.1 (0.02)\n",
      "Passed bias filter:                        1  (0.00487805); expected 4.1 (0.02)\n",
      "Passed Vit filter:                         1  (0.00487805); expected 4.1 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 4.1 (0.02)\n",
      "Initial search space (Z):                205  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.30u 0.01s 00:00:00.31 Elapsed: 00:00:00.18\n",
      "# Mc/sec: 24255.59\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_015680345.1_ASM1568034v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_015680345.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         57  (5387473 residues searched)\n",
      "Passed MSV filter:                         5  (0.0877193); expected 1.1 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 1.1 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 1.1 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.1 (0.02)\n",
      "Initial search space (Z):                 57  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.19u 0.03s 00:00:00.22 Elapsed: 00:00:00.11\n",
      "# Mc/sec: 29202.31\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003096015.2_ASM309601v2_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003096015.2_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (3841771 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.09u 0.00s 00:00:00.09 Elapsed: 00:00:00.08\n",
      "# Mc/sec: 27246.94\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_009790425.1_ASM979042v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_009790425.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.14    4.3   0.0       0.23    3.5   0.0    1.2  1  NZ_WSVY01000128.1  Escherichia coli strain OM16c NODE_128_len\n",
      "        1.8    0.6   0.0        4.2   -0.6   0.0    1.3  1  NZ_WSVY01000088.1  Escherichia coli strain OM16c NODE_88_leng\n",
      "        5.8   -1.1   0.0         11   -2.0   0.0    1.1  1  NZ_WSVY01000148.1  Escherichia coli strain OM16c NODE_148_len\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_WSVY01000128.1  Escherichia coli strain OM16c NODE_128_length_8052_cov_2.014132, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    3.5   0.0    0.0033      0.23      62     108 ..    6938    6984 ..    6908    7004 .. 0.88\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 3.5 bits;  conditional E-value: 0.0033\n",
      "        tp_nucl_aln   62 tgcgggtcagggcgctggAaacgcttctgattgaaAAgGGgctgatc 108 \n",
      "                          gc  gt ag+ cg tggA a   t+ctga+tgaaAA+G g t atc\n",
      "  NZ_WSVY01000128.1 6938 CGCTCGTTAGTACGATGGAGAGTATGCTGACTGAAAAAGAGATTATC 6984\n",
      "                         5778899********************************99998887 PP\n",
      "\n",
      ">> NZ_WSVY01000088.1  Escherichia coli strain OM16c NODE_88_length_14174_cov_2.046914, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.6   0.0     0.058       4.2     269     321 ..    5471    5518 ..    5460    5524 .. 0.86\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.6 bits;  conditional E-value: 0.058\n",
      "        tp_nucl_aln  269 gtcaggggGAacAtAtggtgGcggtggagAAcacggccgaggtgcAcAAtatg 321 \n",
      "                         gtcag g     AtAt gtgGcg tgga+AAc c + +ga  tg AcAA +t \n",
      "  NZ_WSVY01000088.1 5471 GTCAGCG-----ATATTGTGGCGCTGGAAAACGCCCTTGATATGTACAAACTC 5518\n",
      "                         5555554.....899********************************998776 PP\n",
      "\n",
      ">> NZ_WSVY01000148.1  Escherichia coli strain OM16c NODE_148_length_4626_cov_1.451211, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.0   0.0      0.16        11     158     187 ..    2060    2089 ..    2037    2118 .. 0.83\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.0 bits;  conditional E-value: 0.16\n",
      "        tp_nucl_aln  158 gcccgcggAAcGGtGcccacgtgGtgGcgc 187 \n",
      "                         gccc  + AA+GGtGcc++c tgGt Gc c\n",
      "  NZ_WSVY01000148.1 2060 GCCCAGAAAATGGTGCCGCCATGGTTGCTC 2089\n",
      "                         577888899******************987 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        215  (4652790 residues searched)\n",
      "Passed MSV filter:                        18  (0.0837209); expected 4.3 (0.02)\n",
      "Passed bias filter:                        6  (0.027907); expected 4.3 (0.02)\n",
      "Passed Vit filter:                         6  (0.027907); expected 4.3 (0.02)\n",
      "Passed Fwd filter:                         3  (0.0139535); expected 4.3 (0.02)\n",
      "Initial search space (Z):                215  [actual number of targets]\n",
      "Domain search space  (domZ):               3  [number of targets reported over threshold]\n",
      "# CPU time: 0.40u 0.00s 00:00:00.40 Elapsed: 00:00:00.27\n",
      "# Mc/sec: 10668.48\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_014526385.1_ASM1452638v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_014526385.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence             Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------             -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.26    2.2   0.1       0.55    1.1   0.1    1.4  1  NZ_JABEXQ010000029.1  Escherichia coli strain ECR.1 ECR1-N733\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_JABEXQ010000029.1  Escherichia coli strain ECR.1 ECR1-N733_trimmed_(paired)_contig_29, whole genome shotgun sequen\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    1.1   0.1     0.006      0.55     541     585 ..   62000   62044 ..   61993   62093 .. 0.82\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 1.1 bits;  conditional E-value: 0.006\n",
      "           tp_nucl_aln   541 ggtctGgacgaggaggcgctggccgcgctgGtcAcccgcgAcagc 585  \n",
      "                             gg ctG+a+ga g  gcgctggc+gcg  gGtcA cc +gAcagc\n",
      "  NZ_JABEXQ010000029.1 62000 GGCCTGAATGAAGGTGCGCTGGCGGCGACGGTCAGCCATGACAGC 62044\n",
      "                             68899************************************9975 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         92  (5154350 residues searched)\n",
      "Passed MSV filter:                         5  (0.0543478); expected 1.8 (0.02)\n",
      "Passed bias filter:                        1  (0.0108696); expected 1.8 (0.02)\n",
      "Passed Vit filter:                         1  (0.0108696); expected 1.8 (0.02)\n",
      "Passed Fwd filter:                         1  (0.0108696); expected 1.8 (0.02)\n",
      "Initial search space (Z):                 92  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.34u 0.02s 00:00:00.36 Elapsed: 00:00:00.27\n",
      "# Mc/sec: 11596.86\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_009904725.1_ASM990472v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_009904725.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "        2.1    1.6   0.0        3.4    0.9   0.0    1.1  1  NZ_WXWT01000386.1  Vibrio sp. V43_P6S15P86 scaffold79_size153\n",
      "          3    1.0   0.0        5.2    0.3   0.0    1.2  1  NZ_WXWT01000439.1  Vibrio sp. V43_P6S15P86 scaffold179_size92\n",
      "        3.4    0.9   0.0         16   -1.4   0.0    1.6  3  NZ_WXWT01000462.1  Vibrio sp. V43_P6S15P86 scaffold46_size200\n",
      "        3.6    0.8   0.0        6.3   -0.0   0.0    1.2  1  NZ_WXWT01000117.1  Vibrio sp. V43_P6S15P86 scaffold15_size307\n",
      "        6.6   -0.1   0.0         14   -1.1   0.0    1.2  1  NZ_WXWT01000414.1  Vibrio sp. V43_P6S15P86 scaffold148_size10\n",
      "        8.6   -0.5   0.0         17   -1.4   0.0    1.2  1  NZ_WXWT01000296.1  Vibrio sp. V43_P6S15P86 scaffold266_size63\n",
      "        8.7   -0.5   0.0         16   -1.3   0.0    1.1  1  NZ_WXWT01000052.1  Vibrio sp. V43_P6S15P86 scaffold349_size43\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_WXWT01000386.1  Vibrio sp. V43_P6S15P86 scaffold79_size15346, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    0.9   0.0     0.048       3.4      69     145 ..   12356   12423 ..   12317   12438 .. 0.74\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 0.9 bits;  conditional E-value: 0.048\n",
      "        tp_nucl_aln    69 cagggcgctggAaacgcttctgattgaaAAgGGgctgatcgatcccgcagcgctggatgagatcatcgacacctatg 145  \n",
      "                           ++ gcgctg Aa+ g t  tg t  aa A+GGg+tgatcgat    ++g   t gatga+at+         ta+g\n",
      "  NZ_WXWT01000386.1 12356 GCAAGCGCTGCAAGAGTTCTTGTTAAAAGAAGGGTTGATCGATATAAACGGTATTGATGAAATT---------TACG 12423\n",
      "                          46789***********999999999999999***********9865554444455555555444.........4444 PP\n",
      "\n",
      ">> NZ_WXWT01000439.1  Vibrio sp. V43_P6S15P86 scaffold179_size9296, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    0.3   0.0     0.074       5.2     109     200 ..    1943    2031 ..    1932    2046 .. 0.72\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 0.3 bits;  conditional E-value: 0.074\n",
      "        tp_nucl_aln  109 gatcccgcagcgctggatgagatcatcgacacctatgaga..acaagatcGgcccgcggAAcGGtGcccacgtgGtgGcgcgggcctGGa 196 \n",
      "                         gat  c   gcg  ggatg+gat+a   a a + at ag    c a     g cc    AA+GGtGcc   g  Gt Gc cg gc+t Ga\n",
      "  NZ_WXWT01000439.1 1943 GATAGCATGGCGTCGGATGCGATTAATCAGAGTCATCAGTttTCTAA----G-CCTGCAAATGGTGCCATTGCCGTAGCTCGTGCATTGA 2027\n",
      "                         56666666677777777777777765555554444444330033332....3.3444469**************************9888 PP\n",
      "\n",
      "        tp_nucl_aln  197 ccGA 200 \n",
      "                           GA\n",
      "  NZ_WXWT01000439.1 2028 GAGA 2031\n",
      "                         8777 PP\n",
      "\n",
      ">> NZ_WXWT01000462.1  Vibrio sp. V43_P6S15P86 scaffold46_size20005, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -3.8   0.0       1.3        90     211     293 ..   10382   10467 ..   10371   10476 .. 0.75\n",
      "   2 ?   -1.4   0.0      0.23        16     182     243 ..   12261   12322 ..   12250   12366 .. 0.72\n",
      "   3 ?   -3.8   0.0       1.3        91     176     248 ..   12598   12666 ..   12530   12675 .. 0.73\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -3.8 bits;  conditional E-value: 1.3\n",
      "        tp_nucl_aln   211 cgggcgcggctgctggagGA........tgccactgcgGcgatcgccgagctgGggttttctggacgtcaggggGAacAtAtggtgGc 290  \n",
      "                          cg  +g+ +ctg+t ga GA        tgcc  tgc+G++a cgccg     Gg t+tt      g ca gg GAacA At  t G \n",
      "  NZ_WXWT01000462.1 10382 CGTAAGTACCTGTTCGATGAacttcgttTGCCTGTGCTGTTAGCGCCGTTGGCGGATATTT-----GGCAAGGTGAACAAATTATTGA 10464\n",
      "                          5566666777888888888777666666789999999**********99999999998875.....679**********999777665 PP\n",
      "\n",
      "        tp_nucl_aln   291 ggt 293  \n",
      "                           g \n",
      "  NZ_WXWT01000462.1 10465 TGG 10467\n",
      "                          444 PP\n",
      "\n",
      "  == domain 2  score: -1.4 bits;  conditional E-value: 0.23\n",
      "        tp_nucl_aln   182 tgGcgcgggcctGGaccGAcccggagttccgggcgcggctgctggagGAtgccactgcgGcg 243  \n",
      "                          t Gc cg g +tGG   GA+ c gagtt+    cg+  c g t g+ GA+g+ +  g+ G  \n",
      "  NZ_WXWT01000462.1 12261 TTGCTCGCGTATGGCATGATGCTGAGTTTGTTACGGTTCCGGTCGCAGACGGTGGCGAAGGC 12322\n",
      "                          6899**********************************************998877765533 PP\n",
      "\n",
      "  == domain 3  score: -3.8 bits;  conditional E-value: 1.3\n",
      "        tp_nucl_aln   176 acgtgGtgGcgcgggcctGGaccGAcccggagttccgggcgcggctgctggagGAtgccactgcgGcgatcgc 248  \n",
      "                             tgGtgGcg gggc+tG  + GAc  +g gtt  ggg g++g+t ct ga GA  c + t ++Gc at+ c\n",
      "  NZ_WXWT01000462.1 12598 TGATGGTGGCGTGGGCATG--TTGAC--CGCGTTGGGGGTGGCGTTTCTTGATGAGTCTGGTCATGCAATTAC 12666\n",
      "                          34599*************8..78886..5999**********************9999999988888887765 PP\n",
      "\n",
      ">> NZ_WXWT01000117.1  Vibrio sp. V43_P6S15P86 scaffold15_size30703, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.0   0.0     0.089       6.3     436     462 ..   26167   26193 ..   26161   26211 .. 0.87\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.0 bits;  conditional E-value: 0.089\n",
      "        tp_nucl_aln   436 gcgGAgtttGGcgtcaccctGcccgat 462  \n",
      "                          g++GAgtttGGcgtca++ctGcc ga+\n",
      "  NZ_WXWT01000117.1 26167 GATGAGTTTGGCGTCATTCTGCCAGAA 26193\n",
      "                          5789*******************9985 PP\n",
      "\n",
      ">> NZ_WXWT01000414.1  Vibrio sp. V43_P6S15P86 scaffold148_size10684, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.1   0.0      0.19        14     192     245 ..    9456    9505 ..    9454    9515 .. 0.85\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.1 bits;  conditional E-value: 0.19\n",
      "        tp_nucl_aln  192 ctGGaccGAcccggagttccgggcgcggctgctggagGAtgccactgcgGcgat 245 \n",
      "                         ctGGa  GA+cc gagtt+cg+gc + ++tg+t     At c a tgc+Gc+at\n",
      "  NZ_WXWT01000414.1 9456 CTGGAAAGATCCTGAGTTTCGCGCCTTCTTGTTT----ATTCAAGTGCTGCTAT 9505\n",
      "                         5899*******************99999999887....6777777777777765 PP\n",
      "\n",
      ">> NZ_WXWT01000296.1  Vibrio sp. V43_P6S15P86 scaffold266_size6391, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.4   0.0      0.24        17     182     217 ..    1495    1530 ..    1470    1542 .. 0.84\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.4 bits;  conditional E-value: 0.24\n",
      "        tp_nucl_aln  182 tgGcgcgggcctGGaccGAcccggagttccgggcgc 217 \n",
      "                         tgG  cg gcctGG c GA+ c+ga  tc  +g+++\n",
      "  NZ_WXWT01000296.1 1495 TGGTTCGAGCCTGGTCGGATGCCGAACTCGCCGAAG 1530\n",
      "                         78999*******************998887777655 PP\n",
      "\n",
      ">> NZ_WXWT01000052.1  Vibrio sp. V43_P6S15P86 scaffold349_size4364, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.3   0.0      0.22        16     561     605 ..     331     375 ..     324     388 .. 0.85\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.3 bits;  conditional E-value: 0.22\n",
      "        tp_nucl_aln 561 ggccgcgctgGtcAcccgcgAcagcAtGatcGGtacgGggctggc 605\n",
      "                         g    + tgGtcAc c + Ac+ c tGat GG +  G+g t gc\n",
      "  NZ_WXWT01000052.1 331 CGTTAATTTGGTCACACCTAACTACGTGATTGGGGTTGCGATTGC 375\n",
      "                        555666779*******************************99999 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        494  (4845908 residues searched)\n",
      "Passed MSV filter:                        17  (0.034413); expected 9.9 (0.02)\n",
      "Passed bias filter:                       17  (0.034413); expected 9.9 (0.02)\n",
      "Passed Vit filter:                        17  (0.034413); expected 9.9 (0.02)\n",
      "Passed Fwd filter:                        10  (0.0202429); expected 9.9 (0.02)\n",
      "Initial search space (Z):                494  [actual number of targets]\n",
      "Domain search space  (domZ):               7  [number of targets reported over threshold]\n",
      "# CPU time: 0.56u 0.01s 00:00:00.57 Elapsed: 00:00:00.32\n",
      "# Mc/sec: 9380.92\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006369235.1_ASM636923v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_006369235.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.96    0.1   0.0        2.5   -1.3   0.0    1.3  1  NZ_NNDW01000004.1  Vibrio parahaemolyticus strain F8_2 F8_2_c\n",
      "        2.2   -1.1   0.0        4.9   -2.3   0.0    1.2  1  NZ_NNDW01000033.1  Vibrio parahaemolyticus strain F8_2 F8_2_c\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_NNDW01000004.1  Vibrio parahaemolyticus strain F8_2 F8_2_contig004, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.3   0.0     0.062       2.5      70     149 ..  121688  121758 ..  121665  121770 .. 0.80\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.3 bits;  conditional E-value: 0.062\n",
      "        tp_nucl_aln     70 agggcgctggAaacgcttctgattgaaAAgGGgctgatcgatcccgcagcgctggatgagatcatcgacacctatgagaa 149   \n",
      "                           +++gc +tg Aa+ g tt tg t+ aa A+GG ctgat+gat  c ++g   t+ga+ga atc  cg           aa\n",
      "  NZ_NNDW01000004.1 121688 CATGCTTTGCAAGAGTTTTTGCTCAAAGAAGGCCTGATTGATATCAACGGTATCGACGACATCTACG---------CCAA 121758\n",
      "                           5678999***********************************9888777777777777766554444.........3333 PP\n",
      "\n",
      ">> NZ_NNDW01000033.1  Vibrio parahaemolyticus strain F8_2 F8_2_contig033, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.3   0.0      0.12       4.9     108     153 ..   20347   20392 ..   20333   20396 .. 0.83\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.3 bits;  conditional E-value: 0.12\n",
      "        tp_nucl_aln   108 cgatcccgcagcgctggatgagatcatcgacacctatgagaacaag 153  \n",
      "                           gat ccgc+gc ct+gatg+g t atc a a ct +  +  caa \n",
      "  NZ_NNDW01000033.1 20347 AGATACCGCCGCACTCGATGCGTTGATCAATAGCTTCTCAGGCAAT 20392\n",
      "                          47899***********************999988887766666665 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         80  (5076754 residues searched)\n",
      "Passed MSV filter:                         2  (0.025); expected 1.6 (0.02)\n",
      "Passed bias filter:                        2  (0.025); expected 1.6 (0.02)\n",
      "Passed Vit filter:                         2  (0.025); expected 1.6 (0.02)\n",
      "Passed Fwd filter:                         2  (0.025); expected 1.6 (0.02)\n",
      "Initial search space (Z):                 80  [actual number of targets]\n",
      "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
      "# CPU time: 0.76u 0.02s 00:00:00.78 Elapsed: 00:00:00.63\n",
      "# Mc/sec: 4994.03\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003449015.1_ASM344901v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003449015.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         51  (5108092 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 1.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 1.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 1.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.0 (0.02)\n",
      "Initial search space (Z):                 51  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.10u 0.01s 00:00:00.11 Elapsed: 00:00:00.05\n",
      "# Mc/sec: 56437.19\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_016863745.1_ASM1686374v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_016863745.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence             Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------             -----------\n",
      "  ------ inclusion threshold ------\n",
      "        0.8    0.8   0.0        1.2    0.2   0.0    1.2  1  NZ_JAFDUK010000076.1  Salmonella enterica subsp. enterica ser\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_JAFDUK010000076.1  Salmonella enterica subsp. enterica serovar Mountpleasant strain FW-S5-Mar NODE_110_length_296_\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    0.2   0.0     0.011       1.2      17      91 ..     311     379 ..     297     392 .] 0.62\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 0.2 bits;  conditional E-value: 0.011\n",
      "           tp_nucl_aln  17 ccgaccacgaccacccgcacgctcttttgccgccggaaactgctttgcgggtcagggcgctggAaacgcttctga 91 \n",
      "                           ccgaccacg + a+ cg a++       gc g   gaa+ +g    gc g t   ggc  tggAaac  t+ctg+\n",
      "  NZ_JAFDUK010000076.1 311 CCGACCACGTTAATACGTATAT------GCTGTTTGAACACGAGAAGCTGCTGGTGGCAATGGAAACCATGCTGG 379\n",
      "                           4455555555555555554444......44444444444444444444444444579999999999999999874 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        107  (5410873 residues searched)\n",
      "Passed MSV filter:                         7  (0.0654206); expected 2.1 (0.02)\n",
      "Passed bias filter:                        1  (0.00934579); expected 2.1 (0.02)\n",
      "Passed Vit filter:                         1  (0.00934579); expected 2.1 (0.02)\n",
      "Passed Fwd filter:                         1  (0.00934579); expected 2.1 (0.02)\n",
      "Initial search space (Z):                107  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.16u 0.01s 00:00:00.17 Elapsed: 00:00:00.09\n",
      "# Mc/sec: 34569.50\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001507335.1_ASM150733v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001507335.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         16  (3874551 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.3 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.3 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.3 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.3 (0.02)\n",
      "Initial search space (Z):                 16  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.09u 0.00s 00:00:00.09 Elapsed: 00:00:00.05\n",
      "# Mc/sec: 41604.20\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003710255.1_ASM371025v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003710255.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         12  (4178084 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.2 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.2 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.2 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.2 (0.02)\n",
      "Initial search space (Z):                 12  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.09u 0.01s 00:00:00.10 Elapsed: 00:00:00.05\n",
      "# Mc/sec: 44190.96\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003299525.1_ASM329952v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003299525.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.39    4.2   0.1        1.4    2.3   0.1    1.8  2  NZ_QNPM01000459.1  Xanthomonas oryzae pv. oryzae strain YN11 \n",
      "        7.7   -0.1   0.0         12   -0.8   0.0    1.1  1  NZ_QNPM01000315.1  Xanthomonas oryzae pv. oryzae strain YN11 \n",
      "        9.6   -0.5   0.0         28   -2.0   0.0    1.3  1  NZ_QNPM01000023.1  Xanthomonas oryzae pv. oryzae strain YN11 \n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_QNPM01000459.1  Xanthomonas oryzae pv. oryzae strain YN11 scaffold458, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    2.3   0.1    0.0075       1.4     547     610 ..     529     592 ..     519     612 .. 0.86\n",
      "   2 ?   -2.5   0.0      0.22        41      94     132 ..    1210    1248 ..    1181    1263 .. 0.84\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 2.3 bits;  conditional E-value: 0.0075\n",
      "        tp_nucl_aln 547 gacgaggaggcgctggccgcgctgGtcAcccgcgAcagcAtGatcGGtacgGggctggcgcagg 610\n",
      "                        g+cg+ gaggcgctgg c      GtcA c  cg c gc tGat GG+a gGg  t gcgc+g \n",
      "  NZ_QNPM01000459.1 529 GGCGCAGAGGCGCTGGTCAACACCGTCAACACCGTCGGCGTGATGGGCAAGGGCATCGCGCTGA 592\n",
      "                        568999**************999****99999999***********************977665 PP\n",
      "\n",
      "  == domain 2  score: -2.5 bits;  conditional E-value: 0.22\n",
      "        tp_nucl_aln   94 gaaAAgGGgctgatcgatcccgcagcgctggatgagatc 132 \n",
      "                         ga AA+ Gg t a cga+ ccg++ cgctgga+g gatc\n",
      "  NZ_QNPM01000459.1 1210 GACAAACGGATCAGCGACGCCGACCCGCTGGACGTGATC 1248\n",
      "                         566777788999999******************999875 PP\n",
      "\n",
      ">> NZ_QNPM01000315.1  Xanthomonas oryzae pv. oryzae strain YN11 scaffold314, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.8   0.0     0.064        12     284     308 ..      83     107 ..      72     121 .. 0.85\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.8 bits;  conditional E-value: 0.064\n",
      "        tp_nucl_aln 284 tggtgGcggtggagAAcacggccga 308\n",
      "                        tg t+G+gg  gagAAcac +cc a\n",
      "  NZ_QNPM01000315.1  83 TGATCGTGGCTGAGAACACTCCCTA 107\n",
      "                        78899*************9887755 PP\n",
      "\n",
      ">> NZ_QNPM01000023.1  Xanthomonas oryzae pv. oryzae strain YN11 scaffold22, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.0   0.0      0.15        28      88     118 ..    2208    2238 ..    2203    2275 .. 0.77\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.0 bits;  conditional E-value: 0.15\n",
      "        tp_nucl_aln   88 ctgattgaaAAgGGgctgatcgatcccgcag 118 \n",
      "                         ct a++g +AAgGG+ tgat+gat ccg+ag\n",
      "  NZ_QNPM01000023.1 2208 CTCACCGCGAAGGGTGTGATTGATACCGAAG 2238\n",
      "                         6889999*******************99865 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        556  (4328543 residues searched)\n",
      "Passed MSV filter:                       104  (0.18705); expected 11.1 (0.02)\n",
      "Passed bias filter:                        7  (0.0125899); expected 11.1 (0.02)\n",
      "Passed Vit filter:                         7  (0.0125899); expected 11.1 (0.02)\n",
      "Passed Fwd filter:                         3  (0.00539568); expected 11.1 (0.02)\n",
      "Initial search space (Z):                556  [actual number of targets]\n",
      "Domain search space  (domZ):               3  [number of targets reported over threshold]\n",
      "# CPU time: 0.18u 0.00s 00:00:00.18 Elapsed: 00:00:00.10\n",
      "# Mc/sec: 26475.72\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_900379955.1_ICMP_6278_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_900379955.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        416  (4170496 residues searched)\n",
      "Passed MSV filter:                       115  (0.276442); expected 8.3 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 8.3 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 8.3 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 8.3 (0.02)\n",
      "Initial search space (Z):                416  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.15u 0.00s 00:00:00.15 Elapsed: 00:00:00.08\n",
      "# Mc/sec: 30107.60\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_014893375.1_ASM1489337v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_014893375.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence             Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------             -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.42    3.3   0.0       0.68    2.6   0.0    1.2  1  NZ_JADBJU010000133.1  Escherichia coli strain UPMP1750 contig\n",
      "        0.7    2.5   0.5        1.5    1.4   0.5    1.4  1  NZ_JADBJU010000141.1  Escherichia coli strain UPMP1750 contig\n",
      "       0.76    2.4   0.1        1.3    1.6   0.1    1.7  1  NZ_JADBJU010000321.1  Escherichia coli strain UPMP1750 contig\n",
      "        1.1    1.9   0.0        3.4    0.2   0.0    1.5  2  NZ_JADBJU010000232.1  Escherichia coli strain UPMP1750 contig\n",
      "        7.2   -0.8   0.0         20   -2.3   0.0    1.3  1  NZ_JADBJU010000039.1  Escherichia coli strain UPMP1750 contig\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_JADBJU010000133.1  Escherichia coli strain UPMP1750 contig00133, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    2.6   0.0      0.01      0.68      82     183 ..     123     245 ..      42     248 .. 0.76\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 2.6 bits;  conditional E-value: 0.01\n",
      "           tp_nucl_aln  82 acgcttctgattgaaAAgGGgctgatcgatcccgcagcgctggatgagatcatcgacacctatgagaacaagatcGgcccgcg...... 164\n",
      "                           acgct+   +t++ +AA+GG++   t g t  c c g  +tgg tg++ tcat g cac+  tg +  c+a    G  c g g      \n",
      "  NZ_JADBJU010000133.1 123 ACGCTGTCTGTCCTGAAAGGTTATGT-GTTTGCCCTGATGTGGCTGCAGTCATGGCCACTCTTGTACGCCATTCTGAACAGTGccatga 210\n",
      "                           46666666677788888888777765.455566677778899******************99999999998877777666655666668 PP\n",
      "\n",
      "           tp_nucl_aln 165 ................gAAcGGtGcccacgtgGtg 183\n",
      "                                           gAA+GGtGc c+ gtgGtg\n",
      "  NZ_JADBJU010000133.1 211 cattttatgcgaaacaGAATGGTGCGCCGGTGGTG 245\n",
      "                           8888888888888888****************985 PP\n",
      "\n",
      ">> NZ_JADBJU010000141.1  Escherichia coli strain UPMP1750 contig00141, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    1.4   0.5     0.023       1.5     516     557 ..     427     468 ..     401     470 .. 0.95\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 1.4 bits;  conditional E-value: 0.023\n",
      "           tp_nucl_aln 516 gcccgagcGccctgaagggAccGaaggtctGgacgaggaggc 557\n",
      "                           gccc agcG cc  a g  A  Ga+ g  tGg+cgagga gc\n",
      "  NZ_JADBJU010000141.1 427 GCCCCAGCGACCGAACGTTAGTGAGCGAGTGGGCGAGGAAGC 468\n",
      "                           58999*********************************9876 PP\n",
      "\n",
      ">> NZ_JADBJU010000321.1  Escherichia coli strain UPMP1750 contig00321, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    1.6   0.1      0.02       1.3     456     487 ..       5      36 ..       1     194 [. 0.57\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 1.6 bits;  conditional E-value: 0.02\n",
      "           tp_nucl_aln 456 GcccgatgacatggaggtgcgcGtctggGAtt 487\n",
      "                           Gc  g +g+  tgga g +cg Gtc ggGA +\n",
      "  NZ_JADBJU010000321.1   5 GCAGGCCGGACTGGAAGCCCGTGTCCGGGAAA 36 \n",
      "                           55555555555555555555555555555432 PP\n",
      "\n",
      ">> NZ_JADBJU010000232.1  Escherichia coli strain UPMP1750 contig00232, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    0.2   0.0     0.053       3.4     449     486 ..     231     268 ..     215     281 .. 0.90\n",
      "   2 ?   -2.3   0.0      0.31        20      98     133 ..     327     362 ..     321     377 .. 0.81\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 0.2 bits;  conditional E-value: 0.053\n",
      "           tp_nucl_aln 449 tcaccctGcccgatgacatggaggtgcgcGtctggGAt 486\n",
      "                           t ++cctGc  g +g+  tgga g +cg Gtc ggGA \n",
      "  NZ_JADBJU010000232.1 231 TGGACCTGCAGGCCGGACTGGAAGCCCGTGTCCGGGAA 268\n",
      "                           567899*****************************985 PP\n",
      "\n",
      "  == domain 2  score: -2.3 bits;  conditional E-value: 0.31\n",
      "           tp_nucl_aln  98 AgGGgctgatcgatcccgcagcgctggatgagatca 133\n",
      "                           A+GG++  a cg tc  gc++cgctg+atg ga + \n",
      "  NZ_JADBJU010000232.1 327 AAGGTTATAACGGTCAGGCCACGCTGAATGTGACTT 362\n",
      "                           5688899999999***************99876544 PP\n",
      "\n",
      ">> NZ_JADBJU010000039.1  Escherichia coli strain UPMP1750 contig00039, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.3   0.0       0.3        20     270     321 ..   32256   32302 ..   32248   32307 .. 0.86\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.3 bits;  conditional E-value: 0.3\n",
      "           tp_nucl_aln   270 tcaggggGAacAtAtggtgGcggtggagAAcacggccgaggtgcAcAAtatg 321  \n",
      "                             tcag g     AtAt gtgGcg tgga+AAc c + +ga  tg AcAA +t \n",
      "  NZ_JADBJU010000039.1 32256 TCAGCG-----ATATTGTGGCGCTGGAAAACGCCCTTGATATGTACAAACTC 32302\n",
      "                             555554.....899********************************998776 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        324  (5297104 residues searched)\n",
      "Passed MSV filter:                        26  (0.0802469); expected 6.5 (0.02)\n",
      "Passed bias filter:                        9  (0.0277778); expected 6.5 (0.02)\n",
      "Passed Vit filter:                         9  (0.0277778); expected 6.5 (0.02)\n",
      "Passed Fwd filter:                         5  (0.0154321); expected 6.5 (0.02)\n",
      "Initial search space (Z):                324  [actual number of targets]\n",
      "Domain search space  (domZ):               5  [number of targets reported over threshold]\n",
      "# CPU time: 0.34u 0.01s 00:00:00.35 Elapsed: 00:00:00.18\n",
      "# Mc/sec: 17401.71\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_000143845.1_ASM14384v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_000143845.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (2051896 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.06u 0.01s 00:00:00.06 Elapsed: 00:00:00.06\n",
      "# Mc/sec: 20411.59\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_013114355.1_ASM1311435v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_013114355.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.14    3.1   0.0       0.24    2.3   0.0    1.2  1  NZ_VTXQ01000027.1  Vibrio jasicida strain T3G01 scaffold_26, \n",
      "       0.97    0.3   0.0        2.1   -0.9   0.0    1.3  1  NZ_VTXQ01000043.1  Vibrio jasicida strain T3G01 scaffold_42, \n",
      "        1.3   -0.2   0.0        3.1   -1.4   0.0    1.3  1  NZ_VTXQ01000047.1  Vibrio jasicida strain T3G01 scaffold_46, \n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_VTXQ01000027.1  Vibrio jasicida strain T3G01 scaffold_26, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    2.3   0.0    0.0078      0.24      69     151 ..   63763   63836 ..   63721   63847 .. 0.81\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 2.3 bits;  conditional E-value: 0.0078\n",
      "        tp_nucl_aln    69 cagggcgctggAaacgcttctgattgaaAAgGGgctgatcgatcccgcagcgctggatgagatcatcgacacctatgagaaca 151  \n",
      "                           ++ggc ctg Aa+   t  tgat++aa A+GGgct atcga+    + g + t+gatga          a ctatg  aac+\n",
      "  NZ_VTXQ01000027.1 63763 GCAGGCTCTGCAAGAATTCTTGATGCAAGAAGGGCTAATCGACATTAATGGCATCGATGAC---------ATCTATGCTAACG 63836\n",
      "                          46789*************************************9877666666666666655.........4455555555555 PP\n",
      "\n",
      ">> NZ_VTXQ01000043.1  Vibrio jasicida strain T3G01 scaffold_42, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.9   0.0     0.069       2.1       7      42 ..   24281   24316 ..   24272   24354 .. 0.80\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.9 bits;  conditional E-value: 0.069\n",
      "        tp_nucl_aln     7 cAtGatcatgccgaccacgaccacccgcacgctctt 42   \n",
      "                          cAtGatca+  cg+ cacga+cac++ ca+ +tc+ \n",
      "  NZ_VTXQ01000043.1 24281 CATGATCACCACGGACACGATCACGAACATCATCAC 24316\n",
      "                          9**********************9999988777654 PP\n",
      "\n",
      ">> NZ_VTXQ01000047.1  Vibrio jasicida strain T3G01 scaffold_46, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.4   0.0       0.1       3.1     218     261 ..   15000   15043 ..   14988   15060 .. 0.85\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.4 bits;  conditional E-value: 0.1\n",
      "        tp_nucl_aln   218 ggctgctggagGAtgccactgcgGcgatcgccgagctgGggttt 261  \n",
      "                          ggc gctg   GAtg+cac  cgG++  +gccga  tgGggtt \n",
      "  NZ_VTXQ01000047.1 15000 GGCCGCTGTTCGATGGCACGACGGTTTCTGCCGAATTGGGGTTA 15043\n",
      "                          45567888889*****************************9975 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         93  (5862164 residues searched)\n",
      "Passed MSV filter:                         3  (0.0322581); expected 1.9 (0.02)\n",
      "Passed bias filter:                        3  (0.0322581); expected 1.9 (0.02)\n",
      "Passed Vit filter:                         3  (0.0322581); expected 1.9 (0.02)\n",
      "Passed Fwd filter:                         3  (0.0322581); expected 1.9 (0.02)\n",
      "Initial search space (Z):                 93  [actual number of targets]\n",
      "Domain search space  (domZ):               3  [number of targets reported over threshold]\n",
      "# CPU time: 0.73u 0.00s 00:00:00.73 Elapsed: 00:00:00.38\n",
      "# Mc/sec: 9463.53\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_010367435.1_ASM1036743v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_010367435.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         74  (2030655 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 1.5 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 1.5 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 1.5 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.5 (0.02)\n",
      "Initial search space (Z):                 74  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.09u 0.00s 00:00:00.09 Elapsed: 00:00:00.05\n",
      "# Mc/sec: 23037.18\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003574085.1_ASM357408v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003574085.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        224  (2874609 residues searched)\n",
      "Passed MSV filter:                        47  (0.209821); expected 4.5 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 4.5 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 4.5 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 4.5 (0.02)\n",
      "Initial search space (Z):                224  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.09u 0.00s 00:00:00.09 Elapsed: 00:00:00.05\n",
      "# Mc/sec: 32125.39\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003049515.1_ASM304951v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003049515.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "        1.1   -0.3   0.0          2   -1.1   0.0    1.2  1  NZ_PYWH01000002.1  Lysinibacillus parviboronicapiens strain V\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_PYWH01000002.1  Lysinibacillus parviboronicapiens strain VT1066 NODE_2_length_333203_cov_48.9101_ID_577, whole gen\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.1   0.0     0.028         2     424     468 ..  145642  145686 ..  145640  145691 .. 0.89\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.1 bits;  conditional E-value: 0.028\n",
      "        tp_nucl_aln    424 cgcagcGtgctggcgGAgtttGGcgtcaccctGcccgatgacatg 468   \n",
      "                           cg+ ++Gtgct g++GA+tttGGcg  ++ ct cc gatg  at \n",
      "  NZ_PYWH01000002.1 145642 CGACAGGTGCTTGATGAATTTGGCGCAGAACTACCAGATGCAATT 145686\n",
      "                           677789********************************9987765 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         71  (4746191 residues searched)\n",
      "Passed MSV filter:                         1  (0.0140845); expected 1.4 (0.02)\n",
      "Passed bias filter:                        1  (0.0140845); expected 1.4 (0.02)\n",
      "Passed Vit filter:                         1  (0.0140845); expected 1.4 (0.02)\n",
      "Passed Fwd filter:                         1  (0.0140845); expected 1.4 (0.02)\n",
      "Initial search space (Z):                 71  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 1.12u 0.01s 00:00:01.13 Elapsed: 00:00:01.07\n",
      "# Mc/sec: 2755.21\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_015594835.1_ASM1559483v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_015594835.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        356  (1487847 residues searched)\n",
      "Passed MSV filter:                         3  (0.00842697); expected 7.1 (0.02)\n",
      "Passed bias filter:                        3  (0.00842697); expected 7.1 (0.02)\n",
      "Passed Vit filter:                         3  (0.00842697); expected 7.1 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 7.1 (0.02)\n",
      "Initial search space (Z):                356  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.05u 0.00s 00:00:00.05 Elapsed: 00:00:00.03\n",
      "# Mc/sec: 25714.18\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003967135.1_ASM396713v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003967135.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          6  (2224823 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.1 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.1 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.1 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.1 (0.02)\n",
      "Initial search space (Z):                  6  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.05u 0.01s 00:00:00.06 Elapsed: 00:00:00.05\n",
      "# Mc/sec: 25842.38\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_016603185.1_ASM1660318v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_016603185.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         62  (2027656 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 1.2 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 1.2 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 1.2 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.2 (0.02)\n",
      "Initial search space (Z):                 62  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.07u 0.00s 00:00:00.07 Elapsed: 00:00:00.04\n",
      "# Mc/sec: 27530.82\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_017255165.1_ASM1725516v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_017255165.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         23  (3882313 residues searched)\n",
      "Passed MSV filter:                         2  (0.0869565); expected 0.5 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.5 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.5 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.5 (0.02)\n",
      "Initial search space (Z):                 23  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.13u 0.02s 00:00:00.15 Elapsed: 00:00:00.08\n",
      "# Mc/sec: 29778.82\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006227585.2_ASM622758v2_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_006227585.2_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "        0.9    1.3   0.1        1.2    0.8   0.1    1.1  1  NZ_VDEH02000159.1  Salmonella enterica subsp. enterica serova\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_VDEH02000159.1  Salmonella enterica subsp. enterica serovar London strain HIY0010 NODE_159_length_237_cov_74.80000\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    0.8   0.1    0.0072       1.2     340     377 ..     144     186 ..     138     195 .. 0.75\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 0.8 bits;  conditional E-value: 0.0072\n",
      "        tp_nucl_aln 340 tcctGctAcc......cgtGgcctctgctgGGcctgccgccggc 377\n",
      "                        tcctG+tAcc      c t gc t+tg t GGcctgccgcc gc\n",
      "  NZ_VDEH02000159.1 144 TCCTGTTACCgttcgaCTT-GCATGTGTTAGGCCTGCCGCCAGC 186\n",
      "                        5788888888333222222.45678999*************986 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        170  (4680004 residues searched)\n",
      "Passed MSV filter:                        20  (0.117647); expected 3.4 (0.02)\n",
      "Passed bias filter:                        2  (0.0117647); expected 3.4 (0.02)\n",
      "Passed Vit filter:                         2  (0.0117647); expected 3.4 (0.02)\n",
      "Passed Fwd filter:                         1  (0.00588235); expected 3.4 (0.02)\n",
      "Initial search space (Z):                170  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.14u 0.01s 00:00:00.15 Elapsed: 00:00:00.08\n",
      "# Mc/sec: 33218.51\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003698745.1_ASM369874v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003698745.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         69  (7002034 residues searched)\n",
      "Passed MSV filter:                        23  (0.333333); expected 1.4 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 1.4 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 1.4 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.4 (0.02)\n",
      "Initial search space (Z):                 69  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.22u 0.03s 00:00:00.25 Elapsed: 00:00:00.13\n",
      "# Mc/sec: 32355.68\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004920675.1_ASM492067v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_004920675.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "        2.3   -1.4   0.0        4.3   -2.2   0.0    1.1  1  NZ_QFGY01000030.1  Legionella pneumophila strain RD04 NODE_30\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_QFGY01000030.1  Legionella pneumophila strain RD04 NODE_30_length_20864_cov_30.4063_ID_3570, whole genome shotgun \n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.2   0.0      0.06       4.3     116     164 ..   12657   12705 ..   12642   12711 .. 0.89\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.2 bits;  conditional E-value: 0.06\n",
      "        tp_nucl_aln   116 cagcgctggatgagatcatcgacacctatgagaacaagatcGgcccgcg 164  \n",
      "                          +agc+ctggat agat  t gaca + a g g ac+a a  Gg  cgc+\n",
      "  NZ_QFGY01000030.1 12657 AAGCCCTGGATAAGATGTTAGACAATAAAGGGTACCAAACAGGATCGCA 12705\n",
      "                          57999******************************99999999998875 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         71  (3634142 residues searched)\n",
      "Passed MSV filter:                         2  (0.028169); expected 1.4 (0.02)\n",
      "Passed bias filter:                        2  (0.028169); expected 1.4 (0.02)\n",
      "Passed Vit filter:                         2  (0.028169); expected 1.4 (0.02)\n",
      "Passed Fwd filter:                         1  (0.0140845); expected 1.4 (0.02)\n",
      "Initial search space (Z):                 71  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.15u 0.01s 00:00:00.16 Elapsed: 00:00:00.10\n",
      "# Mc/sec: 21533.76\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004345025.1_ASM434502v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_004345025.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (3308353 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.08u 0.00s 00:00:00.08 Elapsed: 00:00:00.08\n",
      "# Mc/sec: 24793.09\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004134845.1_ASM413484v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_004134845.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.11    3.6   0.0        0.2    2.8   0.0    1.5  1  NZ_SDRE01000107.1  Staphylococcus sp. SNAZ 75 s75_Ayuni_conti\n",
      "        2.7   -1.0   0.0        4.7   -1.8   0.0    1.1  1  NZ_SDRE01000002.1  Staphylococcus sp. SNAZ 75 s75_Ayuni_conti\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_SDRE01000107.1  Staphylococcus sp. SNAZ 75 s75_Ayuni_contig_107, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    2.8   0.0    0.0037       0.2     546     591 ..     187     232 ..     164     284 .. 0.86\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 2.8 bits;  conditional E-value: 0.0037\n",
      "        tp_nucl_aln 546 GgacgaggaggcgctggccgcgctgGtcAcccgcgAcagcAtGatc 591\n",
      "                        G+a g  gagg +ctggc+gcg tg tcA     gA   cAtGatc\n",
      "  NZ_SDRE01000107.1 187 GAAGGTTGAGGTACTGGCGGCGATGATCAGATCGGAACTCATGATC 232\n",
      "                        6777888999999999999999999999987777777778888875 PP\n",
      "\n",
      ">> NZ_SDRE01000002.1  Staphylococcus sp. SNAZ 75 s75_Ayuni_contig_2, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.8   0.0     0.086       4.7      83     133 ..   53600   53650 ..   53580   53667 .. 0.89\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.8 bits;  conditional E-value: 0.086\n",
      "        tp_nucl_aln    83 cgcttctgattgaaAAgGGgctgatcgatcccgcagcgctggatgagatca 133  \n",
      "                          c  tt  gattgaaAA+G +    tcga+cccgca+cg tgga ga+ t a\n",
      "  NZ_SDRE01000002.1 53600 CAATTACGATTGAAAAAGCTAACTTCGACCCCGCAACGATGGAAGAAGTAA 53650\n",
      "                          567778899***********************************9987755 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        108  (2713405 residues searched)\n",
      "Passed MSV filter:                         2  (0.0185185); expected 2.2 (0.02)\n",
      "Passed bias filter:                        2  (0.0185185); expected 2.2 (0.02)\n",
      "Passed Vit filter:                         2  (0.0185185); expected 2.2 (0.02)\n",
      "Passed Fwd filter:                         2  (0.0185185); expected 2.2 (0.02)\n",
      "Initial search space (Z):                108  [actual number of targets]\n",
      "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
      "# CPU time: 0.27u 0.01s 00:00:00.28 Elapsed: 00:00:00.23\n",
      "# Mc/sec: 7244.12\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002882455.1_FW215-R2_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_002882455.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.04    5.4   0.3       0.14    3.6   0.0    1.9  2  NZ_POFO01000081.1  Pseudomonas sp. FW215-R2 NODE_81_length_27\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_POFO01000081.1  Pseudomonas sp. FW215-R2 NODE_81_length_276_cov_0.718593, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.5   0.0     0.018       2.4     119     138 ..      18      37 ..       7     131 .. 0.56\n",
      "   2 ?    3.6   0.0     0.001      0.14     417     467 ..     223     273 ..     208     276 .] 0.89\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.5 bits;  conditional E-value: 0.018\n",
      "        tp_nucl_aln 119 cgctggatgagatcatcgac 138\n",
      "                        cgctg+a g+g tc tcg c\n",
      "  NZ_POFO01000081.1  18 CGCTGAAGGCGTTCGTCGCC 37 \n",
      "                        44444444444444444432 PP\n",
      "\n",
      "  == domain 2  score: 3.6 bits;  conditional E-value: 0.001\n",
      "        tp_nucl_aln 417 cGAaccgcgcagcGtgctggcgGAgtttGGcgtcaccctGcccgatgacat 467\n",
      "                        cGA c g gc gcGtg t g++GA    GGcgtca++ctG   gatg ca \n",
      "  NZ_POFO01000081.1 223 CGAGCTGGGCCGCGTGATCGATGATCCGGGCGTCAAGCTGGAGGATGTCAG 273\n",
      "                        6888999999999***********9999************99999987765 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        138  (6529841 residues searched)\n",
      "Passed MSV filter:                        28  (0.202899); expected 2.8 (0.02)\n",
      "Passed bias filter:                        2  (0.0144928); expected 2.8 (0.02)\n",
      "Passed Vit filter:                         2  (0.0144928); expected 2.8 (0.02)\n",
      "Passed Fwd filter:                         1  (0.00724638); expected 2.8 (0.02)\n",
      "Initial search space (Z):                138  [actual number of targets]\n",
      "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
      "# CPU time: 0.23u 0.01s 00:00:00.24 Elapsed: 00:00:00.11\n",
      "# Mc/sec: 35866.58\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003295925.1_ASM329592v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003295925.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        468  (4554424 residues searched)\n",
      "Passed MSV filter:                        90  (0.192308); expected 9.4 (0.02)\n",
      "Passed bias filter:                        1  (0.00213675); expected 9.4 (0.02)\n",
      "Passed Vit filter:                         1  (0.00213675); expected 9.4 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 9.4 (0.02)\n",
      "Initial search space (Z):                468  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.16u 0.01s 00:00:00.17 Elapsed: 00:00:00.09\n",
      "# Mc/sec: 31534.73\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_013735435.1_ASM1373543v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_013735435.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          6  (6134804 residues searched)\n",
      "Passed MSV filter:                         1  (0.166667); expected 0.1 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.1 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.1 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.1 (0.02)\n",
      "Initial search space (Z):                  6  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.12u 0.00s 00:00:00.12 Elapsed: 00:00:00.12\n",
      "# Mc/sec: 30522.62\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001882845.1_ASM188284v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001882845.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         44  (7262156 residues searched)\n",
      "Passed MSV filter:                        33  (0.75); expected 0.9 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.9 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.9 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.9 (0.02)\n",
      "Initial search space (Z):                 44  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.39u 0.02s 00:00:00.41 Elapsed: 00:00:00.22\n",
      "# Mc/sec: 20539.47\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_008642395.1_ASM864239v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_008642395.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          6  (5406033 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.1 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.1 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.1 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.1 (0.02)\n",
      "Initial search space (Z):                  6  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.12u 0.00s 00:00:00.12 Elapsed: 00:00:00.11\n",
      "# Mc/sec: 28831.18\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006371235.1_ASM637123v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_006371235.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "        5.8    0.6   0.0        9.4   -0.1   0.0    1.1  1  NZ_NNFX01000260.1  Vibrio parahaemolyticus strain F2_5 F2_5_c\n",
      "        8.3    0.1   0.0         13   -0.5   0.0    1.1  1  NZ_NNFX01000525.1  Vibrio parahaemolyticus strain F2_5 F2_5_c\n",
      "        8.5    0.1   0.0         14   -0.7   0.0    1.1  1  NZ_NNFX01000219.1  Vibrio parahaemolyticus strain F2_5 F2_5_c\n",
      "        9.5   -0.1   0.0         24   -1.4   0.0    1.3  1  NZ_NNFX01000155.1  Vibrio parahaemolyticus strain F2_5 F2_5_c\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_NNFX01000260.1  Vibrio parahaemolyticus strain F2_5 F2_5_contig260, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.1   0.0     0.054       9.4      84     122 ..    3861    3899 ..    3855    3923 .. 0.86\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.1 bits;  conditional E-value: 0.054\n",
      "        tp_nucl_aln   84 gcttctgattgaaAAgGGgctgatcgatcccgcagcgct 122 \n",
      "                         g t+ct +t gaaAA+GGg+tg tcgatcc g+a c +t\n",
      "  NZ_NNFX01000260.1 3861 GTTGCTTGTAGAAAAAGGGTTGCTCGATCCGGAATCAGT 3899\n",
      "                         568999************************998877555 PP\n",
      "\n",
      ">> NZ_NNFX01000525.1  Vibrio parahaemolyticus strain F2_5 F2_5_contig525, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.5   0.0     0.072        13      22      87 ..     169     234 ..     156     245 .. 0.88\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.5 bits;  conditional E-value: 0.072\n",
      "        tp_nucl_aln  22 cacgaccacccgcacgctcttttgccgccggaaactgctttgcgggtcagggcgctggAaacgctt 87 \n",
      "                         acgac ac+c  +c++tc+ttt c +   ga++ t c tt   gg  ag gc  tggAa+c ctt\n",
      "  NZ_NNFX01000525.1 169 TACGACTACGCCATCAATCATTTACTCAGTGAGCATCCGTTCAAGGAAAGCGCAGTGGAAGCCCTT 234\n",
      "                        689999999999999999***999999999******************************999886 PP\n",
      "\n",
      ">> NZ_NNFX01000219.1  Vibrio parahaemolyticus strain F2_5 F2_5_contig219, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.7   0.0     0.082        14     133     173 ..      53      93 ..      12      99 .. 0.73\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.7 bits;  conditional E-value: 0.082\n",
      "        tp_nucl_aln 133 atcgacacctatgagaacaagatcGgcccgcggAAcGGtGc 173\n",
      "                         t g   c t  ga a caa a cGgcccg  +AAcGGt c\n",
      "  NZ_NNFX01000219.1  53 GTGGTGTCGTTAGATACCAATAACGGCCCGGCCAACGGTAC 93 \n",
      "                        4667778889999************************9855 PP\n",
      "\n",
      ">> NZ_NNFX01000155.1  Vibrio parahaemolyticus strain F2_5 F2_5_contig155, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.4   0.0      0.14        24     524     566 ..   19733   19774 ..   19725   19784 .. 0.86\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.4 bits;  conditional E-value: 0.14\n",
      "        tp_nucl_aln   524 GccctgaagggAccGaaggtctGgacgaggaggcgctggccgc 566  \n",
      "                          G cc gaagggAc Gaa   +tGg+cg+gg g ++ctggc  c\n",
      "  NZ_NNFX01000155.1 19733 G-CCAGAAGGGACGGAATTGATGGGCGCGGTGAATCTGGCTCC 19774\n",
      "                          4.6899********************************99755 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        698  (5050995 residues searched)\n",
      "Passed MSV filter:                        15  (0.02149); expected 14.0 (0.02)\n",
      "Passed bias filter:                       15  (0.02149); expected 14.0 (0.02)\n",
      "Passed Vit filter:                        15  (0.02149); expected 14.0 (0.02)\n",
      "Passed Fwd filter:                         9  (0.012894); expected 14.0 (0.02)\n",
      "Initial search space (Z):                698  [actual number of targets]\n",
      "Domain search space  (domZ):               4  [number of targets reported over threshold]\n",
      "# CPU time: 0.81u 0.01s 00:00:00.82 Elapsed: 00:00:00.44\n",
      "# Mc/sec: 7158.22\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_005502615.1_ASM550261v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_005502615.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        103  (5204864 residues searched)\n",
      "Passed MSV filter:                        45  (0.436893); expected 2.1 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 2.1 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 2.1 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 2.1 (0.02)\n",
      "Initial search space (Z):                103  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.19u 0.01s 00:00:00.20 Elapsed: 00:00:00.11\n",
      "# Mc/sec: 28477.39\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003265965.1_ASM326596v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003265965.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        324  (4546384 residues searched)\n",
      "Passed MSV filter:                        87  (0.268519); expected 6.5 (0.02)\n",
      "Passed bias filter:                        1  (0.00308642); expected 6.5 (0.02)\n",
      "Passed Vit filter:                         1  (0.00308642); expected 6.5 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 6.5 (0.02)\n",
      "Initial search space (Z):                324  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.19u 0.02s 00:00:00.21 Elapsed: 00:00:00.10\n",
      "# Mc/sec: 27701.62\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_018458215.1_ASM1845821v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_018458215.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence             Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------             -----------\n",
      "  ------ inclusion threshold ------\n",
      "        3.3   -1.1   0.0        8.4   -2.4   0.0    1.2  1  NZ_JAGDKA010000082.1  Vibrio alginolyticus strain VA98 contig\n",
      "        3.5   -1.2   0.0         11   -2.8   0.0    1.3  1  NZ_JAGDKA010000009.1  Vibrio alginolyticus strain VA98 contig\n",
      "        3.6   -1.2   0.0        8.4   -2.5   0.0    1.2  1  NZ_JAGDKA010000032.1  Vibrio alginolyticus strain VA98 contig\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_JAGDKA010000082.1  Vibrio alginolyticus strain VA98 contig00082, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.4   0.0      0.21       8.4     290     321 ..     781     812 ..     776     815 .. 0.91\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.4 bits;  conditional E-value: 0.21\n",
      "           tp_nucl_aln 290 cggtggagAAcacggccgaggtgcAcAAtatg 321\n",
      "                           +ggtg a+AA+ cggc  a gtgc cAAtat \n",
      "  NZ_JAGDKA010000082.1 781 TGGTGAAAAATGCGGCAAAAGTGCTCAATATC 812\n",
      "                           5789999*********************9986 PP\n",
      "\n",
      ">> NZ_JAGDKA010000009.1  Vibrio alginolyticus strain VA98 contig00009, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.8   0.0      0.26        11     436     462 ..   68267   68293 ..   68263   68310 .. 0.88\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.8 bits;  conditional E-value: 0.26\n",
      "           tp_nucl_aln   436 gcgGAgtttGGcgtcaccctGcccgat 462  \n",
      "                             g++GAgtttGGcgtca++ctGcc ga+\n",
      "  NZ_JAGDKA010000009.1 68267 GATGAGTTTGGCGTCATTCTGCCAGAA 68293\n",
      "                             5789*******************9985 PP\n",
      "\n",
      ">> NZ_JAGDKA010000032.1  Vibrio alginolyticus strain VA98 contig00032, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.5   0.0      0.21       8.4     303     335 ..    3944    3976 ..    3933    3989 .. 0.83\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.5 bits;  conditional E-value: 0.21\n",
      "           tp_nucl_aln  303 ggccgaggtgcAcAAtatggtcgtctGcAccct 335 \n",
      "                             g+ ga  t cAcAA+ t +tcgtctG Accc \n",
      "  NZ_JAGDKA010000032.1 3944 AGAAGAACTTCACAACTTCATCGTCTGGACCCG 3976\n",
      "                            4677788899*******************9874 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        121  (5147913 residues searched)\n",
      "Passed MSV filter:                         7  (0.0578512); expected 2.4 (0.02)\n",
      "Passed bias filter:                        7  (0.0578512); expected 2.4 (0.02)\n",
      "Passed Vit filter:                         7  (0.0578512); expected 2.4 (0.02)\n",
      "Passed Fwd filter:                         3  (0.0247934); expected 2.4 (0.02)\n",
      "Initial search space (Z):                121  [actual number of targets]\n",
      "Domain search space  (domZ):               3  [number of targets reported over threshold]\n",
      "# CPU time: 1.43u 0.02s 00:00:01.45 Elapsed: 00:00:00.80\n",
      "# Mc/sec: 3983.87\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_014534785.1_ASM1453478v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_014534785.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          7  (3272810 residues searched)\n",
      "Passed MSV filter:                         2  (0.285714); expected 0.1 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.1 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.1 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.1 (0.02)\n",
      "Initial search space (Z):                  7  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.16u 0.01s 00:00:00.17 Elapsed: 00:00:00.09\n",
      "# Mc/sec: 22185.14\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_009176545.1_ASM917654v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_009176545.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          5  (4690830 residues searched)\n",
      "Passed MSV filter:                         2  (0.4); expected 0.1 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.1 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.1 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.1 (0.02)\n",
      "Initial search space (Z):                  5  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.24u 0.03s 00:00:00.27 Elapsed: 00:00:00.15\n",
      "# Mc/sec: 18756.37\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_006370635.1_ASM637063v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_006370635.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.66    2.1   0.0        1.2    1.1   0.0    1.3  1  NZ_NNFL01000054.1  Vibrio parahaemolyticus strain F3_7 F3_7_c\n",
      "        2.4    0.2   0.0        4.5   -0.7   0.0    1.2  1  NZ_NNFL01000048.1  Vibrio parahaemolyticus strain F3_7 F3_7_c\n",
      "        2.7    0.0   0.0        6.1   -1.1   0.0    1.3  1  NZ_NNFL01000041.1  Vibrio parahaemolyticus strain F3_7 F3_7_c\n",
      "        6.9   -1.3   0.0         16   -2.5   0.0    1.2  1  NZ_NNFL01000045.1  Vibrio parahaemolyticus strain F3_7 F3_7_c\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_NNFL01000054.1  Vibrio parahaemolyticus strain F3_7 F3_7_contig054, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    1.1   0.0     0.023       1.2      70     139 ..   24931   25000 ..   24909   25014 .. 0.85\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 1.1 bits;  conditional E-value: 0.023\n",
      "        tp_nucl_aln    70 agggcgctggAaacgcttctgattgaaAAgGGgctgatcgatcccgcagcgctggatgagatcatcgaca 139  \n",
      "                          +++gc +tg Aa+ g tt tg t+ aa A+GG ctgat+gat  c ++g + t+ga+ga atc  cg ca\n",
      "  NZ_NNFL01000054.1 24931 CATGCTTTGCAAGAGTTTTTGCTCAAAGAAGGCCTGATTGATATCAACGGCATCGACGACATCTACGCCA 25000\n",
      "                          678999*************************************999999888999988887665444443 PP\n",
      "\n",
      ">> NZ_NNFL01000048.1  Vibrio parahaemolyticus strain F3_7 F3_7_contig048, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -0.7   0.0     0.082       4.5       6      42 ..    3500    3542 ..    3495    3570 .. 0.65\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -0.7 bits;  conditional E-value: 0.082\n",
      "        tp_nucl_aln    6 ccAtGatca......tgccgaccacgaccacccgcacgctctt 42  \n",
      "                          cAtGatca      t  +ga+cacgaccaccc ca+g+tc+t\n",
      "  NZ_NNFL01000048.1 3500 GCATGATCAccatcaTCATGATCACGACCACCCACATGATCAT 3542\n",
      "                         5667766550000005566777888888888888888777665 PP\n",
      "\n",
      ">> NZ_NNFL01000041.1  Vibrio parahaemolyticus strain F3_7 F3_7_contig041, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.1   0.0      0.11       6.1     423     462 ..   90880   90919 ..   90863   90950 .. 0.83\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.1 bits;  conditional E-value: 0.11\n",
      "        tp_nucl_aln   423 gcgcagcGtgctggcgGAgtttGGcgtcaccctGcccgat 462  \n",
      "                          gc c+gcGt ct gc+GA+tt+GGcgt +++ tG  +g +\n",
      "  NZ_NNFL01000041.1 90880 GCACGGCGTTCTTGCTGAATTCGGCGTAGAGATGATTGGC 90919\n",
      "                          566899**********************998877555554 PP\n",
      "\n",
      ">> NZ_NNFL01000045.1  Vibrio parahaemolyticus strain F3_7 F3_7_contig045, whole genome shotgun sequence\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -2.5   0.0      0.29        16     109     153 ..   20645   20689 ..   20632   20693 .. 0.83\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -2.5 bits;  conditional E-value: 0.29\n",
      "        tp_nucl_aln   109 gatcccgcagcgctggatgagatcatcgacacctatgagaacaag 153  \n",
      "                          gat ccgc+gc ct+gatg+g t atc a a ct +  +  caa \n",
      "  NZ_NNFL01000045.1 20645 GATACCGCCGCACTCGATGCGTTGATCAATAGCTTCTCAGGCAAT 20689\n",
      "                          6899***********************999988887766666665 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        219  (5064766 residues searched)\n",
      "Passed MSV filter:                         8  (0.0365297); expected 4.4 (0.02)\n",
      "Passed bias filter:                        8  (0.0365297); expected 4.4 (0.02)\n",
      "Passed Vit filter:                         8  (0.0365297); expected 4.4 (0.02)\n",
      "Passed Fwd filter:                         4  (0.0182648); expected 4.4 (0.02)\n",
      "Initial search space (Z):                219  [actual number of targets]\n",
      "Domain search space  (domZ):               4  [number of targets reported over threshold]\n",
      "# CPU time: 1.36u 0.01s 00:00:01.37 Elapsed: 00:00:00.80\n",
      "# Mc/sec: 3946.97\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003839525.1_ASM383952v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003839525.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         51  (6432866 residues searched)\n",
      "Passed MSV filter:                        20  (0.392157); expected 1.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 1.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 1.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 1.0 (0.02)\n",
      "Initial search space (Z):                 51  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.22u 0.03s 00:00:00.25 Elapsed: 00:00:00.12\n",
      "# Mc/sec: 32301.72\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003427055.1_ASM342705v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003427055.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          2  (5006569 residues searched)\n",
      "Passed MSV filter:                         1  (0.5); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  2  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.23u 0.03s 00:00:00.26 Elapsed: 00:00:00.25\n",
      "# Mc/sec: 12026.54\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_017491835.1_ASM1749183v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_017491835.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        134  (5908317 residues searched)\n",
      "Passed MSV filter:                        23  (0.171642); expected 2.7 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 2.7 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 2.7 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 2.7 (0.02)\n",
      "Initial search space (Z):                134  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.20u 0.01s 00:00:00.21 Elapsed: 00:00:00.10\n",
      "# Mc/sec: 34214.30\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001710555.1_ASM171055v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001710555.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         18  (2935319 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.4 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.4 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.4 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.4 (0.02)\n",
      "Initial search space (Z):                 18  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.07u 0.00s 00:00:00.07 Elapsed: 00:00:00.04\n",
      "# Mc/sec: 41468.03\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_002849795.1_ASM284979v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_002849795.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          2  (3913006 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  2  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.08u 0.01s 00:00:00.09 Elapsed: 00:00:00.05\n",
      "# Mc/sec: 44887.70\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003294835.1_ASM329483v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003294835.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                        435  (4592680 residues searched)\n",
      "Passed MSV filter:                       100  (0.229885); expected 8.7 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 8.7 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 8.7 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 8.7 (0.02)\n",
      "Initial search space (Z):                435  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.15u 0.00s 00:00:00.15 Elapsed: 00:00:00.07\n",
      "# Mc/sec: 36539.87\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003034845.1_ASM303484v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003034845.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         41  (3663288 residues searched)\n",
      "Passed MSV filter:                        18  (0.439024); expected 0.8 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.8 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.8 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.8 (0.02)\n",
      "Initial search space (Z):                 41  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.14u 0.01s 00:00:00.15 Elapsed: 00:00:00.08\n",
      "# Mc/sec: 27324.15\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_004214365.1_ASM421436v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_004214365.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------\n",
      "  ------ inclusion threshold ------\n",
      "       0.12    3.2   0.0       0.21    2.5   0.0    1.3  1  NZ_RHHK01000086.1  Vibrio vulnificus strain VA-WGS-18054 NODE\n",
      "        1.4   -0.2   0.0        2.9   -1.3   0.0    1.2  1  NZ_RHHK01000049.1  Vibrio vulnificus strain VA-WGS-18054 NODE\n",
      "        1.7   -0.5   0.1        4.1   -1.8   0.1    1.3  1  NZ_RHHK01000021.1  Vibrio vulnificus strain VA-WGS-18054 NODE\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      ">> NZ_RHHK01000086.1  Vibrio vulnificus strain VA-WGS-18054 NODE_86_length_728_cov_0.940100, whole genome shotgun sequen\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?    2.5   0.0    0.0068      0.21     222     267 ..     608     653 ..     581     664 .. 0.87\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: 2.5 bits;  conditional E-value: 0.0068\n",
      "        tp_nucl_aln 222 gctggagGAtgccactgcgGcgatcgccgagctgGggttttctgga 267\n",
      "                        gctgga GAtg c  tg+ Gcg t gccgag t Gg  tt+c+gg+\n",
      "  NZ_RHHK01000086.1 608 GCTGGACGATGTCCGTGAAGCGCTGGCCGAGGTAGGTATTACCGGC 653\n",
      "                        6899********************************9999999886 PP\n",
      "\n",
      ">> NZ_RHHK01000049.1  Vibrio vulnificus strain VA-WGS-18054 NODE_49_length_6701_cov_36.336325, whole genome shotgun sequ\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.3   0.0     0.093       2.9      49     102 ..    4437    4490 ..    4402    4521 .. 0.74\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.3 bits;  conditional E-value: 0.093\n",
      "        tp_nucl_aln   49 ccggaaactgctttgcgggtcagggcgctggAaacgcttctgattgaaAAgGGg 102 \n",
      "                         cc    a+tgct+tg   g   ++gc ctggAa+  cttc  a+ g +AA+G  \n",
      "  NZ_RHHK01000049.1 4437 CCCAGCATTGCTGTGGATGCTGATGCTCTGGAAGTTCTTCATACAGCGAAAGAC 4490\n",
      "                         3336677889999999999999*********************99999999843 PP\n",
      "\n",
      ">> NZ_RHHK01000021.1  Vibrio vulnificus strain VA-WGS-18054 NODE_21_length_83447_cov_39.151488, whole genome shotgun seq\n",
      "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
      " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
      "   1 ?   -1.8   0.1      0.13       4.1     536     605 ..   61650   61716 ..   61648   61733 .. 0.79\n",
      "\n",
      "  Alignments for each domain:\n",
      "  == domain 1  score: -1.8 bits;  conditional E-value: 0.13\n",
      "        tp_nucl_aln   536 ccGaaggtctGgacgaggaggcgctggccgcgctgGtcAcccgcgAcagcAtGatcGGtacgGggctggc 605  \n",
      "                          c Gaag t tGga g  ga gcgctggcc c ctgGt        Aca cAtG  cGG+a  G g   g+\n",
      "  NZ_RHHK01000021.1 61650 CAGAAGATGTGGAAGGTGAAGCGCTGGCCACACTGGTTGT---GAACAACATGCGCGGCATCGTGAAAGT 61716\n",
      "                          5799999*****************************9632...246677777777777776666655555 PP\n",
      "\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         93  (4778453 residues searched)\n",
      "Passed MSV filter:                         4  (0.0430108); expected 1.9 (0.02)\n",
      "Passed bias filter:                        4  (0.0430108); expected 1.9 (0.02)\n",
      "Passed Vit filter:                         4  (0.0430108); expected 1.9 (0.02)\n",
      "Passed Fwd filter:                         3  (0.0322581); expected 1.9 (0.02)\n",
      "Initial search space (Z):                 93  [actual number of targets]\n",
      "Domain search space  (domZ):               3  [number of targets reported over threshold]\n",
      "# CPU time: 0.48u 0.02s 00:00:00.50 Elapsed: 00:00:00.31\n",
      "# Mc/sec: 9419.98\n",
      "//\n",
      "[ok]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_003353065.1_ASM335306v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_003353065.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                          1  (4294247 residues searched)\n",
      "Passed MSV filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.0 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.0 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.0 (0.02)\n",
      "Initial search space (Z):                  1  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.11u 0.00s 00:00:00.11 Elapsed: 00:00:00.11\n",
      "# Mc/sec: 22352.32\n",
      "//\n",
      "[ok]\n",
      "# hmmsearch :: search profile(s) against a sequence database\n",
      "# HMMER 3.3 (Nov 2019); http://hmmer.org/\n",
      "# Copyright (C) 2019 Howard Hughes Medical Institute.\n",
      "# Freely distributed under the BSD open source license.\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "# query HMM file:                  /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/databases/nitrile_hydratase_alpha_tp_nucl.hmm\n",
      "# target sequence database:        /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/temp_neg_genomes/GCF_001831805.1_ASM183180v1_genomic.fna\n",
      "# per-seq hits tabular output:     /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_hmm/results/GCF_001831805.1_hmm_result.tbl\n",
      "# Vit filter P threshold:       <= 0.02\n",
      "# Fwd filter P threshold:       <= 0.02\n",
      "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
      "\n",
      "Query:       tp_nucl_aln  [M=624]\n",
      "Scores for complete sequences (score includes all domains):\n",
      "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
      "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
      "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
      "\n",
      "   [No hits detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Domain annotation for each sequence (and alignments):\n",
      "\n",
      "   [No targets detected that satisfy reporting thresholds]\n",
      "\n",
      "\n",
      "Internal pipeline statistics summary:\n",
      "-------------------------------------\n",
      "Query model(s):                            1  (624 nodes)\n",
      "Target sequences:                         28  (4933226 residues searched)\n",
      "Passed MSV filter:                         4  (0.142857); expected 0.6 (0.02)\n",
      "Passed bias filter:                        0  (0); expected 0.6 (0.02)\n",
      "Passed Vit filter:                         0  (0); expected 0.6 (0.02)\n",
      "Passed Fwd filter:                         0  (0); expected 0.6 (0.02)\n",
      "Initial search space (Z):                 28  [actual number of targets]\n",
      "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
      "# CPU time: 0.17u 0.02s 00:00:00.19 Elapsed: 00:00:00.10\n",
      "# Mc/sec: 28013.02\n",
      "//\n",
      "[ok]\n",
      "moving negative GCF_000520615.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_000355675.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_013277795.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_013249065.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_001529305.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003050585.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_001044395.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_018199995.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_001854765.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003171655.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_015864865.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_009846755.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_005938125.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_017168275.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003086815.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_016837255.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003015025.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_011806565.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_004803475.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_000371805.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_001730205.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_014858715.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_002021755.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_000737535.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_006246515.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_018531475.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_017948225.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_001015055.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_004360145.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_009362315.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_900185035.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_009857605.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_005281475.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_006381105.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_001999205.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "moving negative GCF_000294775.2 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_001951155.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_009789075.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_002573675.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_013363245.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_002893095.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_014164675.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_001888265.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_001296255.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_001655635.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_001720485.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_007859875.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_018122435.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003198945.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_004022565.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_002127625.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_016812975.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_000968265.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_001611905.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_014764705.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_001593605.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_008386235.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_011393335.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_005280335.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_013726215.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_001494035.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003063435.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003925855.2 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_008632455.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_014856455.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_002882435.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_002507915.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_002929435.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003340555.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_000769825.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_001833665.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003296495.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_001573015.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_000737775.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_900166155.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_001870285.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_002156885.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_013032445.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003295805.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_012275045.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_001832095.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_000186125.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003344765.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_001707555.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_017570045.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_002106175.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_018449525.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_000400485.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003351365.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_002216445.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_900167585.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_006517585.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_010983895.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_004745955.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_002883245.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_002355535.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_002103335.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_004921205.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_002241775.2 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_012224465.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_013790265.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_017696275.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003854195.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_015276975.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_002119585.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_017349495.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_015680345.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003096015.2 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "moving negative GCF_003449015.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_001507335.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003710255.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_900379955.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_000143845.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_010367435.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003574085.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_015594835.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003967135.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_016603185.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_017255165.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003698745.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_004345025.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003295925.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_013735435.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_001882845.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_008642395.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_006371235.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_005502615.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003265965.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_018458215.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_014534785.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_009176545.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003839525.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003427055.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_017491835.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_001710555.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_002849795.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003294835.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003034845.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_003353065.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "moving negative GCF_001831805.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_alpha/base_genomes/neg_genomes\n",
      "Done\n"
     ]
    }
   ],
   "source": [
    "# Use hmmer to find tp_gene contamination in negative samples\n",
    "# This is to some extent dependent on which tp genes are chosen. If fewer neg_genomes result from the decontamination\n",
    "#     process than are specified by select_neg_genomes, it might be worth trying a different set of tp_genes\n",
    "\n",
    "with open(BGC_path+'/'+'report_2_generate_tp.txt', 'a') as f:\n",
    "    f.write('\\n'+'Using hmmer to check negative genomes for contamination'+'\\n\\n')\n",
    "\n",
    "# Build net alignment of tp_genes\n",
    "!muscle -in \"{BGC_path}\"/\"{BGC_type}\"_selected_tp_genes.fasta -out \"{neg_hmm_db_path}\"/\"{BGC_type}\"_selected_tp_gene_alignment.fasta -loga \"{BGC_path}\"/report_2_generate_tp.txt\n",
    "\n",
    "with open(BGC_path+'/'+'report_2_generate_tp.txt', 'a') as f:\n",
    "    f.write('\\n'+'Building nucleotide hmm of selected true positive genes'+'\\n\\n'+'Searching neg genomes with nucleotide hmm.\\n\\n')\n",
    "\n",
    "# Build nucleotide based hmm of tp_genes alignment\n",
    "!hmmbuild -n tp_nucl_aln --dna \"{neg_hmm_db_path}\"/\"{BGC_type}\"_tp_nucl.hmm \"{neg_hmm_db_path}\"/\"{BGC_type}\"_selected_tp_gene_alignment.fasta\n",
    "\n",
    "# Search neg genomes with hmm\n",
    "for genomes in neg_genomes_list:\n",
    "    !hmmsearch --F1 0.02 --F2 0.02 --F3 0.02 --tblout \"{neg_hmm_results_path}\"/\"{genomes}\"_hmm_result.tbl \"{neg_hmm_db_path}\"/\"{BGC_type}\"_tp_nucl.hmm \"{neg_genomes_path}\"/\"{genomes}\"*\n",
    "#    !nhmmer --F1 0.02 --F2 0.02 --F3 0.02 --tblout \"{neg_hmm_results_path}\"/\"{genomes}\"_hmm_result.tbl \"{neg_hmm_db_path}\"/\"{BGC_type}\"_tp_nucl.hmm \"{neg_genomes_path}\"/\"{genomes}\"*\n",
    "\n",
    "# Parse hmm output\n",
    "#https://stackoverflow.com/questions/62012615/convert-a-hmmer-tblout-output-to-a-pandas-dataframe\n",
    "\n",
    "sample_list = []\n",
    "id_list=[]\n",
    "evalue_list=[]\n",
    "\n",
    "for filename in os.listdir(neg_hmm_results_path):\n",
    "    samplename = '_'.join(filename.split('_')[0:2])\n",
    "    with open(neg_hmm_results_path+'/'+filename) as handle:\n",
    "        for queryresult in SearchIO.parse(handle, 'hmmer3-tab'):\n",
    "            for hit in queryresult.hits:\n",
    "                sample_list.append(samplename)\n",
    "                id_list.append(hit.id)\n",
    "                evalue_list.append(hit.evalue)\n",
    "\n",
    "# Generate hit_df based on found tp_hits in neg genomes\n",
    "hmm_dict = {'sample': sample_list, 'target_name': id_list, 'evalue': evalue_list}             \n",
    "hit_df = pd.DataFrame.from_dict(hmm_dict)\n",
    "hit_df.to_csv(neg_hmm_path+'/all_hmm_hits.csv', index=False)\n",
    "\n",
    "#filter hit_df by evalue cutoff\n",
    "evalue_filter = hit_df['evalue'] <= hmm_evalue_cutoff\n",
    "cutoff_hit_df = hit_df[evalue_filter]\n",
    "cutoff_hit_df.to_csv(neg_hmm_path+'/below_cutoff_hits.csv', index=False)\n",
    "\n",
    "#deduplicate samples\n",
    "unique_hit_df = cutoff_hit_df.drop_duplicates(subset='sample', inplace=False)\n",
    "unique_hit_df.to_csv(neg_hmm_path+'/removed_hmm_hits.csv', index=False)\n",
    "\n",
    "# Generate a list of contaminated negative genomes with combined results from blast and hmm hits\n",
    "hmm_remove_list = list(unique_hit_df.iloc[:,0])\n",
    "remove_list = blast_remove_list\n",
    "remove_list.extend(x for x in hmm_remove_list if x not in remove_list)\n",
    "\n",
    "with open(BGC_path+'/'+'report_2_generate_tp.txt', 'a') as f:\n",
    "    f.write('\\n'+'Contamination with tp_seqs found in samples:\\n'+str(hmm_remove_list)+'\\n\\n'+'Found '+str(len(remove_list))+' total samples with tp contamination in neg genomes.')\n",
    "\n",
    "# Remove these contaminated samples from the possible pool of negative sequences\n",
    "cleaned = ~unique_neg_df['assembly'].isin(remove_list)\n",
    "cleaned_unique_neg_df = unique_neg_df[cleaned]\n",
    "\n",
    "# Select a preset number of negative genomes (this could lead to fewer genomes available than selected. Handle)\n",
    "selected_neg_genomes = list(cleaned_unique_neg_df.iloc[:,1])[0:select_neg_genomes]\n",
    "\n",
    "# report block\n",
    "with open(BGC_path+'/'+'report_2_generate_tp.txt', 'a') as f:\n",
    "    f.write('\\n\\n decontaminated selected_neg_genomes are:\\n')\n",
    "    f.write(str(selected_neg_genomes))\n",
    "\n",
    "for genome in selected_neg_genomes:\n",
    "    print('moving negative', genome, 'to', output_neg_path)\n",
    "    !mv \"{neg_genomes_path}\"/\"{genome}\"* \"{output_neg_path}\"\n",
    "    \n",
    "print('Done')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "# checking out pos genomes with hmm build from tp genes for confirmation\n",
    "#for genomes in selected_neg_genomes:\n",
    "#    print(genomes)\n",
    "#    !nhmmer --tblout \"{neg_hmm_path}\"/pos_results/\"{genomes}\"_hmm_result.tbl \"{neg_hmm_db_path}\"/\"{BGC_type}\"_tp_nucl.hmm \"{output_neg_path}\"/\"{genomes}\"*"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The cell below generates a muscle alignment of the protein sequences from genomes that were not chosen as pos_genomes. However, in order to follow the procedure of the 2019 Sugimoto paper more closely, instead download the seed alignment for the respective protein of interest from pfam in fasta format and save is as {BGC_type}_selected_tp_alignment.fasta"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "#!module load MUSCLE/3.8.1551\n",
    "#not sure why full path is required?\n",
    "#with open(BGC_path+'/'+'report_2_generate_tp.txt', 'a') as f:\n",
    "#    f.write('\\n\\nMUSCLE alignment details:\\n')\n",
    "\n",
    "#!muscle -in \"{BGC_path}\"/\"{BGC_type}\"_selected_tp_aa.fasta -out \"{BGC_path}\"/\"{BGC_type}\"_selected_tp_alignment.fasta -loga \"{BGC_path}\"/report_2_generate_tp.txt\n",
    "#!module unload MUSCLE/3.8.1551"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Introduce a step here where all negative genomes are concatenated, a blastdb is built from the concatenated file, and all selected TP genes are used to query the blast db. If there are any hits, remove that sequence from the selected_neg_genomes_list, and add another one, then check again."
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
