{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This script generates the 2 remaining files required for input:\n",
    "- True positive genes (nucleotide fasta)\n",
    "- protein alignment (amino acid fasta)\n",
    "IMPORTANTLY, these 2 files are created from non-overlapping sequences!\n",
    "\n",
    "\n",
    "And also moves the corresponding files to new directories so that coverage tables can be generated from them.\n",
    "\n",
    "In the first cell specify:\n",
    "- BGC type (This name must stay constant throughout the scripts)\n",
    "- select_neg_genomes, i.e. the amount of negative genomes to be transferred to the neg_genomes directory\n",
    "- select_pos_genomes, i.e. the amount of positive genomes to be transferred to the pos_genomes directory and to generate the tp_genes file from (the surplus amount will be used to generate the protein alignment from)\n",
    "- pos_isolation_source_filter, if these terms are found in the isolation_source column of the positive samples in the summary file, they will be scored higher in a scoring column, i.e. samples from a known and desired isolation source will be used preferentially.\n",
    "- neg_isolation_source_filter, accordingly\n",
    "- avoid_list. These terms are scored with a 0, end at the bottom of the table, and will be picked last. This is useful when an uncommon gene is searched for and more, and/or more tenuous isolation sources have been allowed during download. These are generally words that contain one of the search terms, e.g. 'sea' in 'diseased'."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Modify in such a way that TP genes are used as query against all individual negative genomes. Negative genomes are only moved from temp directory to neg_genomes directory if the blast search comes back negative"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "BGC_type = 'nitrile_hydratase_beta'\n",
    "select_neg_genomes = 140\n",
    "select_pos_genomes = 10\n",
    "\n",
    "avoid_list = ['', 'isolation_source not annotated', 'diseased', 'mice', 'spice', 'septicemic', 'research', 'crevice']\n",
    "#these are identical to first script, but don't have to be\n",
    "pos_isolation_source_filter =  ['marine', 'sea', 'sponge', 'ocean', 'porifera', 'seafloor','sediment', 'water', 'tidal', 'coral', 'reef', 'coast', 'ship', 'fish', 'aquaculture', 'atlantic', 'pacific', 'mediterranean', 'baltic', 'pond', 'river', 'ice', 'carribean', 'lake', 'fjord', 'marina', 'hydro', 'algal', 'algae']\n",
    "neg_isolation_source_filter = ['marine', 'sea', 'sponge', 'ocean', 'porifera', 'seafloor', 'sediment', 'water', 'tidal', 'coral', 'reef', 'coast', 'ship', 'fish', 'aquaculture', 'atlantic', 'pacific', 'mediterranean', 'baltic', 'pond', 'river', 'ice', 'carribean', 'lake', 'fjord', 'marina', 'hydro', 'algal', 'algae']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "from os import listdir, mkdir\n",
    "from os.path import isfile, join\n",
    "from pathlib import Path\n",
    "import pandas as pd\n",
    "import random\n",
    "import glob\n",
    "import warnings"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Making /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "Making /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/pos_genomes\n"
     ]
    }
   ],
   "source": [
    "def makedir(dirpath):\n",
    "    if os.path.isdir(dirpath):\n",
    "        print(dirpath,'exists already')\n",
    "    else:\n",
    "        print('Making', dirpath)    \n",
    "        os.mkdir(dirpath)\n",
    "\n",
    "        \n",
    "# Defining paths for required directory structure for input and output files relative to parent directory\n",
    "#parent_dir='/media/manu/RiPP_Prioritiser/'\n",
    "#will make directories relative to the path the notebook was opened in\n",
    "parent_dir= !echo $(pwd)\n",
    "BGC_path=os.path.join(parent_dir[0], BGC_type)\n",
    "neg_genomes_path=os.path.join(BGC_path, 'base_genomes/temp_neg_genomes')\n",
    "pos_genomes_path=os.path.join(BGC_path, 'base_genomes/temp_pos_genomes')\n",
    "output_neg_path=os.path.join(BGC_path, 'base_genomes/neg_genomes')\n",
    "output_pos_path=os.path.join(BGC_path, 'base_genomes/pos_genomes')\n",
    "\n",
    "\n",
    "# Calling function to make directories if they don't exist yet\n",
    "makedir(output_neg_path)\n",
    "makedir(output_pos_path)\n",
    "\n",
    "os.chdir(BGC_path)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Generating a report file for this script\n",
    "with open(BGC_path+'/'+'report_2_generate_tp.txt', 'w') as f:\n",
    "    f.write('Output directory is: '+BGC_path+'\\n')\n",
    "    f.write('\\nBGC_type = '+BGC_type)\n",
    "    f.write('\\nselect_neg_genomes = '+str(select_neg_genomes))\n",
    "    f.write('\\nselect_pos_genomes = '+str(select_pos_genomes))\n",
    "    f.write('\\navoid_list = '+str(avoid_list))\n",
    "    f.write('\\nneg_isolation_source_filter = '+str(neg_isolation_source_filter))\n",
    "    f.write('\\npos_isolation_source_filter = '+str(pos_isolation_source_filter)+'\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "# load summary table into data frame () output from 1.)\n",
    "summary_file = pd.read_csv('summary.tsv', sep='\\t')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Change order of tables to prioritize samples that have an isolation source"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "warnings.filterwarnings('ignore')\n",
    "\n",
    "#filter positives and drop all duplicate protein sequences originating from different organisms\n",
    "pos_mask = (summary_file['dir'] == '+')\n",
    "pos_df = summary_file[pos_mask]\n",
    "pos_df.drop_duplicates(subset='protein_id', keep=False, inplace=True)\n",
    "\n",
    "\n",
    "#filter negatives\n",
    "neg_mask = (summary_file['dir'] == '-')\n",
    "neg_df = summary_file[neg_mask]\n",
    "\n",
    "#scoring words in isolation source so as to preferentially pick samples with chosen isolation sources\n",
    "\n",
    "def custom_sorting(source,isolation_source_filter):\n",
    "    score = 1\n",
    "    if isolation_source_filter=='pos':\n",
    "        for word in pos_isolation_source_filter:\n",
    "            if word in source:\n",
    "                score +=1\n",
    "        for word in avoid_list:\n",
    "            if source == word:\n",
    "                score=0\n",
    "    elif isolation_source_filter=='neg':\n",
    "        for word in neg_isolation_source_filter:\n",
    "            if word in source:\n",
    "                score +=1\n",
    "        for word in avoid_list:\n",
    "            if source == word:\n",
    "                score=0\n",
    "    return score\n",
    "\n",
    "\n",
    "pos_df['scoring_column'] = pos_df.apply(lambda x: custom_sorting(x['isolation_source'],'pos'),axis=1)\n",
    "neg_df['scoring_column'] = neg_df.apply(lambda x: custom_sorting(x['isolation_source'],'neg'),axis=1)\n",
    "\n",
    "pos_df.sort_values(by=['scoring_column'], axis=0, ascending=False, inplace=True)\n",
    "neg_df.sort_values(by=['scoring_column'], axis=0, ascending=False, inplace=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "moving positive GCF_002983865.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/pos_genomes\n",
      "moving positive GCF_011290545.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/pos_genomes\n",
      "moving positive GCF_009363555.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/pos_genomes\n",
      "moving positive GCF_011040495.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/pos_genomes\n",
      "moving positive GCF_009363335.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/pos_genomes\n",
      "moving positive GCF_009363615.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/pos_genomes\n",
      "moving positive GCF_002393525.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/pos_genomes\n",
      "moving positive GCF_006970865.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/pos_genomes\n",
      "moving positive GCF_003544895.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/pos_genomes\n",
      "moving positive GCF_006517275.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/pos_genomes\n",
      "generating selected_tp_genes.fasta\n",
      "generating selected_tp_aa.fasta\n",
      "moving negative GCF_009649175.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002880415.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_013283165.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_013283535.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_015356855.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002835515.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_003026515.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_003336245.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002925765.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_008634005.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002872565.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_000287135.2 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002237615.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002237815.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_000319385.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_001180305.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_001402575.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_014077345.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_001304795.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_004217665.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_001273715.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_000256485.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_004762035.2 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_006365195.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_003990085.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_003312345.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_013394655.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002000025.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002162335.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_015143645.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_007197555.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_005043705.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_005280205.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_009846745.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_013364335.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002875525.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002875045.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_004104035.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002088275.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_005040945.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_000286875.2 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_012910715.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_001971725.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_001635675.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_900163965.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002876165.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_008370595.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002836195.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_005040755.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_001005315.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_017498075.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_015709065.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_000175215.2 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_005960665.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_007859935.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_003145015.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_015594625.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_017350455.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_014168675.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_013374095.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_009790255.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_008973755.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_013416525.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "moving negative GCF_009371885.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002241655.2 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_003959225.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_000772065.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_003067445.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_018402175.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_000214435.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_000317675.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_003993825.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_014490845.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_014142695.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_001418695.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_005886095.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_004922165.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_016919125.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_013357605.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_003357915.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_006374455.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002382885.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_016603815.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002243345.2 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_016805345.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_011634775.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_015207885.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_003410255.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_001309035.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_018402495.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_003975025.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_000317265.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_003056305.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_003358145.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002891685.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_000733475.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_003253145.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_003299635.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_017354985.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002916795.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_003206625.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_011316935.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_001483235.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_009908205.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_001542775.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_007744255.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_016801025.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_016813365.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_015275235.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_017909235.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_001705665.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002230145.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_003011125.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_014141725.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_003840715.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_013393725.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_001582465.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_004922475.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002507905.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002140155.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_000491175.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_900165375.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002013415.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_000379705.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002883235.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_000332295.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_013783065.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_000313955.2 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_001953075.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_003357785.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_003348145.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002241545.2 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_014104155.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_000691865.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_001493275.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_000218095.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_003070665.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "moving negative GCF_000604425.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_004215015.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "moving negative GCF_002874535.1 to /media/manu/RiPP_Prioritiser/nitrile_hydratase_beta/base_genomes/neg_genomes\n",
      "Done\n"
     ]
    }
   ],
   "source": [
    "#Split positive genomes into 2 bins, one goes towards tp-genes and is the pos-genomes used for synthesising metagenomes\n",
    "#the other one constitutes a source of protein sequences for alignment as an input file\n",
    "\n",
    "# Genomes selected in such a way that they are from the top of the pre-sorted pos_df\n",
    "unique_pos_df = pos_df.drop_duplicates(subset='assembly', inplace=False)\n",
    "selected_tp_genomes = list(unique_pos_df.iloc[:,1])[0:select_pos_genomes]\n",
    "remaining_pos_genomes = list(unique_pos_df.iloc[:,1])[select_pos_genomes:]\n",
    "\n",
    "with open(BGC_path+'/'+'report_2_generate_tp.txt', 'a') as f:\n",
    "    f.write('\\nselected_tp_genomes are:\\n')\n",
    "    f.write(str(selected_tp_genomes)+'\\n')\n",
    "\n",
    "#select genomes and isolate GCF number from them, move selected tp genomes to final pos_genomes directory\n",
    "for genome in selected_tp_genomes:\n",
    "    print('moving positive', genome, 'to', output_pos_path)\n",
    "    !mv \"{pos_genomes_path}\"/\"{genome}\"* \"{output_pos_path}\"\n",
    "    \n",
    "#generate dataframe containing all tp-genomes and all the tp-genes contained in it\n",
    "filtered_pos_df = pos_df[pos_df['assembly'].isin(selected_tp_genomes)]\n",
    "remaining_pos_df = pos_df[~pos_df['assembly'].isin(selected_tp_genomes)]\n",
    "\n",
    "#isolate all the headers and transfer them to the selected_tp_genes file\n",
    "full_header_list = []\n",
    "for i in range(0,len(filtered_pos_df)):\n",
    "    full_header=str('>')+filtered_pos_df.iloc[i,1]+str('_')+filtered_pos_df.iloc[i,3]+str('_')+filtered_pos_df.iloc[i,5]\n",
    "    full_header_list.append(full_header)\n",
    "\n",
    "# generate fasta file with selected tp genes found in the selected genomes\n",
    "print('generating selected_tp_genes.fasta')\n",
    "with open(BGC_path+'/'+'report_2_generate_tp.txt', 'a') as f:\n",
    "    f.write('\\nselected_tp_genes in positive genomes are:\\n')\n",
    "tp_gene_counter=0\n",
    "with open(BGC_path+'/'+BGC_type+'_tp_genes.fasta') as fh:\n",
    "    lines=fh.readlines()\n",
    "    for i in range(0,len(lines)):\n",
    "        for j in range(0,len(full_header_list)):\n",
    "            if full_header_list[j] in lines[i]:\n",
    "                tp_gene_counter+=1\n",
    "                with open(BGC_path+'/'+'report_2_generate_tp.txt', 'a') as f:\n",
    "                    f.write(lines[i][1:-1]+'\\n')\n",
    "                with open(BGC_path+'/'+BGC_type+'_selected_tp_genes.fasta', 'a') as outfile:\n",
    "                    outfile.write(lines[i]+lines[i+1])    \n",
    "with open(BGC_path+'/'+'report_2_generate_tp.txt', 'a') as f:\n",
    "    f.write('\\n'+str(len(selected_tp_genomes))+' unique genomes with '+ str(tp_gene_counter)+' unique tp genes.\\n\\n')\n",
    "                    \n",
    "                    \n",
    "# transfer all amino acid sequences that are not part of the tp-genomes to a fasta file\n",
    "print('generating selected_tp_aa.fasta')\n",
    "with open(BGC_path+'/'+'report_2_generate_tp.txt', 'a') as f:\n",
    "    f.write('\\nselected_tp_aa sequences for muscle alignment are:\\n')\n",
    "tp_aa_counter = 0\n",
    "with open(BGC_path+'/'+BGC_type+'_selected_tp_aa.fasta', 'a') as outfile:\n",
    "    for i in range(0,len(remaining_pos_df)):\n",
    "        tp_aa_counter+=1\n",
    "        fasta_header=str('>')+remaining_pos_df.iloc[i,1]+str('_')+remaining_pos_df.iloc[i,3]+str('_')+remaining_pos_df.iloc[i,5]+'\\n'\n",
    "        sequence = remaining_pos_df.iloc[i,6][2:-2]+'\\n'\n",
    "        with open(BGC_path+'/'+'report_2_generate_tp.txt', 'a') as f:\n",
    "            f.write(fasta_header[1:-1]+'\\n')\n",
    "        outfile.write(fasta_header)\n",
    "        outfile.write(sequence)\n",
    "with open(BGC_path+'/'+'report_2_generate_tp.txt', 'a') as f:\n",
    "    f.write('\\n'+str(len(remaining_pos_genomes))+' unique genomes with '+ str(tp_aa_counter)+' unique aa sequences.\\n\\n')        \n",
    "        \n",
    "#Move selected neg genome files to different location\n",
    "unique_neg_df = neg_df.drop_duplicates(subset='assembly', inplace=False)\n",
    "selected_neg_genomes = list(unique_neg_df.iloc[:,1])[0:select_neg_genomes]\n",
    "\n",
    "with open(BGC_path+'/'+'report_2_generate_tp.txt', 'a') as f:\n",
    "    f.write('\\nselected_neg_genomes are:\\n')\n",
    "    f.write(str(selected_neg_genomes))\n",
    "\n",
    "for genome in selected_neg_genomes:\n",
    "    print('moving negative', genome, 'to', output_neg_path)\n",
    "    !mv \"{neg_genomes_path}\"/\"{genome}\"* \"{output_neg_path}\"\n",
    "\n",
    "    \n",
    "print('Done')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The cell below generates a muscle alignment of the protein sequences from genomes that were not chosen as pos_genomes. However, in order to follow the procedure of the 2019 Sugimoto paper more closely, instead download the seed alignment for the respective protein of interest from pfam in fasta format and save is as {BGC_type}_selected_tp_alignment.fasta"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "MUSCLE v3.8.1551 by Robert C. Edgar\n",
      "\n",
      "http://www.drive5.com/muscle\n",
      "This software is donated to the public domain.\n",
      "Please cite: Edgar, R.C. Nucleic Acids Res 32(5), 1792-97.\n",
      "\n",
      "nitrile_hydratase_beta_selected 53 seqs, lengths min 90, max 319, avg 175\n",
      "00:00:00    15 MB(-3%)  Iter   1  100.00%  K-mer dist pass 1\n",
      "00:00:00    15 MB(-3%)  Iter   1  100.00%  K-mer dist pass 2\n",
      "00:00:00    24 MB(-4%)  Iter   1  100.00%  Align node       \n",
      "00:00:00    24 MB(-4%)  Iter   1  100.00%  Root alignment\n",
      "00:00:00    24 MB(-4%)  Iter   2  100.00%  Refine tree   \n",
      "00:00:00    24 MB(-4%)  Iter   2  100.00%  Root alignment\n",
      "00:00:00    24 MB(-4%)  Iter   2  100.00%  Root alignment\n",
      "00:00:01    24 MB(-4%)  Iter   3  100.00%  Refine biparts\n",
      "00:00:02    24 MB(-4%)  Iter   4  100.00%  Refine biparts\n",
      "00:00:02    24 MB(-4%)  Iter   5  100.00%  Refine biparts\n",
      "00:00:02    24 MB(-4%)  Iter   5  100.00%  Refine biparts\n"
     ]
    }
   ],
   "source": [
    "#!module load MUSCLE/3.8.1551\n",
    "#not sure why full path is required?\n",
    "with open(BGC_path+'/'+'report_2_generate_tp.txt', 'a') as f:\n",
    "    f.write('\\n\\nMUSCLE alignment details:\\n')\n",
    "\n",
    "!muscle -in \"{BGC_path}\"/\"{BGC_type}\"_selected_tp_aa.fasta -out \"{BGC_path}\"/\"{BGC_type}\"_selected_tp_alignment.fasta -loga \"{BGC_path}\"/report_2_generate_tp.txt\n",
    "#!module unload MUSCLE/3.8.1551"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Introduce a step here where all negative genomes are concatenated, a blastdb is built from the concatenated file, and all selected TP genes are used to query the blast db. If there are any hits, remove that sequence from the selected_neg_genomes_list, and add another one, then check again."
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
